---
publish-date: '2012-05-04T09:09:00'
tags: Genomes and health
title: 'On twins, genomes, and health'
---

# On twins, genomes, and health

> Twins have sometimes no resemblance to each other.<br>
> – Aristotle, _The History of Animals_

A much-[discussed](http://blogs.nature.com/news/2012/04/questioning-value-personal-genomics.html) new [paper](http://stm.sciencemag.org/content/early/2012/04/02/scitranslmed.3003380) from Nicholas Roberts, Bert Vogelstein, and colleagues starts with a simple but important observation: so-called identical twins don't always get the same diseases.

They use that to tackle an age-old question: how genetically [_heritable_](http://en.wikipedia.org/wiki/Heritability) is each disease overall? And, beyond that, how does genetic risk vary among people? That is, for a given disease, do genomes contribute a portion of risk that varies widely, but smoothly, from person to person--or a portion of risk that varies more sharply from person to person--or some mix in between?

Below are some thoughts on reading the paper (which, while [not originally freely available](http://www.michaeleisen.org/blog/), is now free to download if you register at [Science Translational Medicine's](http://stm.sciencemag.org/) site).

## Twin studies: Gee, these findings look familiar...

Geneticists have long looked to twins to measure genetic heritability, and the new paper -- which presents no new data, but instead re-analyzes past twin studies -- follows in that tradition. Unsurprising, then, that the main conclusion boils down to the well established starting observation: *even [twins from the same egg](https://en.wikipedia.org/wiki/Twin#Monozygotic_.28identical.29_twins), who are as genetically similar as people can be, don't always get sick the same way*.

Where Vogelstein and colleagues help refine that classic question of heritability in twins is in trying to _sort_ particular diseases by estimating how well, on average, one might forecast those diseases from genomes alone.

To do so, they propose a simplistic model of the genetic basis of disease risk. Note that simplistic models are common in science -- and, in statistical genetics, in particular, we still have a lot to learn about the joint distributions of

- [allele frequency](http://en.wikipedia.org/wiki/Allele_frequency) (_How common is a given genetic variant in people around the world?_)

- allelic effects on [phenotype](http://en.wikipedia.org/wiki/Phenotype) (_How much does having a copy (or two) of this variant affect one's odds of having or getting disease X, overall?_)

- [epistasis](http://en.wikipedia.org/wiki/Epistasis) (_How do the effects of this variant depend on the presence of other variants in your genome?_)

and

- [pleiotropy](http://en.wikipedia.org/wiki/Pleiotropy) (_What other phenotypes does this variant help govern, and how do those effects interact?_).

Given those open challenges, I won't dwell here on how well assumptions in the author's model likely fit biological reality, other than to note that the authors do implicitly presume simple answers (often necessarily so) to the foregoing questions, along with some arbitrary hard thresholds for defining what it means for two people to informatively differ in disease risk. Having set their model, they then tune it to previous twin data for each of 24 diseases, to estimate how robust and variable (from person to person) resulting estimates of individual genetic risk might be.

They forecast that, for the average person, the spectrum of such risk estimates based solely on the genome may be just modestly helpful overall -- a guess that, again, accords the basic observation that monozygotic twins don't always get the same diseases. The authors go on to posit that genome data may best help clarify our personal risks better for some particular diseases –- they call out autoimmune diseases like [type-I diabetes](http://www.mayoclinic.com/health/type-1-diabetes/DS00329) and forms of [thyroid disease](http://thyroid.about.com/cs/hypothyroidism/a/hashivshypo.htm), as well as the nerve disease [Alzheimer's](http://www.mayoclinic.com/health/alzheimers-disease/DS00161) -- than for others.

And here arises a key caveat: many of the diseases they look at are _cancers_, which have long been known to be less genetically heritable than some other diseases. And, in this sense, the paper recasts old-hat knowledge as if it were a new grain of salt for the coming era of genomically personalized healthcare.

## Health: the integrative view abides.

In full disclosure, my work at [Knome](http://www.knome.com/) entails analyzing human whole genomes in order to better understand health; thus we clearly have a stake in public discussion of the prospective clinical utility of whole genome sequencing. But, in that role (where I've had to explain many a genome to many a curious, fairly healthy person), I've always stressed that one key duty of our work is to help people understand what our genomes can -- _and can't_ -- tell us. And, looking ahead to the prospect of genomically personalized healthcare, we have always understood that genomes will crucially _complement_, not replace, more conventional cornerstones of clinical care.

That is, face-to-face doctor visits, family history, lab tests, and so forth will remain essential pieces of the healthcare puzzle, soon to be joined by genome sequences. But no such piece, alone, stands to tell us everything we need to know about disease risk: lab tests often happen too late, for example; and family history is limited in utility for precisely the same reasons that this new paper highlights.

As a rough analog, note that weather forecasters use [satellite photos](http://en.wikipedia.org/wiki/Satellite_imagery) every day, for remarkably detailed insight into what's happening in the atmosphere; nonetheless, they wouldn't try to predict today's high temperature in Springfield (any of them) solely from such photos. Rather, they merge the modern, comprehensive data from orbiting cameras with data from older earthbound instruments to predict the weather (and, of course, they still get it wrong sometimes...).

Likewise, modern, comprehensive data from our genomes will likely come to play a key role in healthcare for many of us -- but always together with other key sources of medical insight. Good geneticists know that it's crucial not to hype genomes as silver bullets of healthcare. To do so would a) disserve the public, and b) in raising unreasonable expectations, risk a backlash from private and public funders of healthcare and genomic research itself.

Yet here it's worth stressing that [discoveries from individual genomes](http://www.pbs.org/wgbh/nova/body/cracking-your-genetic-code.html) have already helped many families for many decades, ever since the first readily screened [penetrant](http://en.wikipedia.org/wiki/Penetrance) genetic disease variant, for [sickle cell anemia](http://en.wikipedia.org/wiki/Sickle_cell_anemia), was [characterized](http://www.ncbi.nlm.nih.gov/pubmed/13369537) in the late 1950s. Looking ahead, as we survey more people's whole genomes, along with their diseases and other traits, our DNA will indeed tell us more and more about what makes each of us unique, and about the distinctive health risks we face.

## Toward cautious optimism

Overall, findings from the new paper model temper any undue expectation that whole genome interpretation might offer slam-dunk insight into longterm risks for the spectrum of major diseases for a typical fairly healthy person. But, perhaps promisingly, they suggest that whole genome interpretation may nonetheless offer most people -- the authors estimate more than nine in ten of us -- some significant hint of distinctive genetic risk for at least one such major disease.

As noted, they call out particular diseases as likely most often informatively predictable; admirably, in doing so, they offer specific predictions that will be testable as science progresses. In the long run, we will indeed learn whether readily genetically predictable common diseases tend to fit particular profiles; given that, as a class, cancers appear not to fit that bill, it's plausible that some other classes of disease will tend to fit better.

Moreover, the paper acknowledges the clear point that whole genome sequencing can help us spot strong risks for much rarer serious diseases that may lurk in our genomes. This may be especially useful in planning families, where we'd like to know what few rare but potentially functionally important variants, if any, we may share with a spouse (and that might be harmful if, by chance, inherited together in a child).

## Genomic medicine _presumes_ it's not all in your genes.

It's important to understand whether the new paper debunks some notion that our fates are all written in our genes. But spinning the '[no crystal ball](http://www.salon.com/2012/04/02/gene_mapping_for_everyone_study_says_not_so_fast/)' finding as big news in itself would be spinning a straw man: we've long known that genetic risk is not immutably deterministic –- and the whole field of genomic medicine is actually _founded_ on that fact. That is, in trying to understand genetic risk, we hope learn how to mitigate it, by changing the environment of our habits –- through what we eat, what drugs we take, and how/where we otherwise spend our time. Our genomes should eventually help many of us refine those choices in key ways.

## Whole genome sequencing and cancers

Although the paper rehashes the long-known point that cancer risk tends to show fairly little genetic heritability (other than for sex-specific cancers tracing to the sex chromosomes themselves), some of the first clear examples of how useful genome sequencing can be are nonetheless in familial cancers, such as cases of breast and ovarian cancers tracing to inherited variants in the [_BRCA1_](http://www.omim.org/entry/113705) and _[BRCA2](http://www.omim.org/entry/600185?search=brca2&highlight=brca2)_ genes.

Moreover, sequencing the genomes of _tumors_ (which the paper doesn't address) is already revolutionizing how cancer is treated, by finding key changes to the genomes of particular cells in the body that let them grow out of control. Such sequencing is reshaping how oncologists think of cancers – from a simplistic tissue-specific view, to one that highlights recurrent genetic changes shared by tumors from different tissues, which may represent vulnerable targets for particular drugs or other treatments.

## Further quibbles: Twins R Us?

In re-analyzing data from earlier twin studies, the paper risks some small concerns that always dog such studies. As the authors acknowledge, their papers presumes that European monozygotic twins validly represent everyone, _i.e.,_ that their assertions:

- will generalize to other ethnicities

- aren't confounded by ascertainment bias ('_Doc, my twin has disease X. Do I?_'...). Note that such bias may which might tend to _overestimate_ heritability (if twins tend to be more thoroughly diagnosed, or more often misdiagnosed thanks to another person's diagnosis, than do other people), or _underestimate_ it (if one twin, on seeing the other get sick, takes better and earlier preventive measures than (s)he would otherwise have taken).

- aren't distorted by monozygotic twins' distinctive health profiles. Potential factors to consider include: low birth weight (perhaps reflecting unusually stringent competition for resources in the womb); distinctive profiles of maternal age or genotype; subtle underlying quirks of early embryonic cell division/other physiology; lifelong social support from having a very similar sibling; etc.

The last few concerns are less damning than those above them; after all, as noted, the basic findings of the paper boil down to the fact that twins don't get the same diseases. But, like all the foregoing, they're worth keeping in mind as research into the causes of disease -- genetic and otherwise -- continues on all our behalf.
