<!doctype html>
<html lang="en-US">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">

	<!-- Mobile viewport optimized -->
	<meta name="viewport" content="width=device-width,initial-scale=1">


	<!-- Title -->

	<link rel="profile" href="http://gmpg.org/xfn/11" />
	<link rel="pingback" href="/xmlrpc.php" />

	<!-- WP Head -->
	<title>The Shock of the New: What we can say about HeLa&#8217;s novel variants &#8211; Genomena</title>
<meta name="generator" content="WordPress 4.5.6" />
<meta name="template" content="Oxygen 0.6.0" />



		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/72x72\/","ext":".png","source":{"concatemoji":"http:\/\/genomena.com\/wp-includes\/js\/wp-emoji-release.min.js?ver=4.5.6"}};
			!function(a,b,c){function d(a){var c,d,e,f=b.createElement("canvas"),g=f.getContext&&f.getContext("2d"),h=String.fromCharCode;if(!g||!g.fillText)return!1;switch(g.textBaseline="top",g.font="600 32px Arial",a){case"flag":return g.fillText(h(55356,56806,55356,56826),0,0),f.toDataURL().length>3e3;case"diversity":return g.fillText(h(55356,57221),0,0),c=g.getImageData(16,16,1,1).data,d=c[0]+","+c[1]+","+c[2]+","+c[3],g.fillText(h(55356,57221,55356,57343),0,0),c=g.getImageData(16,16,1,1).data,e=c[0]+","+c[1]+","+c[2]+","+c[3],d!==e;case"simple":return g.fillText(h(55357,56835),0,0),0!==g.getImageData(16,16,1,1).data[0];case"unicode8":return g.fillText(h(55356,57135),0,0),0!==g.getImageData(16,16,1,1).data[0]}return!1}function e(a){var c=b.createElement("script");c.src=a,c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var f,g,h,i;for(i=Array("simple","flag","unicode8","diversity"),c.supports={everything:!0,everythingExceptFlag:!0},h=0;h<i.length;h++)c.supports[i[h]]=d(i[h]),c.supports.everything=c.supports.everything&&c.supports[i[h]],"flag"!==i[h]&&(c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&c.supports[i[h]]);c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&!c.supports.flag,c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.everything||(g=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",g,!1),a.addEventListener("load",g,!1)):(a.attachEvent("onload",g),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),f=c.source||{},f.concatemoji?e(f.concatemoji):f.wpemoji&&f.twemoji&&(e(f.twemoji),e(f.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
<link rel='stylesheet' id='style-css'  href='/wp-content/themes/oxygen/style.min.css?ver=0.6.0' type='text/css' media='all' />
<link rel='stylesheet' id='font-abel-css'  href='http://fonts.googleapis.com/css?family=Abel&#038;ver=1' type='text/css' media='screen' />
<script type='text/javascript' src='/wp-includes/js/jquery/jquery.js?ver=1.12.4'></script>
<script type='text/javascript' src='/wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1'></script>
<link rel='https://api.w.org/' href='/wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="/xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="/wp-includes/wlwmanifest.xml" />
<link rel='prev' title='Harmful by any other name: On clinical variant classification' href='/2013/03/22/harmful-by-any-other-name-on-clinical-variant-classification-2/' />
<link rel='next' title='Harmful by any other name: On clinical variant classification' href='/2013/05/22/harmful-by-any-other-name-on-clinical-variant-classification/' />
<link rel="canonical" href="/2013/04/01/the-shock-of-the-new-what-we-can-say-about-helas-novel-variants/" />

<link rel="alternate" type="application/json+oembed" href="/wp-json/oembed/1.0/embed?url=http%3A%2F%2Fgenomena.com%2F2013%2F04%2F01%2Fthe-shock-of-the-new-what-we-can-say-about-helas-novel-variants%2F" />
<link rel="alternate" type="text/xml+oembed" href="/wp-json/oembed/1.0/embed?url=http%3A%2F%2Fgenomena.com%2F2013%2F04%2F01%2Fthe-shock-of-the-new-what-we-can-say-about-helas-novel-variants%2F&#038;format=xml" />

	<style type="text/css">

		/* Font size. */

		/* Font family. */

		/* Link color. */
				a:hover, a:focus { color: #000; }
		.read-more:hover, #respond #submit:hover, .button:hover, a.button:hover, #subscribe #subbutton:hover, .wpcf7-submit:hover, #loginform .button-primary:hover { background-color: #111; }

		/* Custom CSS. */


	</style>


</head>

<body class="wordpress ltr en_US parent-theme logged-out singular singular-post singular-post-37 no-js">


	<div id="container">

		<div class="wrap">


			<div id="header">


					<div id="branding">

						<div id="site-title"><a href="/" title="Genomena" rel="home"><span>Genomena</span></a></div>
						<div id="site-description"><span>As genomes evolve, so does the world.</span></div>
					</div><!-- #branding -->






			</div><!-- #header -->



			<div id="main">


	<div class="aside">




	<div id="sidebar-primary" class="sidebar">


				<section id="recent-posts-2" class="widget widget_recent_entries widget-widget_recent_entries">		<h3 class="widget-title">Recent Posts</h3>		<ul>
					<li>
				<a href="/2014/04/06/threesomics/">Threesomics</a>
						</li>
					<li>
				<a href="/2013/08/26/three-small-steps-toward-genomically-sensible-healthcare/">Three small steps toward genomically sensible healthcare</a>
						</li>
					<li>
				<a href="/2013/05/22/harmful-by-any-other-name-on-clinical-variant-classification/">Harmful by any other name: On clinical variant classification</a>
						</li>
					<li>
				<a href="/2013/04/01/the-shock-of-the-new-what-we-can-say-about-helas-novel-variants/">The Shock of the New: What we can say about HeLa&#8217;s novel variants</a>
						</li>
					<li>
				<a href="/2013/03/22/harmful-by-any-other-name-on-clinical-variant-classification-2/">Harmful by any other name: On clinical variant classification</a>
						</li>
				</ul>
		</section>		<section id="search-2" class="widget widget_search widget-widget_search">			<div class="search">

				<form method="get" class="search-form" action="/">

				<div>

					<input class="search-text" type="text" name="s" value="Search this site..." onfocus="if(this.value==this.defaultValue)this.value='';" onblur="if(this.value=='')this.value=this.defaultValue;" />

					<input class="search-submit button" name="submit" type="submit" value="Search" />

				</div>

				</form><!-- .search-form -->

			</div><!-- .search --></section><section id="categories-2" class="widget widget_categories widget-widget_categories"><h3 class="widget-title">Categories</h3>		<ul>
	<li class="cat-item cat-item-9"><a href="/category/evolution-101/" title="What the hell are geneticists talking about?  An attempt to put key terms in plain English.">Evolution 101</a>
</li>
	<li class="cat-item cat-item-7"><a href="/category/genetics-evolution-and-policy/" title="A compilation of articles (ok, screeds) on important societal issues in which technologies and insights from genetics play key roles.">Genetics, evolution, and policy</a>
</li>
	<li class="cat-item cat-item-3"><a href="/category/genomes-and-health/" >Genomes and health</a>
</li>
	<li class="cat-item cat-item-2"><a href="/category/human-population-history/" >Human population history</a>
</li>
	<li class="cat-item cat-item-8"><a href="/category/humania-research-by-people-about-people-for-people/" title="It’s true — people get way too much press, given our small numbers.  But deal with it, ok?  After all, if cuttlefish ran the web, we all know who’d get the hype.">Humania: we study…us</a>
</li>
	<li class="cat-item cat-item-10"><a href="/category/technology-how-we-look-at-dna/" title="While often confused in the popular press, science and technology are distinct concepts. The former, loosely speaking, is our systematic quest to understand the world; the latter, building tools to gain that understanding (and for other aims).  Geneticists rely on fancy gear and software to learn about genomes and the organisms that carry them.  The technology used to sequence DNA — that is, to precisely deduce the series of As, Cs, Gs, and Ts in a given genome — is changing particularly fast these days.">Seq: Reading DNA</a>
</li>
	<li class="cat-item cat-item-1"><a href="/category/uncategorized/" >Uncategorized</a>
</li>
		</ul>
</section><section id="nav_menu-8" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Video</h3><div class="menu-video-container"><ul id="menu-video" class="menu"><li id="menu-item-113" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-113"><a href="http://www.tedmed.com/videos-info?name=Ozzy_and_Sharon_Osbourne_with_Nathaniel_Pearson_at_TEDMED_2010&#038;q=updated&#038;year=all&#038;sid=28&#038;vid=2">TedMed 2010 | Your moderator interprets Ozzy Osbourne&#8217;s genome. Really.</a></li>
<li id="menu-item-114" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-114"><a href="http://www.pbs.org/wgbh/nova/body/cracking-your-genetic-code.html">NOVA: Cracking Your Genetic Code | Helping explain the prospect of genomically personalized healthcare</a></li>
<li id="menu-item-115" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-115"><a href="http://www.pbs.org/wnet/facesofamerica/video/episode-4-know-thyself/237/">Faces of America: Know Thyself | Interpreting the genomes of historian Henry Louis Gates, Jr. and dad</a></li>
</ul></div></section>

	</div><!-- #sidebar-primary .aside -->



	</div>


	<div class="content-wrap">

		<div id="content">


		<div class="breadcrumb-trail breadcrumbs" itemprop="breadcrumb">
			<span class="trail-browse">Browse:</span> <span class="trail-begin"><a href="/" title="Genomena" rel="home">Home</a></span>
			 <span class="sep">&nbsp; / &nbsp;</span> <a href="/2013/" title="2013">2013</a>
			 <span class="sep">&nbsp; / &nbsp;</span> <a href="/2013/04/" title="April 2013">April</a>
			 <span class="sep">&nbsp; / &nbsp;</span> <a href="/2013/04/01/" title="April 1, 2013">01</a>
			 <span class="sep">&nbsp; / &nbsp;</span> <span class="trail-end">The Shock of the New: What we can say about HeLa&#8217;s novel variants</span>
		</div>
			<div class="hfeed">




						<div id="post-37" class="hentry post publish post-1 odd author-npearson category-genetics-evolution-and-policy">


							<div class="post-content">


								<h2 class="post-title entry-title"><a href="/2013/04/01/the-shock-of-the-new-what-we-can-say-about-helas-novel-variants/" title="The Shock of the New: What we can say about HeLa&#8217;s novel variants" rel="bookmark">The Shock of the New: What we can say about HeLa&#8217;s novel variants</a></h2>

								<div class="byline"><a href="/author/npearson/" title="Posts by Nathaniel Pearson" rel="author">Nathaniel Pearson</a> / <a href="/2013/04/01/the-shock-of-the-new-what-we-can-say-about-helas-novel-variants/">2013-04-01</a> / <a href="/category/genetics-evolution-and-policy/" rel="category tag">Genetics, evolution, and policy</a></div>

								<div class="entry-content">

									<p>A human genome, remixed, and gone viral. In a nutshell, that&#8217;s the <a href="http://www.nature.com/news/deal-done-over-hela-cell-line-1.13511">conditionally re-published</a> genome of <a href="http://en.wikipedia.org/wiki/HeLa">HeLa</a> cells, which flourish in labs worldwide, but trace back seven decades to the unethically biopsied cervical tumor of American <a href="http://en.wikipedia.org/wiki/Henrietta_Lacks">Henrietta Lacks</a>.</p>
<p>Notably, HeLa&#8217;s genome offers vital glimpses of the woman whose life it stole. But as a posthumous portrait of Lacks, it&#8217;s at best a <a href="http://en.wikipedia.org/wiki/Cubism">cubist</a> one &#8212; her genomic likeness reduced to shards jumbled, mirrored, and spattered by decades of mutation. In time, though, it will prove to recognizably portray <em>her</em>, first in generically broad strokes, but eventually in more intimate ways that only she and her loved ones had known.</p>
<p>And that prospect sparked rancor in March, when the authors who first <a href="www.g3journal.org/content/early/2013/03/12/g3.113.005777.abstract">published</a> (and then withdrew, amid the hubbub) a HeLa genome patly <a href="http://phylogenomics.files.wordpress.com/2013/03/pr-diff.pdf">denied</a> that their data shed light on Lacks&#8217;s own <em>genome</em>, much less on who she was:</p>
<blockquote><p>
  [W]e cannot infer anything about Henrietta Lacks&#8217; genome, or of her descendants, from the data generated in this study [because] it is impossible to distinguish which parts of the genome sequenced here originate from Mrs. Lacks, her tumour, or laboratory adaptation.&#8217;
</p></blockquote>
<p>By this, they hoped to justify publishing a genome derived from one named person, without personal or family consent. That claim anticipated <a href="http://www.michaeleisen.org/blog/?p=1341">lively new discussion</a> of <a href="http://www.technologyreview.com/view/512966/the-dawn-of-genome-trolling/">genomic privacy</a> &#8212; but it was false, roughly like the finder of a rain-soaked wallet conveniently declaring the banknotes legible, but the IDs too wet to read.</p>
<p>Their own paper, after all, noted that HeLa shares nearly every noteworthy letter of its genome with other people (though their log-scaled bar plots downplayed this). And eventually, prodded by <a href="http://www.nytimes.com/2013/03/24/opinion/sunday/the-immortal-life-of-henrietta-lacks-the-sequel.html?src=me&amp;ref=general&amp;_r=1&amp;">other voices</a>, they conceded that we <em>can</em> learn and confirm some things about Lacks from HeLa&#8217;s DNA (or even <a href="http://www.genomesunzipped.org/2013/03/henrietta-lackss-genome-sequence-has-been-publicly-available-for-years.php">RNA</a>)[1] &#8212; but, they insisted, little if anything &#8216;new&#8217;.</p>
<p>And that&#8217;s truth&#8230;if not whole truth. For the authors apparently took <em>learning something</em> to mainly mean figuring out which of the 550 thousand or so <em>novel</em> (never before seen) variants they found had originally belonged to Lacks.[2] And yes, doing so without comparing HeLa&#8217;s genome directly to a non-HeLa genome from Lacks is indeed tough &#8212; at first glance, a genomic one-hand-clapping <a href="http://en.wikipedia.org/wiki/koan">koan</a>.</p>
<p>But even if identifying such variants really were the only route to learning something about Lacks, the authors ignored ways in which we <em>can</em> do so. Below, I&#8217;ll detail a few &#8212; which all boil down to thinking carefully about particular branches of the genetic family trees that link each segment of HeLa&#8217;s genome to the genomes of other cells and people.</p>
<p>Let&#8217;s start with novel variants that likely did <em>not</em> belong to Lacks.</p>
<h2>Novel variants that likely arose only in the HeLa cell lineage</h2>
<h3>Single-copy variants in duplicated haplotypes</h3>
<p>The HeLa genome is not easy reading. It&#8217;s basically a long, gibberish-spiked recipe for making tons of human cells. And if such a postmodern cookbook sounds riveting, you&#8217;ll be pleased to know that it&#8217;s written mostly in lawyerly triplicate.</p>
<p>In the decades after Lacks was born (perhaps in one big event, such as a faulty cell division), HeLa gained an extra copy of nearly every chromosome. And, though the new paper missed the next point, HeLa&#8217;s extensive <em><a href="http://en.wikipedia.org/wiki/Polyploid">triploidy</a></em> can help clarify which of its many distinctive variants arose by new mutation in the cell line, after Lacks was conceived (and perhaps after she died).</p>
<p>Basically, the duplication(s) that turned HeLa triploid effectively sprouted tiny new branches in the human genome&#8217;s family treetops, but <em>within</em> the HeLa genome itself. And some novel variants in HeLa can be confidently traced to those new branches, clarifying that they likely arose by mutation after Lacks was conceived.</p>
<p>Specifically, if a novel variant appears on just one of the three copies of a given chromosome in HeLa, <em>and the surrounding sequence is otherwise nearly identical to just one of the two other copies</em>, then the variant in question most likely arose by such new mutation. That is, Lacks likely wasn&#8217;t born with it. Similar logic can help us pick out likely new mutant variants in parts of the HeLa genome (such as long stretches of chromosomes 1, 5, and 11) that show even <em>more</em> than three copies.</p>
<p>Overall, to figure out which of the half-million or so heterozygous novel variants found in HeLa go into this bin, we&#8217;d need to systematically check which other variants they&#8217;re linked to on particular copies of particular chromosomes. With current lab methods, that&#8217;s hard &#8212; especially for novel variants, where we can&#8217;t turn to reference data from human populations &#8212; but feasible <strong>[Update: a new <a href="http://www.nature.com/nature/journal/v500/n7461/full/nature12064.html">paper</a> from <a href="http://www.gs.washington.edu/faculty/shendure.htm">Jay Shendure</a>&#8216;s group reports doing just that!]</strong>.</p>
<h3>Heterozygous mitochondrial variants</h3>
<p>Mitochondria are our cells&#8217; tiny furnaces. Each one houses a short, gene-crammed ring of DNA, attesting direct descent from a bacterium that, eons ago, slipped inside a bigger cell, to found <a href="http://en.wikipedia.org/wiki/Eukaryote">a line of ancestors</a> that we share mainly with plants, molds, and other animals.</p>
<p>Because all your mitochondria come from the few that your mother&#8217;s egg carried, they tend to be nearly identical within each cell, and throughout your body. That is, though their genomes do mutate (so can vary a bit from mitochondrion to mitochondrion), any resulting variation tends to wash out when we read a person&#8217;s DNA <em>en masse</em>.</p>
<p>As such, in a person&#8217;s mitochondria, we expect to find very few genomic sites &#8212; typically zero, in some people one &#8212; that look heterozygous (coming in two or more distinct flavors within a given genome) in the classic genetic sense. In fact, if we see more than one such site, we can guess fairly strongly that something has gone wrong either with sequencing itself (such as sample cross-contamination), or with the genome we&#8217;re looking at.</p>
<p>The HeLa researchers didn&#8217;t report how many of the novel mitochondrial variants, if any, appeared heterozygous in the HeLa genome. But if any did, it&#8217;s a good bet that most or all of them arose long after Lacks was born. And though this bin is likely very small, it&#8217;s one that can be fairly confidently assessed, in part by picking through raw sequence data more carefully; as such, it might tell us a <em>bit</em> about the point mutation rate in HeLa, relative to other tumor cell lines.</p>
<h3>Variants that don&#8217;t turn up in other HeLa lines</h3>
<p>This is the first big, obvious bin that will eventually subsume many of HeLa&#8217;s currently puzzling variants. Many variants that appear in the recently sequenced HeLa line will turn out not to appear in most or all other lines that we might eventually look at. While this can reflect <em>loss</em> of such variants, by deletion of segments of DNA in some HeLa lines, it more often will reflect <em>gain</em> of the variant in question in this particular line.</p>
<p>As such, fleshing out the genetic family tree of other HeLa lines will help clarify which such variants trace specifically to the branch that the new paper focuses solely on. Now, on to the other class of novel variants that interest us: those that likely belonged to Lacks from the start&#8230;</p>
<h2>Novel variants that Henrietta Lacks was likely conceived (and born) with</h2>
<h3>Novel ancestral variants</h3>
<p>In evolutionary parlance, an ancestral variant is, specifically, a variant carried by the last common ancestor of <em>everyone</em> in a population, for a given stretch of the genome, <em>before</em> any mutation there gave rise to a new variant that&#8217;s now found in some but not all individuals.</p>
<p>Most ancestral variants are still common among us, so were discovered long ago. But the haul of novel variants found in a given person may include the odd ancestral one that, til now, had never been hooked by researchers fishing in our gene pool.</p>
<p>Most such variants can be spotted easily, because they&#8217;re shared with many other mammals. Sometimes, such matches just reflect <em><a href="http://en.wikipedia.org/wiki/Convergent_evolution">homoplasy</a></em> – that is, mutation that happened to yield a variant already found in other organisms, perhaps at sites that tend to mutate a lot (such as Cs before Gs). But the commoner case &#8212; and the simpler guess, given how rarely mutation&#8217;s lightning strikes a typical site in the genome (and how rarer still it is for the mutation to happen to yield a particular variant seen elsewhere too) &#8212; is that a variant shared with most other mammals is really ancestral.</p>
<p>As such, among the novel variants called in HeLa, a few will confidently look ancestral &#8212; and most of these were, very likely, inherited by Lacks from her parents.</p>
<h3>Homozygous novel variants, outside LOH and/or cancer-implicated regions</h3>
<p>Whether ancestral or otherwise, novel variants tend to be rare &#8212; so rare that, by definition, they&#8217;d never been seen in a person before. As such, a given novel variant in your genome is very unlikely to also be carried by a randomly (er, carefully) chosen mate. And even if that person does carry a copy of the same very rare variant, there&#8217;s rarely more than a one in four chance that a child you&#8217;d have together will get two copies of it (one from each parent).</p>
<p>As such, nearly all novel variants found in your genome (or anyone&#8217;s) are <em>heterozygous</em> – that is, found on just one of your two copies of a given chromosome. Nonetheless, a smattering of novel variants may instead be homozygous, by the jackpot luck described above.</p>
<p>The HeLa genome reportedly harbors roughly 50 thousand homozygous novel variants &#8212; and that&#8217;s quite a lot, compared to most human genomes. Given that Lacks&#8217;s parents weren&#8217;t close relatives, I&#8217;d guess that many of these are in long stretches of the HeLa genome (such as most of chromosomes 6 and X) that look uniformly homozygous, likely due to <a href="http://en.wikipedia.org/wiki/Loss_of_heterozygosity">loss of heterozygosity</a>, where HeLa gained or lost copies of the given genome segment. Within these segments, and in genes that likely help govern HeLa&#8217;s growth, the origins of homozygous novel variants are hard to guess. Some may indeed have popped up by mutation in HeLa lineage, then become homozygous by secondary mutation. But others may have been in Lacks&#8217;s genome from the start.</p>
<p>But outside such regions, most homozygous variants found in HeLa &#8212; novel or otherwise &#8212; likely belonged to Lacks from the start. And we&#8217;ll eventually be able to confirm many of these guesses by looking at the next bin, below&#8230;</p>
<h3>Variants that turn up in other people</h3>
<p>This is the <em>other</em> big, obvious bin that will eventually subsume many of HeLa&#8217;s currently puzzling variants (the converse of the variants-that-don&#8217;t-turn-up-in-other-HeLa-lines bin described above). Of the 550-thousand-odd variants so far seen only in HeLa, many thousands will likely eventually be found in the genomes of living people whom we&#8217;ve yet to sequence. And most of these &#8212; especially if flanked by stretches of sequence that distinctively resemble the sequence flanking them in HeLa &#8212; will indeed be variants that Lacks was born with (rather than variants that arose in parallel by new mutation).</p>
<p>As such, it&#8217;s just a matter of time before we can trace many of the novel variants in HeLa to particular branches of the bigger human genetic family tree.</p>
<p>Importantly, some of these variants will, in turn, become functionally informative, once we&#8217;ve seen them enough times to see what kinds of traits people who carry them may distinctively share. Looking ahead, this is where HeLa&#8217;s genome will eventually show similarities to other genetic portraits of individuals, which in turn refine not just our picture of Lacks&#8217;s genome, but of what she was like as a person, inside and out.</p>
<h2>Further thoughts</h2>
<p>On first hearing that HeLa had been publicly sequenced, I smiled &#8212; a bit wistfully &#8212; and recalled stumping, in 2009, to do just that[3]. If you&#8217;re working at a genome interpretation company, and want to pick just one human(oid) genome to sequence for public good, you could do far worse than HeLa. It&#8217;s a ubiquitous cell line, after all, whose genome says a bit about cervical cancer; a bit more about how lab-farmed cells find ways to thrive; and, yes, a lot about Henrietta Lacks, who, suffering in rough conditions in a segregation-era cancer ward, made an unwitting, coerced, but remarkable bequest to human knowledge and health.</p>
<p>In these senses, the HeLa genome could anchor an engaging public panel discussion among geneticists, oncologists, ethicists, and, crucially, members of the Lacks family. Glibly, it might start with a bird&#8217;s eye look at cancer as <a href="/genomes-and-health/why-cancers-are-the-least-and-most-genetic-of-diseases#hive">doomed cellular mutiny</a>. Next, meeting Henrietta herself through family memory, we&#8217;d learn how her particular tumor <a href="/genomes-and-health/why-cancers-are-the-least-and-most-genetic-of-diseases#HeLa">eluded such doom</a> by appealing, ironically, to humanity&#8217;s needs. That notion raises the thorny ethics of using a useful but unethically obtained resource (with no direct recompense to the family who suffered most in giving it to the world), and in turn opens the meta-wormcan of genomic privacy. And new insights from the HeLa genome itself help weave these threads into emerging genomic portraits of the cell line and the little-sung woman it traces to.</p>
<p>As noted, such a discussion would most enlighten if it directly included members of the Lacks family, who could reflect personally on Henrietta, HeLa, and genetic privacy &#8212; and who embody how segments of her genome also live on, most vitally, in the kin she loved and would have loved.</p>
<p>Pat, yes. But HeLa&#8217;s genome clearly adds something to an already compelling tale &#8212; one that <a href="http://rebeccaskloot.com/">Rebecca Skloot</a>, who had <a href="http://www.jhu.edu/~jhumag/0400web/01.html">long been writing</a> about Lacks and her cells, soon told in much finer detail, in <em><a href="http://www.nytimes.com/2010/02/07/books/review/Margonelli-t.html?pagewanted=all">The Immortal Life of Henrietta Lacks</a></em>.</p>
<p>As that book makes clear, the HeLa saga has long blended travesty, triumph, and quandary in roughly equal parts. The debate over its genome is thus just a new eddy of an old storm. The last few months, like the previous 71 years, have seen useful data tainted by heedless ethics; public efforts to mend, if not undo, a wrong; and, perhaps most pressingly, lasting questions about how we should use scientific resources whose provenance is, at best, murky.</p>
<p>We haven&#8217;t dwelled here on how, in bypassing the Lackses, the first HeLa genome paper furthered a history of disregard for real <em>people</em> who carry forth Henrietta&#8217;s DNA, and who mourn her most deeply. Nor have we tried to define what genetic privacy means, given that far more of <em>any</em> genome is complicatedly shared &#8212; with kin, both known and unknown &#8212; than is truly private.</p>
<p>Rather, we&#8217;ve focused on the question of what we can indeed learn from a single genome, which matters for three main reasons.</p>
<p>First, the question cuts across the authors&#8217; initial claim that the genome of HeLa (read: any tumor or cell line) told us nothing about Henrietta Lacks (read: any person from whom such cells are taken). That technical claim doesn&#8217;t hold up, as we&#8217;ve seen.</p>
<p>Second, it points, more broadly, to what we can, and cannot, reasonably hope to learn today about a person just by looking at a genome. The paper itself confirmed, via HeLa&#8217;s genome, that Lacks was indeed a woman with largely west African and European ancestry. And some of us noted that we could go deeper, using <a href="http://www.1000genomes.org/">public data</a> to guess which variants were attached to which others in Lacks&#8217;s chromosomes, ultimately clarifying which parts of the world particular stretches of her genome trace to, and even which particular people are her close cousins in particular parts of the genome.</p>
<p>Moreover, the paper&#8217;s authors soon acknowledged that well studied variants that HeLa shares with other human genomes can let us guess at particular traits that Lacks may have had, and at various disease risks that she might have faced, had she not died of cervical cancer at thirty-one. But the bottom line on this front is that there&#8217;s still much we <em>can&#8217;t</em> guess from looking at one genome &#8212; even the <em>best</em> sequenced one. As more people&#8217;s genomes and traits are surveyed over time, however, the HeLa genome will indeed reveal deeper insights about who Lacks was. Facets of her uniqueness &#8212; voice and gait, quirks, ailments, and interests &#8212; may someday be traceable, in part, to distinctive genomic spellings that for now hide mum in plain sight.</p>
<p>Third, the technical question of what we can learn raises thornier <em>policy</em> questions about data privacy, ownership, publishing, and consent. And here, perhaps, likening HeLa&#8217;s genome to a remixed or cubist rendition of Lacks&#8217;s was apt. A person&#8217;s DNA uniquely quilts together genome segments shared with others (kin, both near and far), peppered with a few brand new variants (by mutation). As such, it&#8217;s neither fully original, nor fully derivative &#8212; like a work of art, necessarily influenced by broader culture. And, perhaps notably, tough questions about sampling and provenance (<a href="http://en.wikipedia.org/wiki/Elgin_Marbles">Elgin marbles</a>, anyone?) have long raged in the art world, with few clear answers. As such, prepare for a vocal, but likely indefinite societal conversation on that front.</p>
<p>[^1] The paper itself made this clear by showing, from HeLa’s genome, that Lacks was indeed a woman with largely west African and European ancestry. Some of us noted that we could go deeper, using public data to guess which variants were attached to which others in Lacks’s chromosomes, clarifying which parts of the world particular stretches of her genome trace to, and even which particular people are her close cousins in particular parts of the genome.</p>
<p>Moreover, some common variants that HeLa shares with other human genomes let us guess at traits that Lacks may have had, and at various disease risks that she might have faced, had she not died of cervical cancer at thirty-one.</p>
<p>[^2] In this sense, the HeLa genome controversy echoes recent debate over <a href="http://www.genome.gov/10005107">ENCODE</a>, an effort to survey which parts of our genomes govern how much of each protein gets made, where, and when. The ENCODE controversy has hinged, in part, on what the word <em>functional</em> means; for HeLa, the key question is what it means to &#8216;learn nothing&#8217; about a genome or person.</p>
<p>Note here that the HeLa authors might likewise have gone out of their way to tell us that we can learn &#8216;nothing about cell lines&#8217; from the HeLa genome. After all, it&#8217;s -possible- (though astoundingly unlikely) that the whole HeLa genome &#8212; triploid chromosomes and all &#8212; is exactly the same genome, letter for letter, that Lacks was born with.</p>
<p>[^3] While HeLa&#8217;s genome long intrigued me and others, several technical, funding, and strategic hurdles thwarted the idea back then. And here it bears note, at risk of sanctimony, that our brainstorming on HeLa was <em>always</em> premised on the blessing and direct participation of the Lacks family; had the project been green-lighted internally, step one would have been reaching out to them. So when I learned that the authors of the HeLa paper hadn&#8217;t engaged the Lackses, my heart, like others, sank&#8230;and when I then read the authors&#8217; original FAQ, which protested (too much) that their data said nothing about Lacks herself, my gorge rose, prompting this post.</p>
<hr />
<h2>Update (7 August 2013):</h2>
<p>Lacks&#8217;s descendants will <a href="http://www.nature.com/news/deal-done-over-hela-cell-line-1.13511">reportedly</a> anchor a new committee to consider, case-by-case, requests to use HeLa genome data in biomedical research. Brokered by the family, Skloot, <a href="https://en.wikipedia.org/wiki/Francis_Collins">Francis Collins</a>, and others, the arrangement won&#8217;t make the HeLa data fully open (so most useful), keeping it instead in NIH&#8217;s credential-walled <a href="http://www.ncbi.nlm.nih.gov/gap">dbGAP</a>. Nonetheless, it&#8217;s a gracious and scientifically substantive gesture by the family, whose preference understandably trumps other factors.</p>
<p>HeLa was, after all, derived surreptitiously from their own foremother (and, if all her descendants agreed, no one else&#8217;s) without informed consent. That wrong, along with the decades more of scientific ill-treatment of the Lackses that followed, remain the heart of the matter, even if the HeLa saga also highlights</p>
<ul>
<li>broader dilemmas around publishing genome data from explicitly named people, given that doing so makes it much easier to guess who else (their identifiable kin) likely carries many of their distinctive genetic spellings too.</li>
<li>more hearteningly, the potential benefits of directly engaging layfolk as partners in interpreting data they give for societal benefit</li>
</ul>
<p>For now, here&#8217;s to new discoveries that the soon-to-be accessible data will unlock, including those about Lacks herself. Happily that&#8217;s starting already: the data access plan itself was announced alongside a new <a href="http://www.nature.com/nature/journal/v500/n7461/full/nature12064.html">paper</a> dissecting the HeLa genome in unprecedented detail &#8212; notably, with phased haplotypes that should, as noted above, help us better guess which variants Lacks was indeed born with, and which arose only in the cells that became her tumor.</p>


								</div><!-- .entry-content -->

								<div class="entry-meta"></div>


							</div><!-- .post-content -->

						</div><!-- .hentry -->
			</div><!-- .hfeed -->




		<div class="loop-nav">
			<div class="previous"><a href="/2013/03/22/harmful-by-any-other-name-on-clinical-variant-classification-2/" rel="prev">&larr; Harmful by any other name: On clinical variant classification</a></div>			<div class="next"><a href="/2013/05/22/harmful-by-any-other-name-on-clinical-variant-classification/" rel="next">Harmful by any other name: On clinical variant classification &rarr;</a></div>		</div><!-- .loop-nav -->


		</div><!-- #content -->






	<div id="sidebar-secondary" class="sidebar">


		<section id="nav_menu-5" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Reference</h3><div class="menu-reference-container"><ul id="menu-reference" class="menu"><li id="menu-item-95" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-95"><a href="http://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI): Most biologists&#8217; de facto home page. You can use it too.</a></li>
<li id="menu-item-96" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-96"><a href="http://tolweb.org/tree/">Tree of Life: Earth&#8217;s diversity of life forms, and how they&#8217;re related to one another</a></li>
<li id="menu-item-97" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-97"><a href="http://genome.ucsc.edu/">UCSC genome browser: Fast, small-footprint rival to NCBI for genome sequence data-mining</a></li>
</ul></div></section><section id="nav_menu-4" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Fun</h3><div class="menu-fun-container"><ul id="menu-fun" class="menu"><li id="menu-item-98" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-98"><a href="http://www.facebook.com/group.php?sid=0b12e602a592d4dc019506c99b641cf2&#038;gid=53660447230&#038;ref=search">Effective Human Population (at Facebook): The Few. The Proud. Damn, it feels good to be effective.</a></li>
<li id="menu-item-99" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-99"><a href="http://helenpearsonphotography.com/music">Photo break: atmosphere-soaked images of music by National Geographic award-winning photographer Helen Pearson</a></li>
</ul></div></section><section id="nav_menu-6" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Data analysis</h3><div class="menu-data-analysis-container"><ul id="menu-data-analysis" class="menu"><li id="menu-item-100" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-100"><a href="http://opensnp.org/">openSNP: Portal for sharing, and fruitfully comparing, personal genome data.</a></li>
</ul></div></section><section id="nav_menu-7" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Blogroll</h3><div class="menu-blogroll-container"><ul id="menu-blogroll" class="menu"><li id="menu-item-101" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-101"><a href="http://gcbias.org/">GCbias: Graham Coop makes genomic complexity, well, fun.</a></li>
<li id="menu-item-102" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-102"><a href="http://genomesunzipped.org/">Genomes Unzipped: Thoughtful reflections on personal genomics by leading lights in the field.</a></li>
<li id="menu-item-103" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-103"><a href="http://haldanessieve.org/">Haldane&#8217;s Sieve: Read about new research here first, and straight from authors.</a></li>
<li id="menu-item-104" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-104"><a href="http://caseybergman.wordpress.com/">I Wish You&#8217;d Made Me Angry Earlier: Casey Bergman, voice of genomic reason and memory.</a></li>
<li id="menu-item-105" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-105"><a href="http://www.michaeleisen.org/blog/">it is NOT junk: Trenchancy from Michael Eisen. Read this.</a></li>
<li id="menu-item-106" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-106"><a href="http://massgenomics.org/">MassGenomics: Worthwhile thoughts from Dan Koboldt</a></li>
<li id="menu-item-107" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-107"><a href="http://www.yuzuki.org/">Next Generation Technologist: Dale Yuzuki opines on deep-sequencing methods and insights</a></li>
<li id="menu-item-108" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-108"><a href="http://omicsomics.blogspot.com/">Omics! Omics! Plumb Keith Robison&#8217;s knowhow on methods and tools for understanding genomes.</a></li>
<li id="menu-item-109" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-109"><a href="http://phenomena.nationalgeographic.com/">Phenomena: Lively summaries of eclectic new research from Ed Yong, Carl Zimmer, and others. (And where else would Genomena take you, after all?…)</a></li>
<li id="menu-item-110" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-110"><a href="http://phylogenomics.blogspot.com/">Phylogenomics: Jonathan Eisen surfs the sea of sequence</a></li>
<li id="menu-item-111" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-111"><a href="http://feeds.feedburner.com/RazibKhansTotalFeed">Razib Khan&#8217;s feed: Savvy (often bitingly so) genomic insights into human ancestry and behavior.</a></li>
<li id="menu-item-112" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-112"><a href="http://whyevolutionistrue.wordpress.com/">Why evolution is true: Sageness from the broad and incisive mind of Jerry Coyne.</a></li>
</ul></div></section>

	</div><!-- #sidebar-secondary .aside -->



				</div><!-- .content-wrap -->


		</div><!-- #main -->





		<div id="footer">


			<div id="footer-content" class="footer-content">
            <p class="copyright">Copyright &#169; 2017 <p class="credit"> Powered by <a href="http://alienwp.com">Oxygen Theme</a>.</p>



			</div>




		</div><!-- #footer -->


		</div><!-- .wrap -->

	</div><!-- #container -->


	<script type='text/javascript' src='/wp-content/themes/oxygen/js/jquery.imagesloaded.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/jquery.masonry.min.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/cycle/jquery.cycle.min.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/fitvids/jquery.fitvids.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/navigation.js?ver=20130301'></script>
<script type='text/javascript'>
/* <![CDATA[ */
var slider_settings = {"timeout":"6000"};
/* ]]> */
</script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/footer-scripts-light.js?ver=1.0'></script>

<script type='text/javascript' src='/wp-content/themes/oxygen/library/js/drop-downs.min.js?ver=20130805'></script>
<script type='text/javascript' src='/wp-includes/js/wp-embed.min.js?ver=4.5.6'></script>

</body>
</html>
