<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Evolution 101 &#8211; Genomena</title>
	<atom:link href="/category/evolution-101/feed/" rel="self" type="application/rss+xml" />
	<link>http://genomena.com</link>
	<description>As genomes evolve, so does the world.</description>
	<lastBuildDate>Wed, 01 Feb 2017 02:46:36 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.5.6</generator>
	<item>
		<title>The myriad throng: Human effective population size</title>
		<link>/2012/06/24/the-myriad-throng-human-effective-population-size/</link>
		<comments>/2012/06/24/the-myriad-throng-human-effective-population-size/#respond</comments>
		<pubDate>Sun, 24 Jun 2012 10:42:34 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Evolution 101]]></category>

		<guid isPermaLink="false">/?p=46</guid>
		<description><![CDATA[Economy recapitulates demography? (image source) Iranians count their rials in superunits of the toman, from a Mongol word for ten thousand soldiers. Remarkably, such a...]]></description>
				<content:encoded><![CDATA[<p><img src="/wp-content/uploads/2012/06/10000-iranian-rial-zero-toman32.jpeg" alt="" title="toman" /></p>
<p><strong>Economy recapitulates demography?</strong> (<a href="http://gietalks.wordpress.com/2012/01/31/bills-for-thrills/10000-iranian-rial-zero-toman-2/">image source</a>) <a href="/humania-research-by-people-about-people-for-people/the-human-geneticist-at-large-notes-from-the-field/human-genetic-diversity-fieldwork-in-iran/">Iranians</a> count their <em>rials</em> in superunits of the <em>toman</em>, from a Mongol word for <a href="http://en.wikipedia.org/wiki/Tumen">ten thousand soldiers</a>. Remarkably, such a modest-sized horde could readily harbor much of the genetic diversity that long characterized people worldwide. But, with prices rising in Iran, a toman can <a href="http://en.wikipedia.org/wiki/Iranian_toman">reportedly</a> now mean ten <em>million</em> rials. And a new genetic census of humanity hints, likewise, at rampant inflation in our ranks&#8230;</p>
<p>Several weeks ago, we reviewed how a <a href="/genomes-and-health/rare-variants-disease-and-population-size/">newly documented trove</a> of rare variants in human genomes tracks our ballooning population. Most such variants, we saw, arose by recent mutations near the tips of our family tree &#8212; and now, like yuletide lights, mark out the tree&#8217;s fast-sprouting outer branches. The lengths of those branches, in turn, reflect a great proliferation of human ancestors as our population has exploded during the past few millenia.</p>
<p>But despite the rare variants that track our recent population growth, human genomes are still strikingly alike. Geneticists have long known, in particular, that two randomly chosen copies of a typical human chromosome tend to differ in spelling at less than one in a thousand letters.</p>
<p>Given what&#8217;s known about how fast DNA mutates, this is remarkably little variation. So little, in fact, that, despite our teeming numbers, the stock of variation in our genomes is, at first glance, roughly what we&#8217;d expect to find in a steady-sized population of just <em>ten thousand</em> randomly mating individuals.</p>
<p>So what gives? How can our genomes suggest a population that&#8217;s bursting the planet&#8217;s seams, yet small enough to gather in <a href="http://en.wikipedia.org/wiki/Centre_Court">Wimbledon Centre Court</a>?</p>
<p>Below we&#8217;ll learn how to guess directly at a population&#8217;s size from the profile of genetic variation that it carries, retracing how geneticists originally arrived at the ten thousand figure. And we&#8217;ll see how more nuanced math is revising our genetic census to better reflect the explosive growth that has loaded today&#8217;s billions of real human genomes with so many rare new variants.</p>
<h2>Consider a spherical gene pool&#8230;</h2>
<p>Geneticists use the term <a href="http://en.wikipedia.org/wiki/Effective_population_size">effective population size</a> to mean the number of idealized individuals who would typically harbor the same genetic diversity seen in a set of real organisms (whom the idealized ones &#8216;effectively&#8217; represent). In shorthand, this size is denoted <em>Nₑ</em>, where <em>N</em> stands for &#8216;number&#8217; and <em>e</em> stands for, well, &#8216;effective&#8217;.[1]</p>
<p>Ultimately, the concept of <em>Nₑ</em> is the kind of simplification that a physicist might make in predicting the arc of a cow launched from a circus cannon. Bear with me on this, ok?</p>
<p>A cow is a complex lump of stuff, and modeling its flight precisely, down to the last whisker, would take great knowledge and computing power. But, where reasonable, scientists like to lazily simplify things. So, rather than try to account for every cranny of the cow&#8217;s shape, every quirk of density and electric charge inside it, and every component of its motion (methane release-induced yaw&#8230;), a physicist might model the cow as a chargeless point mass flying in a vacuum.</p>
<p>Because such an idealized mass would fly in a simple path, immune to aerodynamic, electrostatic, buoyant (were this an underwater flight) or other such real-cow effects, it might be, say, a bit lighter than the real cow and still fly just as far. A physicist might think of that smaller mass as the &#8216;effective mass&#8217; of the cow.</p>
<p>Or, more realistically, our physicist might accept that cows fly through real air (or water), and have nonzero volume, and so model the cow (still simplistically, of course) as a symmetrical ball with the cow&#8217;s mass and net density. To the extent that the flight of such a ball approximated that of the real cow, the ball&#8217;s radius could be called the &#8216;effective radius&#8217; of the cow.</p>
<p>Geneticists can model the evolutionary trajectories of populations, on likewise simplistic lines. Rather than try to track (or guess precisely at) every detail of mating, fertility, migration, and mutation in a population&#8217;s history, they often make simple assumptions about those processes. And those assumptions, to the extent valid, can point to an &#8216;effective size&#8217; that gives a rough handle on how a given population is evolving.</p>
<h2>Happy little trees</h2>
<p>The link between population size and genetic variation can be seen most clearly through the lens of genealogical tree shape, as introduced <a href="/genomes-and-health/rare-variants-disease-and-population-size#tree-shape">here</a>.</p>
<p>To put hard numbers on that connection, let&#8217;s sketch the family tree of all copies of just one site in the genomes of a population. Taking a cue from the great <a href="http://www.nytimes.com/2001/11/18/arts/television/18MORF.html?pagewanted=all">Bob Ross</a>, we&#8217;ll paint our tree in simple strokes: it will represent a population of some big, <em>constant</em> number <em>N</em> individuals, who breed at most once, in discrete generations, blindly with respect to their genotypes, without complications from factors like <a href="http://en.wikipedia.org/wiki/Natural_selection">natural selection</a>, <a href="http://pritch.bsd.uchicago.edu/publications/pdfs/RosenbergEtAl02.pdf">geography</a>, or <a href="http://en.wikipedia.org/wiki/Genetic_recombination">recombination</a>.[2]</p>
<p>If these <em>N</em> idealized individuals have simple <a href="http://wikipedia.org/diploidy">two-copy</a> genomes like our own, then each site in those genomes has 2_N_ total copies in the population overall. The family tree for a given site then represents the ancestries of all 2_N_ copies as a skeletal graph, tracing how they&#8217;ve been copied from one generation to the next.</p>
<p>Now, here&#8217;s a crucial point about our model: we&#8217;ll stipulate that, looking back in time, toward the root of the tree, each copy of the site in a given generation traces back <em>at random</em> to one of the 2_N_ copies in the generation before &#8212; and does so regardless of whether any other copy also traces back to that same previous-generation copy.</p>
<p>This means that, while there are always 2_N_ copies of the site in the model population, only <em>some</em> of them, in any given generation, are copied into the next one. That is, a given copy today may give rise to more than one of tomorrow&#8217;s 2_N_ copies &#8212; while other copies, by chance, end up as evolutionary dead ends.[3]</p>
<p>And that sheer randomness is, ironically, where the math gets precisely predictive.</p>
<p><img src="/wp-content/uploads/2012/06/PastNowTree.png" alt="" title="tree" /></p>
<p>Tree (bold green) tracing the ancestry of a population of ten modern copies of a genomic site, amid branches of overall genealogy (gray) of constant-sized population. Arrows point forward in time, linking one generation to the next; coalescences are found by tracing lineages backward (toward left) til they meet. (figure adapted from <a href="http://www.stats.ox.ac.uk/~mcvean/L2slides4.pdf">lectures</a> by <a href="http://www.stats.ox.ac.uk/~mcvean/">Gil McVean</a>)</p>
<h2>Chance encounters</h2>
<p>Notice, in particular, that our simple model gives any two copies of the site in a given generation a one-in-2_N_ chance of tracing back to the same parent copy in the previous generation.</p>
<p>Because <em>N</em> is big, that chance is tiny. But (but!&#8230;) if we look back over enough generations, the lineages of those two copies will eventually meet at some shared ancestor. That is, chance dictates that they won&#8217;t forever keep tracing to separate ancestral copies, but at some point will happen to trace to the same one.</p>
<p>Such random lineage mergers &#8212; which geneticists call <a href="http://en.wikipedia.org/wiki/Coalescent_theory">coalescences</a> &#8212; are, in the end, why the family tree of the <em>2N</em> copies is indeed a tree, rather than just a bundle of 2_N_ endless, unconnected vines.</p>
<p>And, while the math in question looks hairy at first, it works out that, for big <em>N</em> under our simplistic assumptions, the <em>mean</em> number of generations that it takes two copies&#8217; lines to meet (looking back in time) approaches 2_N_. That is, the average time back to a merger becomes the reciprocal of the one-in-2_N_ chance of a merger in any single generation.</p>
<p>A neat, simple result.</p>
<p>Of course, if you use a computer to simulate the ancestries of many paired copies of the site under our model, you&#8217;ll find that the actual number of generations since coalescence varies randomly &#8212; a lot &#8212; around this average. But the average itself is nonetheless very handy.</p>
<h2>Mutations: buds on the branches</h2>
<p>In particular, knowing the average time separating two randomly chosen copies of a site means that, if we also know how often a typical copy <em>mutates</em> (randomly changes in DNA spelling), we can estimate how many mutations will most typically have happened along the two branches that, meandering back through time, ultimately unite the pair of copies.</p>
<p>Let&#8217;s use the letter <em>μ</em> for that per-generation, per-copy chance of mutation at the site. And we&#8217;ll note that, because each of the two branches that separate the paired copies of the site is, on average, 2_N_ or so generational copyings long, they tend to be separated by roughly 4_N_ generation-lengths overall.</p>
<p>With these simple observations, we can estimate the number of mutations that separate the two copies of the site as [drumroll&#8230;] 4_Nμ_.</p>
<p>Importantly, this number closely approximates the cornerstone measure of sequence variation in a population, <a href="http://en.wikipedia.org/wiki/Nucleotide_diversity">nucleotide diversity</a>, which we encountered <a href="/concepts-genetics-jargon/polymorphism-2/polymorphism-iii/">here</a>.[4]</p>
<p>And this result &#8212; that nucleotide diversity should be roughly 4_Nμ_ in our simple model population &#8212; is the basic nugget of insight that ties the historical size of a population directly to the pattern of genetic variation that it carries. That is, if we know two of the variables in question &#8212; nucleotide diversity and <em>μ</em> &#8212; we can in principle estimate the third variable, the size <em>N</em> of the population.</p>
<p>But recall that our simple model presumes the population to be steady-sized, with no natural selection, and so forth &#8212; conditions that don&#8217;t strictly hold in the real world. As such, like physicists modeling a flying cow, we&#8217;re actually using this simple 4_Nμ_ insight to estimate the population&#8217;s <em>effective</em> size, <em>Nₑ</em>.</p>
<h2>The few, the proud, the effective.</h2>
<p>So what, in the end, are the values of the two measurable variables, nucleotide diversity and <em>μ</em>, that suggest how many human genomes are &#8212; effectively &#8212; out there?</p>
<p>Depending on whom you look at, human nucleotide diversity ranges from roughly 0.0007 (for people who descend mainly from the fairly few ancient inhabitants of the Americas) to 0.001 (for people with recent ancestry in Africa, where we&#8217;ve been densely settled the longest, letting the most genetic diversity, <em>per capita</em>, arise and persist). That is, copies of human chromosomes tend to differ from each other at roughly seven to ten letters out of every ten thousand.</p>
<p>And <a href="http://johnhawks.net/weblog/reviews/genomics/variation/human-mutation-rate-review-2010.html">many</a> <a href="http://www.nature.com/ng/journal/v43/n7/full/ng.862.html">studies</a> of human and other DNA suggest that <em>μ</em> is roughly 1 to 3.5 x 10<sup>8</sup> per generation. That is, the genome of a viable human egg or sperm typically acquires a new spelling at roughly one letter per thirty to a hundred million.[5]</p>
<p>Putting these numbers together yields an estimate of human <em>Nₑ</em> at roughly ten (versus one or a hundred) thousand. Note that this is a back-of-the-envelope, order-of-magnitude kind of guess, and that it varies by what regional slice of humanity we look at (reflecting, in the end, real spatial variation in the longterm density of human settlement). More importantly, it clearly underestimates our real population size of roughly seven billion.[6]</p>
<p>And that difference is telling. Effective population size for any organism is typically less than the census size (and need not ever equal it), of course &#8212; that&#8217;s why it&#8217;s called &#8216;effective&#8217;, rather than &#8216;real&#8217;. But the fact that human <em>Nₑ</em> <em>per se</em> looks <em>so</em> low has long been a clue that our real population itself was long much smaller than it is now.[7] The newly documented glut of rare variants in our genomes strongly undergirds this idea, suggesting that very recently (in evolutionary terms) our population has grown much, much bigger.</p>
<p>As such, the overall data on human genetic variation clearly break the assumption of constant population size on which we simplistically estimated <em>Nₑ</em> in our simple model. But that estimate is stubbornly low for other reasons too.</p>
<p><img src="/wp-content/uploads/2012/07/Cypress-1.jpg" alt="" title="cypress" /></p>
<p><strong>Post-diluvian tree.</strong> Bald cypress with flood line, Carbondale, IL. (image copyright Nathaniel Pearson)</p>
<h2>Caution: Populations are bigger than they appear</h2>
<p>First, the trees generated by our model are, technically, most accurate for samples much smaller than the sampled population. As we look at more and more individuals, the model will tend to slightly <a href="http://www.sciencedirect.com/science/article/pii/S0040580905001498">underestimate</a> the lengths of the tree&#8217;s outermost branches &#8212; an effect that may contribute a (tiny) smidgen to the dilation of branches inferred in recent papers on the many rare variants found in samples comprising many thousands of people.</p>
<p>Second, and more importantly, most of the other hidden ways in which real populations can buck our über-simple model (besides change in size) tend to make them look even smaller. For example, a population comprising partly isolated sub-populations of various sizes, or one that undergoes the most common <a href="http://en.wikipedia.org/wiki/Directional_selection">kind</a> of natural selection, tends to look, in genetic terms, even smaller than we&#8217;d otherwise expect.[8]</p>
<p>Third, it turns out that, even if we realistically refine our simple model to let population size vary, the best longterm estimate of <em>Nₑ</em> roughly tracks the <a href="http://wikipedia.com/harmonic_mean">harmonic mean</a> of the population size over time &#8212; which in turn hews closely to the smallest value that the population passes through.</p>
<p>Thus, while our genomes do bear telltale signs of recent population growth, they also inevitably retain strong imprints of earlier eras when our ancestors were far fewer. Genetic diversity tends to be very sensitive to the demographic squeeze of a so-called population bottleneck &#8212; and it&#8217;s thought that our ancestors likely underwent one or, pending where they lived, even several severe ones.</p>
<h2>Grafting trees</h2>
<p>To better detect such a mix of population-size signals, geneticists can readily extend the basic coalescent model by dividing a genetic family tree into two or more parts, and modeling each in relative isolation.</p>
<p>They might, for example, single out one set of branches from the rest, to reflect the geographic isolation of some lineages from others (perhaps compounded by regional differences in sub-population size, natural selection, and so forth).</p>
<p>Or, more to our point, they could simply mark a height across the <em>whole</em> tree &#8212; like a flood mark on a mangrove or cypress &#8212; to specify a moment in the past when the size (or other evolutionary attributes) of the whole ancestral population changed.</p>
<p>In any such case, we get a tree that looks like multiple trees grafted together, as if by <a href="http://www.gutenberg.org/files/84/84-h/84-h.htm">Dr. Frankenstein</a>, to more realistically resemble the <em>true</em> genealogy of real copies of the genomic site in question.</p>
<p>If, for example, a population started growing at a marked time point, the resulting tree might have fairly stunted inner branches root-ward of that point, and longer outer branches leaf-ward of it. Conversely, if the population started shrinking at the chosen moment, the line drawn at the point in question would divide relatively stunted outer branches from longer inner ones.</p>
<p>Like master gardeners sculpting <a href="http://wikipedia.org/bonsai">bonsai</a> to the form of a wild tree, geneticists can tune their demographic assumptions &#8212; how big a population was originally, when and how fast it grew or shrank, <em>etc.</em> &#8212; to best fit their model trees to the real ancestral tree of copies of a genomic site.[9] As we&#8217;ve seen, the shape of that underlying genealogy is roughly discernible by measuring mutational differences among those copies, to proxy the lengths of the various ancestral branches that unite them.</p>
<p>Promisingly, the new papers on human rare variant burden use such tuning to re-estimate our effective population size to be on the order of <em>millions</em>, rather than mere thousands. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22604720">Tennessen <em>et al.</em></a>, for example, estimate the modern effective population sizes of Africans and Europeans to both be roughly half a million each; and <a href="http://www.ncbi.nlm.nih.gov/pubmed/22582263">Keinan and Clark</a> estimate that European <em>Nₑ</em> is now roughly 1.1 million. Note that, in such estimates, exact numbers matter less than orders of magnitude. Models with so many parameters are easy to overfit &#8212; and human population history is obviously complex, defying any attempt to distill real demography to a simple trend in effective population size &#8212; so we shouldn&#8217;t demand especially precise inferences.</p>
<h2>The wheel&#8217;s still in spin</h2>
<p>Most importantly, the new papers make clear that the human population has likely grown even more quickly than exponentially in the past few millenia. Recalling our bovine ballistics analogy, we could liken such growth to a flying cow sprouting strikingly long, skinny legs.</p>
<p>Those legs weigh (and impede the wind) less than the bulk of its body, leaving the cow&#8217;s effective mass or radius somewhat smaller than we might expect, given the gangly havoc it would wreak in a china shop. But just as a calf&#8217;s growth spurt eventually leaves it much heavier, the recent population explosion that sowed our bumper crop of rare genetic variants may, in the long run, boost our effective population size tremendously.</p>
<p>That is, if we stably inhabit earth (or beyond) at or above our current density for a long time, many of our currently rare variants will, by chance or otherwise, become modestly common enough to distinguish many chromosomal copies from each other, pushing our nucleotide diversity well above today&#8217;s humble figures.[10] Our effective population size, as measured by overall diversity in our genomes, should then continue to shift toward a new, if elusive, equilibrium that more closely tracks our teeming numbers.</p>
<p>Along the way, our changing population size may shape public health in complex ways. In particular, a key question will be what happens to the likely sizable subset of newly arisen rare variants that pose health risks to people who carry them. As our population continues to skyrocket, more such variants will come into our midst.</p>
<p>At the same time, continued population growth should ultimately help natural selection purge such variants more efficiently than it can in a small population (where chance dominates the fate of variants, harmful or not).</p>
<p>But, to the extent that our future population growth itself depends on further advances in healthcare, we&#8217;ll also be altering the regime of such natural selection, ideally relaxing it in ways that help people live healthier lives no matter what variants they carry in their genomes.</p>
<p>If we can manage that, of course, we&#8217;ll have become an effective population in a deeper, more important sense yet.</p>
<p>[^1] Except under special conditions, <em>Nₑ</em> will be met or, more commonly, exceeded by the size of the real population.<br />
[^2] On the breed-at-most-once criterion, note that we&#8217;ll let them have more than one child per breeding episode &#8212; an important point, as we&#8217;ll see. Note too that we ignore some subtleties of mating itself, focusing more on ancestral chains of copyings of a genomic site than on the kinship of the individuals who carry those copies. The latter turns out to matter fairly little overall.<br />
[^3] Note that a given copy in one generation can end up as a dead end even if it <em>is</em> copied down into the next generation, if its lineage doesn&#8217;t make it into the <em>last</em> (<em>i.e.</em>, current) generation. Thus the heartwarming truism that you, and everyone you know, descend only from winners in the evolutionary game. In coalescent models, dead-end branches that don&#8217;t reach the current generation are typically ignored, simplifying the tree to just the skeleton that ties together all current copies of the site (which you can picture as the tree&#8217;s leaves).<br />
[^4] This holds if mutation is consistently very rare &#8212; and it generally is &#8212; so that the branches connecting the two copies of a site hardly ever harbor more than one mutation overall.<br />
[^5] A sperm tends to carry more new mutations than an egg does, however, in part because the average sperm derives from many more mutation-prone cell divisions in the body than the average egg does.<br />
[^6] Coincidentally, there are now roughly as many living people as letters in a human genome, if we include the <a href="http://en.wikipedia.org/wiki/Repeated_sequence_(DNA)">repetitive</a> and <a href="http://en.wikipedia.org/wiki/Heterochromatin">tightly bound up</a> genome segments that are especially hard to read.<br />
[^7] Natural selection, non-random mating, and other factors inevitably also play roles in that deviation, in ways that we&#8217;re still figuring out. Much research today aims to understand human evolution in fine strokes, focusing on particular parts of the genome (which may differ in their histories, reflecting their roles in disease, adaptation to particular environments, and so forth) and particular populations (which have split and merged in complex patterns that we can, by carefully surveying shared versus distinctive patterns of genetic variation, start to deduce in some detail).<br />
[^8] Notably, along these lines, the concept of effective population size itself conventionally presumes no natural selection at all. That is, the subscript <em>e</em> can effectively be read as &#8216;if there were no selection&#8217;. When geneticists do explicitly parametrize natural selection in relevant models, they typically drop the &#8216;effective&#8217; from the population size term (while still acknowledging that the remaining term <em>N</em> is a rough estimate). Distinguishing the conventional, singularly definitive <em>Nₑ</em> from other abstractions of population size turns out to be tricky in other complex (read: realistic) evolutionary scenarios too, such as when a population is split into subpopulations, with sporadic migration among them.<br />
[^9] Or, more often, a multi-site segment, which allows more precise measurement of differences among copies (though risks complication from <a href="http://en.wikipedia.org/wiki/Genetic_recombination">recombination</a> history).<br />
[^10] Strikingly, coarse, nucleotide diversity-based estimates of our effective population size are lower not just than those of, say, fruitflies &#8212; but even those of most other great apes, suggesting that, despite our current global prominence, they&#8217;ve outnumbered us for much of our history.</p>
]]></content:encoded>
			<wfw:commentRss>/2012/06/24/the-myriad-throng-human-effective-population-size/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Polymorphism I</title>
		<link>/2009/05/01/polymorphism-i/</link>
		<comments>/2009/05/01/polymorphism-i/#respond</comments>
		<pubDate>Fri, 01 May 2009 06:00:22 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Evolution 101]]></category>

		<guid isPermaLink="false">/?p=44</guid>
		<description><![CDATA[&#8220;If you come to any more conclusions about polymorphism, I shd [sic] be very glad to hear the result: it is delightful to have many...]]></description>
				<content:encoded><![CDATA[<blockquote><p>
  &#8220;If you come to any more conclusions about polymorphism, I shd [sic] be very glad to hear the result: it is delightful to have many points fermenting in one&#8217;s brain&#8221;
</p></blockquote>
<p><a href="http://en.wikipedia.org/wiki/Charles_Darwin">Charles Darwin</a>, letter to <a href="http://en.wikipedia.org/wiki/Joseph_Dalton_Hooker">Joseph Hooker</a>, 1846</p>
<p>The term <em>polymorphism</em>, which peppers much of the biological and medical literature in this age of plentiful DNA sequence data, was well established among biologists by the time it first tickled Darwin&#8217;s brain. Coined from the Greek for &#8216;many shapes&#8217;, the word apparently gained particular currency among early Victorian botanists, who wanted a handy label for the <a href="http://www.hmnh.harvard.edu/on_exhibit/the_glass_flowers_collection.html">spectacular proliferation of structural forms</a> seen among newly collected plants[1] from far-off imperial lands. When Darwin, who was well versed in botany (especially that of <a href="http://darwin-online.org.uk/EditorialIntroductions/Freeman_FertilisationofOrchids.html">orchids</a>), set out to catalog and interpret a <em><a href="http://en.wikipedia.org/wiki/Second_voyage_of_HMS_Beagle">Beagle</a></em>-load of riotous diversity among animals and fungi, too, he naturally reached for <em>polymorphism</em> to summarize his emerging grasp of organismal variation. Many, many shapes indeed.</p>
<p>The concept of polymorphism held knotty mysteries for Darwin, though (as his <a href="http://www.darwinproject.ac.uk/">letter</a> to Hooker attests). In particular, his understanding of the inheritance of highly variable traits suffered, famously, without the insights of <a href="http://en.wikipedia.org/wiki/Gregor_mendel">Gregor Mendel</a>, the gardening monk whose obsessive bean-counting yielded the first clear mechanistic insights on the nature of <em>genetic</em> polymorphism. Darwin skimmed references to Mendel&#8217;s work, but missed its key lesson: that genetic variation is transmitted in discrete (&#8216;digital&#8217;) packets, rather than in infinitely divisible (&#8216;analog&#8217;) amounts_._ Lacking this insight, Darwin found it hard to account for the persistence of a wide variety of heritable organismal forms &#8212; polymorphism, that is &#8212; even in the absence of <a href="http://en.wikipedia.org/wiki/Natural_selection">natural selection</a>.</p>
<p>If, as Darwin assumed, genetic heritability were &#8216;analog&#8217;, rather than &#8216;digital&#8217;, then anything other than strongly assortative (&#8216;like-with-like&#8217;) mating would homogenize a population, just as mixing two paints gives a uniform intermediate hue. And, as a puzzled Darwin realized, not only is non-assortative mating rampant in nature, but strictly assortative mating would tend to prevent any heritable innovation &#8212; that is, the stuff that puts the <em>poly</em> in polymorphism &#8212; from getting a foothold in a population in the first place. Moreover, one cornerstone of Darwin&#8217;s own theory of evolution by natural selection is the idea that such heritable innovations can be functionally useful (<em>i.e.,</em> <a href="http://en.wikipedia.org/wiki/Adaptation">adaptive</a>). Yet, lacking Mendel&#8217;s insight, Darwin couldn&#8217;t explain how even an initially highly useful heritable innovation could spread through a stable-sized population at anything faster than a glacial pace, given that its distinctive functionality would be quickly diluted by each generation of mating between carriers and non-carriers.</p>
<p>So how did Mendel make his revelatory deduction that genetic transmission is discretized? He started with the simple observation that each pea plant in his garden was either violet-flowered or white-flowered, and yielded either very smooth peas or very wrinkly peas, regardless of its flower color. And he went on to find that, when he crossed two plants (by fertilizing one&#8217;s flowers with pollen from another) that either matched, or differed, in flower color and/or pea shape, the offspring did not include lavender-flowered plants that made slightly wrinkly peas. Rather, the plants continued to be either violet- or white-flowered, and to produce either very wrinkly or very smooth peas.</p>
<p>In Mendel&#8217;s careful (and, by his own admission, <a href="http://www.americanscientist.org/bookshelf/pub/csi-mendel">selective</a>) analysis, the actual numbers of each type of plant suggested that each trait of interest (<em>e.g.,</em> flower color, or pea shape) was governed largely by its own discrete, independently transmitted genetic factor, inherited in two copies (one from each parent). For each trait, one type of copy of the underlying factor appeared to be &#8216;<a href="http://en.wikipedia.org/wiki/Dominance_(genetics)">dominant</a>&#8216; to the other &#8212; that is, a plant&#8217;s flower color or pea shape would match the dominant type as long as at it had inherited at least one dominant-type copy of the relevant genetic factor. Such factors came to be called <em>genes</em> (though that word has more specific connotations today), and their variant copies came to be called <em>alleles</em>. Strikingly, the discretized nature of these packets of inheritance turned out to extend even to their own composition, as strings of discrete chemical &#8216;<a href="http://en.wikipedia.org/wiki/Nucleotide">letters</a>&#8216; that, if swapped for one another, differ in kind but not degree.</p>
<p>Mendel was lucky to have a relatively tractable organism around to study: pea plants, like humans, are <a href="http://en.wikipedia.org/wiki/Diploid#Diploid">diploid</a>, meaning that they have only two copies of the genome in most cells; many other crop plants have more complicated inheritance involving many <a href="http://strawberrygenes.unh.edu/history.html">more copies of a genome</a>. And, just as importantly, he focused on traits that had fairly simple genetic underpinnings. Many traits in pea plants, and in other organisms, are governed fairly strongly by variation in <em>many</em> parts of their genomes; such variation would not have been statistically tractable to Mendel, and indeed remains a computationally intensive <a href="http://www.ncbi.nlm.nih.gov/pubmed/19293820">focus of much research</a> (especially research on the genetic underpinnings of diseases) today.</p>
<p>Notably, Mendel&#8217;s work on pea plants (which actually addressed at least seven distinct traits of plant color or/and shape) involved variation that comprised, in the case of each trait, only two forms. Though such &#8216;binary&#8217; factors can indeed generate &#8216;many shapes&#8217; in their combination, Mendel actually discovered genetic <em>dimorphisms.</em> Strikingly, it turns out that, in real world populations of all sorts of organisms, most genetic variation also takes the form of &#8216;dimorphism&#8217;, when viewed at the finest scale saliently relevant to gene function. That is, rarely does a single site in the genome show more than two distinct spelling variants among samples taken from a real population &#8212; and in no case does it show more than five such variants (A, C, G, T or absence). One could therefore argue that a more appropriate umbrella term for a given example of localized genetic variation, <em>per se</em>, might be <em>multimorphism</em> (&#8216;more than one shape&#8217;) or <em>oligomorphism</em> (&#8216;a few shapes&#8217;), rather than polymorphism. The term is well established, however, and has likely transcended etymological connotations of &#8216;many&#8217;. A more substantive usage peeve arises, in both the scientific literature and popular coverage of it, when &#8216;polymorphism&#8217; is used, wrongly, to mean &#8216;variant&#8217; or &#8216;allele&#8217; (typically, an allele assumed to be derived by mutation, rather than an ancestral allele), as in this example from a <a href="http://uscnews2.usc.edu/hscweekly/detail.php?recordnum=9373">press release</a> by the University of Southern California&#8217;s news service:</p>
<blockquote><p>
  &#8220;Because of the way genes are inherited from both parents, each participant could either have two copies of the polymorphism, one copy or no copies.&#8221;
</p></blockquote>
<p>Rather than pester USC&#8217;s editor, though, let&#8217;s raise a bicentennial toast to mister Darwin, and a hearty forthcoming 187th to brother Mendel. Oh, and those conclusions about polymorphism? As noted, the term was first used as a catch-all label for organismal variation visible to the naked eye (or with a rudimentary microscope), but, with increasing knowledge of DNA sequence variation, has come to often denote specific examples of such variation, as in efforts to catalog all the <em>single-nucleotide polymorphisms</em> (<a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">SNPs</a>) in a given genome. Moreover, as comparative genome sequence data have become more thorough and plentiful &#8212; and as approaches to quantitatively assessing such data have emerged &#8212; the term has also become a synonym for <em>the overall level of genome sequence variation within a population</em>. In the latter usage, it stands in contradistinction to <em>divergence</em>, the term of choice for summary measures of the sequence variation that distinguishes one population (ostensibly, &#8216;species&#8217;, though that term is impossible to rigorously define in a way that universally accords intuitive judgments) from another. In future posts, we will explore in greater detail how such measures of variation are calculated. Overall, the origin and maintenance of genetic variation remain, arguably, the chief foci of empirical and theoretical inquiry in evolutionary genetics. As such, you might say that it remains delightful, indeed, to have many points &#8212; and many shapes &#8212; fermenting in one&#8217;s brain.</p>
<p>[^1] Plants are, of course, plentiful; slow-moving; compact, light, and durable when dry; and often highly useful. These qualities made them especially popular subjects for Victorian-era collection and sketching, and for imaging with early (and <a href="http://www.flickr.com/search/?q=flower">modern</a>) photographic materials.</p>
]]></content:encoded>
			<wfw:commentRss>/2009/05/01/polymorphism-i/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>On polymorphism II: Hard-drinking fruitflies</title>
		<link>/2009/04/20/on-polymorphism-ii-hard-drinking-fruitflies/</link>
		<comments>/2009/04/20/on-polymorphism-ii-hard-drinking-fruitflies/#respond</comments>
		<pubDate>Mon, 20 Apr 2009 21:54:23 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Evolution 101]]></category>

		<guid isPermaLink="false">/?p=48</guid>
		<description><![CDATA[The last post hinted at how useful it is to compare the amount of genetic variation found in one population to that found in another....]]></description>
				<content:encoded><![CDATA[<p>The last post hinted at how useful it is to compare the amount of genetic variation found in one population to that found in another. This question arises so often in population genetics, in fact, that the word <em>polymorphism</em> has, as noted, come to often denote a summary measure of DNA sequence variation within a population. But how, exactly, can one put a number on polymorphism, as such? Remarkably, this question was hardly answerable, from a practical perspective, until 1983, when <a href="http://pondside.uchicago.edu/ecol-evol/people/kreitman.html">Marty Kreitman</a> published <a href="http://www.ncbi.nlm.nih.gov/pubmed/6410283">the first definitive data</a> on DNA sequence variation in a wild population of organisms.</p>
<p>Before Kreitman&#8217;s paper, known examples of genetic variation documented at the level of DNA sequence were confined to &#8216;lucky breaks&#8217;, where such variation caused either 1) a sufficiently drastic change in some protein&#8217;s amino acid sequence (thanks to mutation of a codon) to be <a href="http://en.wikipedia.org/wiki/Protein_electrophoresis">detected</a> and <a href="http://en.wikipedia.org/wiki/Edman_degradation">characterized</a> by existing techniques, or 2) a change in the <a href="http://en.wikipedia.org/wiki/RFLP">&#8216;restriction digestion&#8217; signature</a> of that genome itself, as detected by chewing up the genome with <a href="http://en.wikipedia.org/wiki/Restriction_enzyme">bacterial enzymes</a> known to break it apart only at particular short sequence motifs (which can appear and disappear, via mutation).</p>
<p>In many organisms, however, codons make up only a tiny fraction of the genome &#8212; and &#8216;restriction enzyme&#8217; target sites are rarer still. Therefore, if we assume that each site in a genome is roughly as likely to show variation as is any other site, then only by <a href="http://en.wikipedia.org/wiki/DNA_sequencing">obtaining full sequences</a> for long segments of a genome can we hope to discover more than a tiny fraction of the variation that it harbors. The publication of &#8216;population sequence data&#8217; from fruitflies thus signaled a sea change in the study of genetic variation, transforming it from a field reliant on fragmentary and anecdotal evidence, to one that would be built around thorough and systematic survey. Clever as they were, researchers before 1983 had to take shots in the dark in order to bag polymorphisms; Kreitman&#8217;s data marked a flip of the lightswitch. And those data, in their own right, showed resoundingly how useful a quantitative estimate of genetic diversity could be. As such, the data themselves are worth briefly reviewing, before moving on to how genetic diversity is calculated.</p>
<p>Using cutting-edge technology of the early 1980s, Kreitman managed to sequence a 2721-base segment of the fruitfly (<a href="http://en.wikipedia.org/wiki/Drosophila_melanogaster"><em>Drosophila melanogaster</em></a>) genome, in eleven individual flies, each of which hailed from one of five geographically disparate populations. By modern standards, his hard-won data were meager, representing less than one of every sixty thousand bases in the fruitfly genome. By comparison &#8212; and as a striking example of the biotech equivalent of <a href="http://en.wikipedia.org/wiki/Moore">Moore&#8217;s law</a> &#8212; one <a href="http://www.nature.com/nmeth/journal/v5/n1/full/nmeth1156.html">high-throughput DNA sequencer</a> today can sequence the whole <a href="http://en.wikipedia.org/wiki/Euchromatin">euchromatic</a> portions of more than sixty fruitfly genomes in less than a day.</p>
<p>Lacking such a technological embarrassment of riches, Kreitman had to carefully pick the small segment of DNA that he focused on. He chose a gene &#8212; one of 14000 or so in the fruitfly genome &#8212; called <em>Adh,</em> which encodes a functionally intriguing protein called <a href="http://en.wikipedia.org/wiki/Alcohol_dehydrogenase">alcohol dehydrogenase</a>, or Adh (note the lack of italics, which are used for names of genes but not those of the proteins they encode). Adh is an enzyme that helps detoxify alcohol &#8212; an important biochemical task for animals, such as fruitflies and humans, who rely on fleshy fruits for nutrition (because such fruits are also a favorite food for <a href="http://en.wikipedia.org/wiki/Saccharomyces_cerevisiae">yeast</a>, who have mastered the trick of turning fruit sugars into toxic alcohol in order to drive off rivals and predators).</p>
<p>Though humble in scale, Kreitman&#8217;s data bore great fruit. First, they revealed the actual DNA sequence difference underlying a well known, functionally important <a href="http://www.ncbi.nlm.nih.gov/pubmed/115502">dimorphism</a> in the amino acid sequence of fruitfly Adh. Of broader interest, however, were the revelations that 1) the <em>Adh</em> gene showed extensive DNA sequence variation (43 sites out of the 2721 showed variation, even in such a small sample of flies), 2) most of that variation involved single-nucleotide substitutions (rather than tandem nucleotide substitutions, or insertions or deletions of nucleotides), and 3) only one dimorphism (afforementioned) affected the amino acid sequence of the protein. The last observation was particularly important, as many (14) of the observed dimorphisms were actually harbored by the gene&#8217;s codons. Thirteen of the fourteen, however, were &#8216;synonymous&#8217; dimorphisms &#8212; that is, they changed a codon to one of its synonyms (if you were reading earlier posts carefully, you&#8217;ll recall that the <a href="/concepts-genetics-jargon/second-life/">genetic code</a> is <em>degenerate</em>).</p>
<p>Each of these points would become a foundational piece of data in the genomics era, serving as a benchmark by which future sequence data could be compared and debated, particularly in the context of the so-called <em><a href="http://en.wikipedia.org/wiki/Neutral_theory_of_molecular_evolution">neutral theory of molecular evolution</a></em>, as expounded by influential theoretician <a href="http://en.wikipedia.org/wiki/Motoo_Kimura">Motoo Kimura</a>. Saving discussion of such debate for another day, let&#8217;s move on to explore how one can squeeze a sensible and widely useful summary measure of &#8216;polymorphism&#8217; out of DNA sequence data from a population.</p>
]]></content:encoded>
			<wfw:commentRss>/2009/04/20/on-polymorphism-ii-hard-drinking-fruitflies/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Second Life</title>
		<link>/2009/03/15/second-life/</link>
		<comments>/2009/03/15/second-life/#respond</comments>
		<pubDate>Sun, 15 Mar 2009 16:53:45 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Evolution 101]]></category>

		<guid isPermaLink="false">/?p=57</guid>
		<description><![CDATA[New Scientist has some intriguing recent stories on particular fronts of research, including a discussion of the search for so-called &#8216;shadow life&#8216; &#8212; meaning, essentially,...]]></description>
				<content:encoded><![CDATA[<p><a href="http://www.newscientist.com/">New Scientist</a> has some intriguing recent stories on particular fronts of research, including a discussion of the search for so-called &#8216;<a href="http://www.newscientist.com/article/mg20126990.100-second-genesis-its-life-but-not-as-we-know-it.html">shadow life</a>&#8216; &#8212; meaning, essentially, life forms here on earth that don&#8217;t share the last common ancestor of currently known life forms. <a href="http://exobiology.nasa.gov/">Exobiologists</a>, who hope to study life beyond earth, tend to focus on nearby candidate habitats (such as Mars and some volcanic moons elsewhere) that are thought to be energetically and chemically amenable to life as we know it. But it&#8217;s intriguing to consider the possibility that critters with ancestry outside the currently understood ancestral <a href="http://www.tolweb.org/tree/">tree of life</a> are living under (or perhaps even in&#8230;) our noses. In mulling the potential importance of understanding such life, and how it might differ from our own, the New Scientist article quotes <a href="http://spot.colorado.edu/~cleland/">Carol Cleland</a> with an instructive analogy:</p>
<blockquote><p>
  &#8220;If you were an alien biologist who&#8217;s interested in understanding what a mammal was, and all you had was zebras, it&#8217;s very unlikely that you would focus on their mammary glands, because only half the specimens have them. You&#8217;d probably focus on the stripes, which are ubiquitous.&#8221;
</p></blockquote>
<p>This analogy, though pithy and thought-provoking, strikes me as incomplete. If alien biologists thought like human biologists, for example, they&#8217;d likely first focus on traits that <em>they shared</em> with zebras (such a bias toward expecting and recognizing the familiar is reflected, comically, in the vast array of subtle twists on human morphology that seem to characterize most sentient life forms &#8212; cosmos-wide &#8212; in the fictional world of Star Trek). But Cleland&#8217;s point &#8212; that life, perhaps like any phenomenon, is most effectively understood through systematic comparison of many examples &#8212; is well taken.</p>
<p>So how might a &#8216;shadow&#8217; life form differ from us other life forms? Well, for starters, it might not share the micron-scaled compartmental (cell- and other membrane sac-defined) architecture that we know so well. Or, for that matter, the familiar centralized (&#8216;genomic&#8217;) storage of replicative &#8216;instructions&#8217;. Or, more fundamentally, the carbon &#8216;backbone&#8217;-based chemistry that we recognize to be so versatile for building molecules that interact with each other in richly complex ways. Yet even if a newfound life form shared all of the foregoing traits with &#8216;non-shadow&#8217; life, a key way in which it might differ from us might be in not sharing one of the most salient &#8216;stripes&#8217; that characterize life as we know it &#8212; our <a href="http://en.wikipedia.org/wiki/Genetic_code">genetic code</a>.</p>
<p>The latter term is so frequently (and dismayingly) <a href="http://www.msnbc.msn.com/id/9136200/">misused</a> in popular science journalism, that a definition is in order here. By genetic code, biologists literally mean a <a href="http://en.wikipedia.org/wiki/Key_(cryptography)">key</a>: that is, a specific translation scheme for making <a href="http://en.wikipedia.org/wiki/Protein">proteins</a> (the molecules that catalyze or otherwise centrally direct most of the chemical interactions that we recognize as crucial to cellular function) from instructions written in the form of <a href="http://en.wikipedia.org/wiki/Nucleic_acid">nucleic acids</a> (which make up genomes). The &#8216;texts&#8217; to be translated into proteins, via this code, are found in the specialized segments of a genome that we call genes. Such texts are simply strings of 3-letter DNA &#8216;words&#8217; called <a href="http://en.wikipedia.org/wiki/Codon"><em>codons</em></a>, each of which is translated into a particular amino acid &#8216;meaning&#8217; as the protein (a polymer of <a href="http://en.wikipedia.org/wiki/List_of_standard_amino_acids">amino acids</a> strung together in the same order as the codons that specify them) is built.</p>
<p>Because the &#8216;alphabet&#8217; of DNA has exactly four &#8216;letters&#8217; (A,C,G,T), there are exactly sixty-four (i.e., 4 x 4 x 4) distinct codons. But there are, with minor exceptions, just twenty distinct amino acids used to make proteins in life as we know it. Other than these twenty amino acids, the only other &#8216;meaning&#8217; to be conveyed by codons is the &#8216;stop sign&#8217; used to signal that a newly minted protein is complete. The genetic code, having more distinct words (64) than distinct meanings (21), is thus said to be <em>degenerate</em> &#8212; that is, some of its words have the same meaning.</p>
<p>While such degeneracy can help buffer a code against &#8216;transmission error&#8217;, degenerate codes are inherently inefficient. One could, for example, easily devise a genetic code using just sixteen &#8216;2-letter&#8217; codons (one of them with a &#8216;shift key&#8217; meaning), rather than the sixty-four &#8216;3-letter&#8217; ones, to encode exactly the same &#8216;meaning&#8217; information as the real genetic code does, in roughly 30% less &#8216;text&#8217;. And when one considers other optimality criteria, too, the standard genetic code likewise falls short of ideal: <a href="http://www.springerlink.com/content/gc2uarjxfl6307vp/">Stephen Freeland and Laurence Hurst</a>, for example, measured codes&#8217; abilities to buffer the effects of genomic mutation &#8212; a major source of &#8216;transmission error&#8217; &#8212; at the protein level, and estimated that the standard code, while remarkably good, was worse than <em>trillions</em> of possible alternative codes (and they even limited the field of contenders to codes that had exactly the same synonymous codon &#8216;families&#8217; found in the standard code, with the same such family assigned to the &#8216;stop sign&#8217; meaning).</p>
<p>Yet when one surveys earthly life &#8212; from blue whales to bacteria (and including humans, of course) &#8212; one finds that nearly all known organisms use this standard code. Known exceptions (including, incidentally, the code used within mitochondria) are very rare, likewise suboptimal, and involve reassignment of no more than a handful of codons relative to the standard code. The striking near-ubiquity of the standard code, with its readily apparent functional shortcomings, likely won&#8217;t sit well with proponents of &#8216;intelligent design&#8217;, and continues to prompt debate among evolutionary geneticists themselves.</p>
<p>The leading explanation, championed by <a href="http://en.wikipedia.org/wiki/Francis_Crick">Francis Crick</a> (who, in addition to <a href="http://www.ncbi.nlm.nih.gov/pubmed/13054692">discovering</a> the structure of DNA, <a href="http://www.ncbi.nlm.nih.gov/pubmed/13882203">gleaned the first details</a> of the standard genetic code for us), posits that the code is simply a <a href="http://www.ncbi.nlm.nih.gov/pubmed/4887876">&#8216;frozen accident&#8217;</a>. In this view, the standard code, like the <a href="http://en.wikipedia.org/wiki/Qwerty_keyboard">QWERTY</a> keyboard, has been more lucky than good: out of oodles of possible variants, it worked well enough (potentially quite a bit better than early rivals), early in the evolution of a system of information flow, to become overwhelmingly common and well entrenched. As the system (earthly life, or earthly typing) evolved, important infrastructure (in the QWERTY analogy, not just keyboards themselves, but the programming details of word processing software, the synapses of typing instructors, etc.) became well adapted to the variant in question but poorly suited to any <a href="http://colemak.com/Compare">potential rival</a>, even if the latter were much more efficient.</p>
<p>The &#8216;weight&#8217; of such infrastructure now serves, in this view, to anchor the standard genetic code against intrinsic improvement. In evolutionary terms, consider the implications of even a tiny tweak of the genetic code in an organism that, like a human or a banana plant, relies on the code in making thousands of distinct proteins, each with finely tailored functionality. Swapping one amino acid for another, in thousands of proteins at once, would be overwhelmingly likely to wreak utter havoc with organismal function. Simply put, tinker with the code and the organism will, with near perfect certainty, fail to develop.</p>
<p>&#8216;Shadow life&#8217;, however, could readily have a genetic code that differs from the one we know so well. If such differences were slight, we might infer that the newfound critter in question was a fairly close relative, despite its novelty in our <em>systema naturae</em>. Such a discovery might push back our understanding of the origin of life on earth, setting it in a temporally deeper and/or spatially broader context. If the differences between the two codes (assuming the &#8216;shadow&#8217; life form even had a recognizable genetic code) were vast, on the other hand, we might infer that the two domains of life had distinct evolutionary origins.</p>
<p>All of which raises the question &#8212; crucial to consider in any quest to recognize &#8216;shadow life&#8217; &#8212; of how we are to define life in the first place. Historian <a href="http://www.columbia.edu/cu/history/fac-bios/Moyn/faculty.html">Samuel Moyn</a>, a keen thinker and a longtime friend, recently told me that, having informally polled some biologists on how they define life, he&#8217;d been surprised by the lack of consensus &#8212; or even individual clarity &#8212; in their answers. He wondered how a whole field could focus, with any lasting fruitfulness, on something that its researchers could not rigorously define. In noting the definitional slipperiness of life, Moyn is on to something; I&#8217;m not sanguine that any rigorous candidate definition of living matter will fully accord our &#8216;intuitive&#8217; answers to the question of whether something is alive. Textbook cases of ambiguity &#8212; such as &#8216;are viruses alive?&#8217; &#8212; often hinge on some concept of &#8216;autonomous replication&#8217;; but, of course, nothing that we think of as unambiguously living (such as ourselves) can truly replicate &#8216;autonomously&#8217;, i.e., independently of necessary cofactors that we think of as external to the organism.</p>
<p>That said, I&#8217;m not sure that Moyn&#8217;s point applies only to biologists; mathematicians, for example, may ultimately have a hard time rigorously defining, say, <a href="http://mathworld.wolfram.com/Number.html">number</a> (at least in terms that withstand <a href="http://en.wikipedia.org/wiki/G%C3%B6del%27s_incompleteness_theorems">criticism</a> from within their own ranks). While admitting that we biologists sweep a huge question under the rug in failing to rigorously define life, I&#8217;m nonetheless excited to be living and working in an era during which I anticipate that our understanding of the <em>scope</em> of life (however defined) in the cosmos may expand significantly. Here&#8217;s hoping we find something surprising, and not too hungry.</p>
]]></content:encoded>
			<wfw:commentRss>/2009/03/15/second-life/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
