<!doctype html>
<html lang="en-US">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
	<!-- Mobile viewport optimized -->
	<meta name="viewport" content="width=device-width,initial-scale=1">
	<!-- Title -->
	<link rel="profile" href="http://gmpg.org/xfn/11" />
	<link rel="pingback" href="/xmlrpc.php" />
	<!-- WP Head -->
	<title>On polymorphism III: Working with peanuts &#8211; Genomena</title>
<meta name="generator" content="WordPress 4.5.6" />
<meta name="template" content="Oxygen 0.6.0" />
<link rel="alternate" type="application/rss+xml" title="Genomena &raquo; Feed" href="/feed" />
<link rel="alternate" type="application/rss+xml" title="Genomena &raquo; Comments Feed" href="/comments/feed" />
<link rel="alternate" type="application/rss+xml" title="Genomena &raquo; On polymorphism III: Working with peanuts Comments Feed" href="/2009/04/21/on-polymorphism-iii-working-with-peanuts/feed" />
<link rel='stylesheet' id='style-css'  href='/wp-content/themes/oxygen/style.min.css?ver=0.6.0' type='text/css' media='all' />
<link rel='stylesheet' id='font-abel-css'  href='http://fonts.googleapis.com/css?family=Abel&#038;ver=1' type='text/css' media='screen' />
<script type='text/javascript' src='/wp-includes/js/jquery/jquery.js?ver=1.12.4'></script>
<script type='text/javascript' src='/wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1'></script>
<link rel='https://api.w.org/' href='/wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="/xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="/wp-includes/wlwmanifest.xml" />
<link rel='prev' title='On polymorphism II: Hard-drinking fruitflies' href='/2009/04/20/on-polymorphism-ii-hard-drinking-fruitflies/' />
<link rel='next' title='Polymorphism I' href='/2009/05/01/polymorphism-i/' />
<link rel="canonical" href="/2009/04/21/on-polymorphism-iii-working-with-peanuts" />
<link rel="alternate" type="application/json+oembed" href="/wp-json/oembed/1.0/embed?url=http%3A%2F%2Fgenomena.com%2F2009%2F04%2F21%2Fon-polymorphism-iii-working-with-peanuts%2F" />
<link rel="alternate" type="text/xml+oembed" href="/wp-json/oembed/1.0/embed?url=http%3A%2F%2Fgenomena.com%2F2009%2F04%2F21%2Fon-polymorphism-iii-working-with-peanuts%2F&#038;format=xml" />
	<style type="text/css">
		/* Font size. */
		/* Font family. */
		/* Link color. */
				a:hover, a:focus { color: #000; }
		.read-more:hover, #respond #submit:hover, .button:hover, a.button:hover, #subscribe #subbutton:hover, .wpcf7-submit:hover, #loginform .button-primary:hover { background-color: #111; }
		/* Custom CSS. */
	</style>
</head>
<body class="wordpress ltr en_US parent-theme y2017 m02 d28 h12 tuesday logged-out singular singular-post singular-post-126 no-js">
	<div id="container">
		<div class="wrap">
			<div id="header">
					<div id="branding">
						<div id="site-title"><a href="/" title="Genomena" rel="home"><span>Genomena</span></a></div>
						<div id="site-description"><span>As genomes evolve, so does the world.</span></div>
					</div><!-- #branding -->
			</div><!-- #header -->
			<div id="main">
	<div class="aside">
	<div id="sidebar-primary" class="sidebar">
				<section id="recent-posts-2" class="widget widget_recent_entries widget-widget_recent_entries">		<h3 class="widget-title">Recent Posts</h3>		<ul>
					<li>
				<a href="/2014/04/06/threesomics">Threesomics</a>
						</li>
					<li>
				<a href="/2013/08/26/three-small-steps-toward-genomically-sensible-healthcare">Three small steps toward genomically sensible healthcare</a>
						</li>
					<li>
				<a href="/2013/05/22/harmful-by-any-other-name-on-clinical-variant-classification">Harmful by any other name: On clinical variant classification</a>
						</li>
					<li>
				<a href="/2013/04/01/the-shock-of-the-new-what-we-can-say-about-helas-novel-variants">The Shock of the New: What we can say about HeLa&#8217;s novel variants</a>
						</li>
					<li>
				<a href="/2013/03/22/harmful-by-any-other-name-on-clinical-variant-classification-2">Harmful by any other name: On clinical variant classification</a>
						</li>
				</ul>
		</section>
</section><section id="categories-2" class="widget widget_categories widget-widget_categories"><h3 class="widget-title">Categories</h3>		<ul>
	<li class="cat-item cat-item-9"><a href="/category/evolution-101" title="What the hell are geneticists talking about?  An attempt to put key terms in plain English.">Evolution 101</a>
</li>
	<li class="cat-item cat-item-7"><a href="/category/genetics-evolution-and-policy" title="A compilation of articles (ok, screeds) on important societal issues in which technologies and insights from genetics play key roles.">Genetics, evolution, and policy</a>
</li>
	<li class="cat-item cat-item-3"><a href="/category/genomes-and-health" >Genomes and health</a>
</li>
	<li class="cat-item cat-item-2"><a href="/category/human-population-history" >Human population history</a>
</li>
	<li class="cat-item cat-item-8"><a href="/category/humania-research-by-people-about-people-for-people" title="It’s true — people get way too much press, given our small numbers.  But deal with it, ok?  After all, if cuttlefish ran the web, we all know who’d get the hype.">Humania: we study…us</a>
</li>
	<li class="cat-item cat-item-10"><a href="/category/technology-how-we-look-at-dna" title="While often confused in the popular press, science and technology are distinct concepts. The former, loosely speaking, is our systematic quest to understand the world; the latter, building tools to gain that understanding (and for other aims).  Geneticists rely on fancy gear and software to learn about genomes and the organisms that carry them.  The technology used to sequence DNA — that is, to precisely deduce the series of As, Cs, Gs, and Ts in a given genome — is changing particularly fast these days.">Seq: Reading DNA</a>
</li>
	<li class="cat-item cat-item-1"><a href="/category/uncategorized" >Uncategorized</a>
</li>
		</ul>
</section><section id="nav_menu-8" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Video</h3><div class="menu-video-container"><ul id="menu-video" class="menu"><li id="menu-item-113" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-113"><a href="http://www.tedmed.com/videos-info?name=Ozzy_and_Sharon_Osbourne_with_Nathaniel_Pearson_at_TEDMED_2010&#038;q=updated&#038;year=all&#038;sid=28&#038;vid=2">TedMed 2010 | Your moderator interprets Ozzy Osbourne&#8217;s genome. Really.</a></li>
<li id="menu-item-114" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-114"><a href="http://www.pbs.org/wgbh/nova/body/cracking-your-genetic-code.html">NOVA: Cracking Your Genetic Code | Helping explain the prospect of genomically personalized healthcare</a></li>
<li id="menu-item-115" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-115"><a href="http://www.pbs.org/wnet/facesofamerica/video/episode-4-know-thyself/237/">Faces of America: Know Thyself | Interpreting the genomes of historian Henry Louis Gates, Jr. and dad</a></li>
</ul></div></section>
	</div><!-- #sidebar-primary .aside -->
	</div>
	<div class="content-wrap">
		<div id="content">
		<div class="breadcrumb-trail breadcrumbs" itemprop="breadcrumb">
			<span class="trail-browse">Browse:</span> <span class="trail-begin"><a href="/" title="Genomena" rel="home">Home</a></span>
			 <span class="sep">&nbsp; / &nbsp;</span> <a href="/2009" title="2009">2009</a>
			 <span class="sep">&nbsp; / &nbsp;</span> <a href="/2009/04" title="April 2009">April</a>
			 <span class="sep">&nbsp; / &nbsp;</span> <a href="/2009/04/21" title="April 21, 2009">21</a>
			 <span class="sep">&nbsp; / &nbsp;</span> <span class="trail-end">On polymorphism III: Working with peanuts</span>
		</div>
			<div class="hfeed">
						<div id="post-126" class="hentry post publish post-1 odd author-npearson category-evolution-101">
							<div class="post-content">
								<h2 class="post-title entry-title"><a href="/2009/04/21/on-polymorphism-iii-working-with-peanuts" title="On polymorphism III: Working with peanuts" rel="bookmark">On polymorphism III: Working with peanuts</a></h2>
								<div class="byline"><a href="/author/npearson" title="Posts by Nathaniel Pearson" rel="author">Nathaniel Pearson</a> / <a href="/2009/04/21/on-polymorphism-iii-working-with-peanuts">2009-04-21</a> / <a href="/category/evolution-101" rel="category tag">Evolution 101</a></div>
								<div class="entry-content">
									<p><img src="/wp-content/uploads/2009/04/peanuts21.jpg" alt="Delicious Alleles" title="peanuts21" /></p>
<p>We&#8217;re now ready to ask &#8212; and answer &#8212; the question: how much polymorphism is to be found in a given population? Remember that &#8216;polymorphism&#8217; here specifically means genetic diversity, which is ultimately defined by characters (think &#8216;A, C, G, T&#8217;) that differ in <em>kind</em>, not <em>degree</em>.</p>
<p>In this important sense, DNA sequence variation differs from variation in, say, mass, or number-of-toes. The latter traits are inherently <a href="http://en.wikipedia.org/wiki/Quantity" title="quantitative"><em>quantitative</em></a>, whereas genotype is, at the finest scale, inherently <a href="http://en.wikipedia.org/wiki/Quality_(philosophy)" title="qualitative"><em>qualitative</em></a>. As such, conventional measures of quantitative variation &#8212; particularly <a href="http://en.wikipedia.org/wiki/Central_moment" title="univariate moments">univariate moments</a>, such as <a href="http://en.wikipedia.org/wiki/Variance" title="variance">variance</a> &#8212; don&#8217;t apply, at least directly, as measures of sequence variation.</p>
<p>Take heart, though. Even if we have to bypass (for now) old friends like variance, we will indeed be able to arrive at a summary measure (yes, a quantitative one!) of genetic diversity that is well defined and widely applicable. The last point is important: we want to be able to use the same yardstick of diversity to study populations of any given organism. In our example, we&#8217;ll go with the organism most familiar to us: people.</p>
<p>To keep things simple, though, we will agree to reduce a population of living, breathing people &#8212; each of whom comprises trillions of cells, each of which carries a genome billions of nucleotides long &#8212; to a population of <em>alleles</em>. Specifically, we&#8217;ll imagine picking just one cell from each person, then zooming in and focusing on just one site in the genome (keeping in mind that the cell likely contains two copies of that site &#8212; but more on that later). In simplifying things this way, we mentally dissolve away the non-genomic parts of cells, and all the other potentially distracting trappings of our human population, until what remains is simply a bunch of representative copies of that single site in the genome. It is this population of <em>alleles</em> whose level of diversity we aim to estimate, at least for starters.</p>
<h2>Attention: Candy</h2>
<p>Admittedly, even dissolving away everything but a single site in the genome may not simplify things enough to proceed smoothly. After all, mentally manipulating a population of invisibly tiny copies of a genome site is not the most intuitive task. So, to give the brain something concrete and familiar to work with, let&#8217;s invoke a mouthwatering analogy. Specifically, we&#8217;ll liken our population of copies of a genome site to a jarful of colorfully candy-coated <a href="http://en.wikipedia.org/wiki/Peanut" title="peanuts">peanuts</a>. Candied peanuts &#8212; rather than say, jelly beans or gumballs &#8212; are ideal for a subtle reason that I&#8217;ll explain later.</p>
<p>The jar we&#8217;re imagining holds many, many such candy peanuts (more, even than the jar pictured at the top of the post). To be precise, let&#8217;s say the jar holds exactly 1000 nuts. This number is the first of quite a few that will be popping up &#8212; meaning that we&#8217;ve reached a point where a conscientious science guide, like an avuncular airline pilot forecasting a &#8216;patch of rough air&#8217;, is supposed to ruefully warn you about the thorny math to come. Fear not, though: we need basic arithmetic, not <a href="http://en.wikipedia.org/wiki/Steven_hawking" title="Stephen Hawking">Stephen Hawking</a> stuff, to reason our way to a working concept of genetic diversity. So don&#8217;t fret.</p>
<p>Now, in principle, 1000 peanuts, each coated in a shade of candy paint, could sport as many as 1000 unique hues. But our peanuts are typical mass-produced sweets &#8212; so, for industrial efficiency, they come in just a few distinct colors. Four, actually: red, yellow, green, and blue. Picturing our jarful of a thousand such nuts, we might ask our key question: &#8216;<em>How diverse, by color, are the nuts in the jar?</em>&#8216;. Now, in answering that question, let&#8217;s agree to consider only the color of the candy coating (not the roasted interior) of each nut, and to forgo trying to estimate how different from each other two colors might be. That is, we&#8217;ll treat the colors of any two nuts as <em>either the same, or different</em>, with nothing in between. Moreover, we&#8217;ll assume that the candy paint on our peanuts never fades or otherwise changes in color.</p>
<h2>Counting colors: Diversity = 4?</h2>
<p>Right off the bat, it may be tempting to posit that the nuts in our jar, by virtue of coming in exactly four distinct colors, have a &#8216;color diversity&#8217; of 4&#46; And promisingly, this answer would capture something essential (if obvious) about the situation: 4 is both more than 1 (the value we would estimate for a clearly less color-diverse jar of nuts that were all coated the same color) and less than 1000 (the number we&#8217;d assign to a jar of undeniably more color-diverse nuts, in which each had its own unique hue).</p>
<p>But before we settle on simply counting colors to define &#8216;color diversity&#8217;, note that the range of possible values for this count would always depend on the total number of nuts in the jar. If a jar held just three nuts, for example, its &#8216;color diversity&#8217; couldn&#8217;t exceed 3&#46; How are we to compare such an estimate to the value we obtain for a jar of, say, five nuts in just three colors? Would these two jars be precisely <em>equal</em> in color diversity? My gut, and maybe yours too, says no, a five-nut, three-color jar &#8212; which carries more nuts than it &#8216;needs&#8217; to in order to qualify as <a href="http://en.wikipedia.org/wiki/Tricolour" title="tricolore"><em>tricolore</em></a> &#8212; seems less color-diverse than a three-nut, three-color jar.</p>
<h2>Ratio profiling: ~~Diversity = 4?~~ Diversity = 0.004?</h2>
<p>My gut also tells me that a broadly useful measure of diversity should depend as little as possible on other population-specific qualities/quantities (such as the number of individuals in the population). With this aim in mind, we might gravitate to the idea of using a <a href="http://mathworld.wolfram.com/Ratio.html" title="ratio"><em>ratio</em></a> to measure diversity. In particular, why not just divide the number of distinct nut colors by the number of nuts in the jar? Perhaps this approach would churn out &#8216;diversity&#8217; estimates that compare more sensibly, one to the next, even for jars that carry different numbers of nuts. A jar of three nuts in three colors, for example, would have a &#8216;nut colors per nut&#8217; ratio of 3/3, or 1&#46; The round unity of this estimate would suggest that we think of such a jar as &#8216;fully&#8217;, rather than &#8216;partly&#8217;, diverse; and this, in turn, would accord the fact that such a jar is as color-diverse as a three-nut jar can be. By comparison, a jar of five nuts in just three colors would have a lower ratio of 3/5, or 0.6 &#8212; a result that fits our intuition that such a jar isn&#8217;t &#8216;fully&#8217; diverse.</p>
<p>When applied to our original jar of 1000 nuts in four colors, such a ratio-based approach would yield a color diversity estimate of 4/1000, or 0.004&#46; Reassuringly, this value is still more than 1/1000 and less than 1000/1000, per our intuition. Moreover, because we can&#8217;t have more nut colors than nuts, the new ratio-based measure would, conveniently, always take on a value somewhere in the range of zero to one. As such, it would be what scientists call a <em>normalized measure</em>, with a tidy built-in scaling that carries some mathematically useful properties. So maybe we&#8217;re getting somewhere with this ratio idea.</p>
<p>However, by turning to such a simple ratio &#8212; the number of distinct colors in the sample divided by the number of nuts in the sample &#8212; we would iron a new wrinkle into our definition of diversity, again putting it at odds with intuition. To spot that wrinkle, let&#8217;s do something that scientists often do in road-testing an idea: examine a so-called &#8216;boundary case&#8217;, where some variable under consideration takes on its highest (often infinity) or lowest (often negative infinity, zero, or, in cases like ours, one) possible value. In our scenario, we&#8217;ll check the boundary case of a jar that holds just one peanut.</p>
<p>Regardless of how we define diversity, one nut sitting alone in a jar has, of course, a single characteristic color (lonesome <a href="http://en.wikipedia.org/wiki/Blues" title="blue">blue</a>, perhaps). The total number of colors &#8212; here the use of the plural is just a formality &#8212; in a one-nut sample (i.e., jar) must therefore equal the number of nuts in the sample. As such, a ratio-based estimate of such a jar&#8217;s color diversity would be 1/1 &#8212; or, well, 1&#46; If you think about it, this means that we would deem a one-nut, one-color jar to be just as color-diverse as a three-nut, three-color jar. Moreover, we would hold both of these jars to be exactly as diverse as a thousand-nut, thousand-color jar! True, each of these jars, however many nuts it contained, would be as color-diverse as it could be. Nonetheless, intuition tells us &#8216;Sorry, but a jar containing one nut &#8212; whatever its color &#8212; is simply not as color-diverse as a jar containing a thousand nuts in a thousand colors&#8217;.</p>
<h2>Match and mismatch</h2>
<p>Apparently, in trying to confine diversity to a scale that doesn&#8217;t vary with the size of the sample, we lost some ability to resolve how &#8216;surprising&#8217; or &#8216;unsurprising&#8217; a given finding of &#8216;diversity&#8217; is. So why, exactly, did the simple ratio-based definition, which had raised our hopes, fail this way? To see why, imagine doing something drastic: wantonly spilling the contents of the original jar onto a table. Having done so, and taking care not to let any roll over the edge, we&#8217;ll start tallying colors one nut at a time.</p>
<p>Now, in your mind&#8217;s eye, a thousand peanuts coated in four candy colors and strewn across a table may make up a wildly gaudy mosaic &#8212; a bit like a <a href="http://en.wikipedia.org/wiki/Pointillism" title="pointillist">pointillist</a> painting executed, in a stylistic stretch, by <a href="http://en.wikipedia.org/wiki/Jackson_pollock" title="Jackson Pollock">Jackson Pollock</a>. If this is what you&#8217;re picturing, though, I&#8217;m sorry to tell you that your imagination has been led astray (perhaps by eager tastebuds&#8230;or by the misleading photo above). In fact, when we spill out our jar onto the table, here&#8217;s what we find: exactly 994 red-coated peanuts, and 2 peanuts each in yellow, green, and blue.</p>
<p>This hardly qualifies as diversity &#8212; to the contrary, it&#8217;s classic <a href="http://en.wikipedia.org/wiki/Tokenism" title="tokenism">tokenism</a>. Yet the simple ratio-based measure that we defined would estimate the color diversity of these peanuts as 0.004 &#8212; the same value as would be estimated for a jar holding a more &#8216;balanced&#8217; mix with 250 peanuts in each of the four colors. Clearly, the simple ratio-based candidate definition of color diversity fails to capture a key intuition: that an <em>even mix</em> of types is more &#8216;diverse&#8217; than a mix skewed toward just one (think of that boundary case again) or a few types.</p>
<p>To see this another way, imagine that we can&#8217;t view our jar at all, but instead have to gauge the color diversity of its contents by reaching in blindly, fishing out a peanut for inspection, eating it (for a parallel survey of flavor-diversity, of course), and repeating this procedure for a while. Now, if our jar is a tokenist one, then, whenever we blindly plunge a hand in, we can safely bet that we&#8217;ll be pulling out a red peanut. By contrast, if we reach into a jar containing a balanced, cosmopolitan mix of peanuts in several colors, we won&#8217;t be able to guess &#8216;red&#8217; (or anything else) with much confidence.</p>
<p>How do we express this insight in formal terms? For starters, we care little about the identity of any particular color(s); that is, a nearly all-green jar is just as tokenist as a nearly all-red one. Moreover, color itself, the trait that we happen to be surveying, is in no way integral to a general definition of diversity. If not interested in color, for example, we could wonder just as easily about diversity of flavor, country of manufacture, or symbol stamped on the candy shell. The qualitative trait that we keep track of, and the specific values that the trait may take, aren&#8217;t the point – what&#8217;s important is how often we choose an individual that <em>differs</em>, in that chosen trait, from what&#8217;s come before. Diversity is, after all, premised on <em>difference between members of a population</em>.</p>
<h2>Brass tacks: ~~Diversity = 4? Diversity = 0.004?~~ Diversity ≈ 0.012</h2>
<p>If we start thinking of diversity as defined by direct comparison of one member of a population to another, then the diversity of the one-nut &#8216;boundary case&#8217; that doomed our ratio idea becomes a moot question, like the <a href="http://en.wikipedia.org/wiki/Koan" title="koanic">koanic</a> clapping of a single hand. Mathematically, we can succinctly formalize our new insight &#8212; that diversity requires differences between individuals &#8212; by asking &#8216;<em>How often, if we pick two individuals at random from the population, will they differ in the trait of interest?</em>&#8216;. To grind out an actual number using this simple algorithm for assessing diversity, let&#8217;s reason through the possible outcomes of two blind picks from our jar of peanuts.</p>
<p>The first nut we pick will, of course, very likely be red. To be precise, the chance of getting a red nut on the first random pick is 0.994&#46; Now, unless we throw that first nut back in (instead of eating it) and remix the nuts thoroughly, the chance of picking a red nut on our second pick will differ slightly from what it was on the first pick; this is because, on the second pick, we&#8217;ll have only 999 nuts in the jar, of which 993 or 994 will be red. And 994/1000 equals neither 993/999 nor 994/999.</p>
<p>Those three numbers are, however, awfully similar to each other. In fact, we&#8217;re working with such a big &#8216;population&#8217;, and sampling such a small fraction of its &#8216;individuals&#8217;, that we can helpfully simplify our probability calculations by treating the chance of picking red as if it stays the same from pick to pick. Such a simplification is especially helpful in real-world sampling situations where, though unsure of just how many &#8216;nuts&#8217; are in a given &#8216;jar&#8217;, we&#8217;re quite sure that we&#8217;ll be sampling only a small fraction of them.</p>
<p>Under this simplifying &#8216;constant odds&#8217; (or, as it&#8217;s often termed by statisticians, &#8216;<a href="http://stats.oecd.org/glossary/detail.asp?ID=3835">sampling with replacement</a>&#8216;) assumption, the chance of picking two red nuts in our first two picks is 0.994 squared, or a little more than 0.988&#46; Which means, conversely, that our chance of not picking two red nuts in our first two picks is roughly 1 – 0.988, or 0.012&#46; Now, note that this latest number isn&#8217;t exactly the chance that the first two randomly picked peanuts will differ in color. Even if the two picks aren&#8217;t both red, after all, both could be yellow, or both green, or both blue. With our mostly red jarful, each of the latter cases is very unlikely, of course, but we can easily account for them.</p>
<p>To do so, we&#8217;ll again assume that the odds of picking a given color stay constant from pick to pick. For each of the three colors other than red, these odds are roughly 2/1000, or 0.002&#46; So the chance of picking two yellow nuts on consecutive picks will be roughly 0.002 squared, or 0.000004 &#8212; and the same holds for each other rare color in the jar. Overall, then, the chance that we&#8217;ll pick successive nuts that match in color will be roughly 0.988 + 3(0.000004), or, well, about 0.988&#46; Naturally, the chance that we pick successive nuts that differ in color is the complement of this matching probability; in the end, then, it&#8217;s still roughly 0.012&#46; Let&#8217;s adopt this number as our new &#8216;chance of mismatch&#8217;-based estimate of our particular jar&#8217;s nut color diversity.</p>
<p>For comparison, let&#8217;s run through the corresponding figures for a 1000-nut jar containing 250 nuts in each of four colors. In that case, the chance of picking two nuts of the same color on the first two picks will be roughly <code>4 * (0.25 * 0.25)</code>, which works out to 0.25&#46; Thus the chance of picking two nuts that differ in color will be roughly 1 – 0.25, or 0.75&#46; Consistent with intuition, this value is much greater than 0.012, our new estimate of our original jar&#8217;s diversity. And, importantly, our new &#8216;chance-of-mismatch&#8217;-based diversity measure is still normalized (bounded by 0 and 1), and still puts our jar&#8217;s color diversity squarely between the values obtained for a jarful of 1000 nuts in just one color, and a jarful of 1000 nuts in 1000 distinct colors.</p>
<h2>&#8216;Real-jar&#8217; uncertainty</h2>
<p>The foregoing is promising stuff, but we&#8217;re not yet in the clear. Remember that, if we look at a jarful of nuts from the outside, with no extra knowledge, we won&#8217;t actually know the odds of picking any given color ahead of time. As such, if we want a good estimate of &#8216;color diversity&#8217; for an unfamiliar jar of candy peanuts, we will have to repeat the two-pick experiment many times, after which we might notice that the two picked nuts differ in color in, say, roughly 1.2% (or 0.012) of trials. This is an arduous approach to estimating a measure of the nut &#8216;population&#8217;! But what if, instead of picking nuts one at a time, we could grab a whole handful (assuming that the nuts are distributed randomly in the jar) and then compare the colors of each pair of nuts in that &#8216;<a href="http://en.wikipedia.org/wiki/Statistical_sample" title="sample">sample</a>&#8216;, to see how often those paired &#8216;individuals&#8217; differ? Will that give us enough comparisons to make a good estimate of diversity, without having to spend time picking out nuts one at a time from the jar?</p>
<p>Well, suppose that our handful contains a whole number <em>k</em> of peanuts. The number of distinct peanut-pairings within that handful will be <em>k</em>(<em>k</em>-1)/2 (an expression that specialists in <a href="http://en.wikipedia.org/wiki/Discrete_mathematics" title="discrete mathematics">discrete mathematics</a> call &#8216;<a href="http://mathworld.wolfram.com/Choose.html" title="choose"><em>k</em>-choose-2</a>&#8216;). To confirm this, you can start by checking the number of possible pairings for some low values of <em>k</em>: two nuts can be paired in just one way; three nuts yield three distinct pairings; four nuts give six distinct pairings; and five nuts give ten. Notice that, for each of these low values of <em>k</em>, there are indeed <em>k</em>(<em>k</em>-1)/2 distinct pairings. You can then extrapolate that pattern to indefinitely higher values of <em>k</em> by noting that, each time you increase <em>k</em> by one (by adding a new nut to the handful), the number of new pairings that you add to the total number of pairings is simply the old value of <em>k</em> (<em>i.e.</em>, one less than the new value of <em>k</em>). For example, when you add a sixth nut to a group of five, that new nut can be separately paired with each of the first five nuts, but no other new pairings are made (and none are lost). By this insight, the new total number of pairings can be written as ((<em>k</em>-1)((<em>k</em>-1)-1)/2) + (<em>k</em>-1). This can be rewritten as ((<em>k</em>-1)(<em>k</em>-2) + 2(<em>k</em>-1))/2, which simplifies to, you guessed it, <em>k</em>(<em>k</em>-1)/2.</p>
<p>And, luckily, <em>k</em>(<em>k</em>-1)/2 grows quite quickly (nearly as quickly as the square of <em>k</em>, in fact) with increasing <em>k</em>, meaning that even a fairly small sample of individuals from a large population will provide many comparisons with which to assess its diversity. By asking what fraction of those many comparisons are mismatches (<em>caveat</em>: we&#8217;ll average over one less comparison than we can actually make, because the match/mismatch status of that &#8216;last&#8217; comparison turns out to be <a href="http://en.wikipedia.org/wiki/Degrees_of_freedom_(statistics)" title="degrees of freedom">fully determined</a> by the pattern of the other comparisons), we have a handy measure &#8212; the <em>pairwise chance of mismatch</em> &#8212; that can be readily calculated, and that accords basic intuitions about diversity, including the key difference between an overwhelmingly red jarful of candied peanuts, and a more compellingly colorful jar containing equal numbers of nuts in several colors.</p>
<h2>Nucleotide diversity</h2>
<p>In calling this newly defined measure the <em>color diversity</em> of our jar of nuts, we&#8217;ll be making a direct analogy to a key measure in population genetics: <a href="http://en.wikipedia.org/wiki/Nucleotide_diversity" title="nucleotide diversity"><em>nucleotide diversity</em></a> (or, as it&#8217;s often nicknamed, <em>polymorphism</em>). <a href="http://en.wikipedia.org/wiki/Nucleotide_diversity" title="nucleotide">Nucleotides</a> are, of course, the individual &#8216;beads&#8217; that are strung together to make up the genome. And, like the nuts in our jar, each nucleotide, if it&#8217;s there (insertions and deletions of nucleotides do happen, and complicate things by making it harder to tell which nucleotide in one genome &#8216;corresponds&#8217; to which nucleotide in another genome) comes in just one of four &#8216;colors&#8217;: the chemically distinct subunits called adenine, cytosine, guanine, and thymine (A, C, G, and T, for short), one of which, like a little flag, marks each nucleotide.</p>
<p>Before we explore what nucleotide diversity can tell us about a population, another word on why we used peanuts as analogs to human alleles. As anyone who has &#8216;sampled&#8217; through a bag of whole roasted peanuts knows, many a peanut shell cracks open to reveal two nuts, rather than just one. In this sense, a peanut shell can be likened to a human <a href="http://en.wikipedia.org/wiki/Zygote" title="zygote">zygote</a>, which corrals together two copies &#8212; one from mom, one from dad &#8212; of nearly any segment of the genome that we might pick to examine.</p>
<p>So why, in our mathematical musing about candied peanuts, did we conveniently &#8216;paint over&#8217; the fact that peanuts are born paired up in shells? Begging that question, I could plead that it&#8217;s hard to tastily coat (or even see) peanuts in easily countable colors while they&#8217;re still inside their shells. But, more to the point, taking the peanuts out of their shells underscores an analogous point about population genetics: as long as we&#8217;re just interested in estimating diversity for just one site in the genome (and not other, subtler characteristics of genetic variation in the population), the fact that alleles inside real genomes are paired up turns out to matter fairly little.</p>
<p>And, in a subtler extension of the analogy, it&#8217;s worth noting that two-nuts-per-shell is not a hardfast rule in the peanut world: some shells hold just one peanut, and still others hold more than two. Likewise, some human cells (<a href="http://en.wikipedia.org/wiki/Gamete" title="gametes">eggs and sperm</a>) carry no more than one copy of any given segment of the nuclear genome; other cells can carry just one copy of a particular segment, like the <a href="http://en.wikipedia.org/wiki/Y_chromosome" title="Y chromosome">Y chromosome</a>; and still other human cells carry more than two copies of a genome segment. The cells of people with <a href="http://en.wikipedia.org/wiki/Down_syndrome" title="Down syndrome">Down syndrome</a>, to cite a well known example, often have three copies of <a href="http://en.wikipedia.org/wiki/Chromosome_21" title="chromosome 21">chromosome 21</a>, rather than the more familiar two. And, in a <a href="http://www.ncbi.nlm.nih.gov/pubmed/6982661" title="X polysomy">small fraction of women</a>, many cells have three or more copies of the <a href="http://en.wikipedia.org/wiki/X_chromosome" title="X chromosome">X chromosome</a>. There&#8217;s an even more widespread <a href="http://www.ncbi.nlm.nih.gov/pubmed/19302968" title="mtDNA copy number variation">example</a> of this phenomenon, if we consider the little gene-crammed rings of DNA inside <a href="http://en.wikipedia.org/wiki/Mitochondrion" title="mitochondria">mitochondria</a>. These degenerate, energy-liberating bacteria live in the cells of animals, plants, and other so-called eukaryotes, and, along with their genomes, are typically transmitted to the next generation only via eggs, not sperm. Nonetheless, each cell in your body contains dozens to thousands of these little half-captive half-stowaways, each of which may, in turn, contain many copies of its own little genome. So while a typical human cell may hold two copies of, say, <a href="http://en.wikipedia.org/wiki/Chromosome_7" title="chromosome 7">chromosome 7</a>, as well as one or two or more copies of the X chromosome, it also holds many, many copies of the mitochondrial genome.</p>
<h2>A shelf full of jars</h2>
<p>In our analogy, the candied peanuts in one jar represented copies of <em>one site</em> in a genome. We thus derived an estimate of &#8216;diversity&#8217; for that site only. But we can readily expand the scope of our inquiry, and ask about the amount of variation found in a population of copies of a long <em>segment</em> of a genome. To do so, we can simply average the site-specific diversity estimates from throughout that segment, yielding a statistically robust &#8212; and very useful &#8212; estimate of its overall nucleotide diversity. Roughly speaking, this is akin to calculating the diversity of peanuts in each jar lined up on a whole shelf of such jars, and then averaging those estimates, to get a &#8216;shelf-specific&#8217; estimate.</p>
<p>To the extent that we then build a valid population-genetic model to represent the population of copies of our genome segment, we can &#8216;plug in&#8217; our segment-specific estimate of nucleotide diversity in order to shed surprisingly bright light on the <em>ancestral history</em> of the copies of the segment. Some of the cryptic, but newly &#8216;guessable&#8217; characteristics of that history might include: the &#8216;<a href="http://en.wikipedia.org/wiki/Effective_population_size" title="effective population size">effective</a>&#8216; size of the ancestral population, as roughly averaged over time; the average number of generations elapsed since the death of the last common ancestor of two randomly picked copies of the segment; the average <a href="http://en.wikipedia.org/wiki/Mutation_rate" title="mutation rate">mutation rate</a> per nucleotide site, per generation; and/or the average number of migrants exchanged by two partially isolated sub-populations, per generation (provided that the population we&#8217;re looking at is divided into such sub-populations, and we estimate diversity for each one separately, as well as for the pair as a whole). Moreover, if we compare results from our segment of interest to those from other segments in the genomes of the same &#8216;host&#8217; population &#8212; or to results from the &#8216;same&#8217; segment in a different population altogether &#8212; we may find strikingly dissimilar estimates of nucleotide diversity. Such disparities can turn up by chance, but they can also reflect robust underlying variation in, say, mutation rate, population size, or the stringency of natural selection. Future posts will explore, in more meaningful detail, how simple numbers such as <em>nucleotide diversity</em> can be leveraged to draw detailed evolutionary inferences about these and other factors.</p>
								</div><!-- .entry-content -->
								<div class="entry-meta"></div>
							</div><!-- .post-content -->
						</div><!-- .hentry -->
			</div><!-- .hfeed -->
		<div class="loop-nav">
			<div class="previous"><a href="/2009/04/20/on-polymorphism-ii-hard-drinking-fruitflies" rel="prev">&larr; On polymorphism II: Hard-drinking fruitflies</a></div>			<div class="next"><a href="/2009/05/01/polymorphism-i" rel="next">Polymorphism I &rarr;</a></div>		</div><!-- .loop-nav -->
		</div><!-- #content -->
	<div id="sidebar-secondary" class="sidebar">
		<section id="nav_menu-5" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Reference</h3><div class="menu-reference-container"><ul id="menu-reference" class="menu"><li id="menu-item-95" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-95"><a href="http://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI): Most biologists&#8217; de facto home page. You can use it too.</a></li>
<li id="menu-item-96" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-96"><a href="http://tolweb.org/tree/">Tree of Life: Earth&#8217;s diversity of life forms, and how they&#8217;re related to one another</a></li>
<li id="menu-item-97" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-97"><a href="http://genome.ucsc.edu/">UCSC genome browser: Fast, small-footprint rival to NCBI for genome sequence data-mining</a></li>
</ul></div></section><section id="nav_menu-4" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Fun</h3><div class="menu-fun-container"><ul id="menu-fun" class="menu"><li id="menu-item-98" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-98"><a href="http://www.facebook.com/group.php?sid=0b12e602a592d4dc019506c99b641cf2&#038;gid=53660447230&#038;ref=search">Effective Human Population (at Facebook): The Few. The Proud. Damn, it feels good to be effective.</a></li>
<li id="menu-item-99" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-99"><a href="http://helenpearsonphotography.com/music">Photo break: atmosphere-soaked images of music by National Geographic award-winning photographer Helen Pearson</a></li>
</ul></div></section><section id="nav_menu-6" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Data analysis</h3><div class="menu-data-analysis-container"><ul id="menu-data-analysis" class="menu"><li id="menu-item-100" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-100"><a href="http://opensnp.org/">openSNP: Portal for sharing, and fruitfully comparing, personal genome data.</a></li>
</ul></div></section><section id="nav_menu-7" class="widget widget_nav_menu widget-widget_nav_menu"><h3 class="widget-title">Blogroll</h3><div class="menu-blogroll-container"><ul id="menu-blogroll" class="menu"><li id="menu-item-101" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-101"><a href="http://gcbias.org/">GCbias: Graham Coop makes genomic complexity, well, fun.</a></li>
<li id="menu-item-102" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-102"><a href="http://genomesunzipped.org/">Genomes Unzipped: Thoughtful reflections on personal genomics by leading lights in the field.</a></li>
<li id="menu-item-103" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-103"><a href="http://haldanessieve.org/">Haldane&#8217;s Sieve: Read about new research here first, and straight from authors.</a></li>
<li id="menu-item-104" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-104"><a href="http://caseybergman.wordpress.com/">I Wish You&#8217;d Made Me Angry Earlier: Casey Bergman, voice of genomic reason and memory.</a></li>
<li id="menu-item-105" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-105"><a href="http://www.michaeleisen.org/blog/">it is NOT junk: Trenchancy from Michael Eisen. Read this.</a></li>
<li id="menu-item-106" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-106"><a href="http://massgenomics.org/">MassGenomics: Worthwhile thoughts from Dan Koboldt</a></li>
<li id="menu-item-107" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-107"><a href="http://www.yuzuki.org/">Next Generation Technologist: Dale Yuzuki opines on deep-sequencing methods and insights</a></li>
<li id="menu-item-108" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-108"><a href="http://omicsomics.blogspot.com/">Omics! Omics! Plumb Keith Robison&#8217;s knowhow on methods and tools for understanding genomes.</a></li>
<li id="menu-item-109" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-109"><a href="http://phenomena.nationalgeographic.com/">Phenomena: Lively summaries of eclectic new research from Ed Yong, Carl Zimmer, and others. (And where else would Genomena take you, after all?…)</a></li>
<li id="menu-item-110" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-110"><a href="http://phylogenomics.blogspot.com/">Phylogenomics: Jonathan Eisen surfs the sea of sequence</a></li>
<li id="menu-item-111" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-111"><a href="http://feeds.feedburner.com/RazibKhansTotalFeed">Razib Khan&#8217;s feed: Savvy (often bitingly so) genomic insights into human ancestry and behavior.</a></li>
<li id="menu-item-112" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-112"><a href="http://whyevolutionistrue.wordpress.com/">Why evolution is true: Sageness from the broad and incisive mind of Jerry Coyne.</a></li>
</ul></div></section>
	</div><!-- #sidebar-secondary .aside -->
				</div><!-- .content-wrap -->
		</div><!-- #main -->
		<div id="footer">
			<div id="footer-content" class="footer-content">
            <p class="copyright">Copyright &#169; 2017 <p class="credit"> Powered by <a href="http://alienwp.com">Oxygen Theme</a>.</p>
			</div>
		</div><!-- #footer -->
		</div><!-- .wrap -->
	</div><!-- #container -->
	<script type='text/javascript' src='/wp-content/themes/oxygen/js/jquery.imagesloaded.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/jquery.masonry.min.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/cycle/jquery.cycle.min.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/fitvids/jquery.fitvids.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/navigation.js?ver=20130301'></script>
<script type='text/javascript'>
/* <![CDATA[ */
var slider_settings = {"timeout":"6000"};
/* ]]> */
</script>
<script type='text/javascript' src='/wp-content/themes/oxygen/js/footer-scripts-light.js?ver=1.0'></script>
<script type='text/javascript' src='/wp-content/themes/oxygen/library/js/drop-downs.min.js?ver=20130805'></script>
<script type='text/javascript' src='/wp-includes/js/wp-embed.min.js?ver=4.5.6'></script>
</body>
</html>
