<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Genetics, evolution, and policy &#8211; Genomena</title>
	<atom:link href="http://genomena.com/category/genetics-evolution-and-policy/feed/" rel="self" type="application/rss+xml" />
	<link>http://genomena.com</link>
	<description>As genomes evolve, so does the world.</description>
	<lastBuildDate>Wed, 01 Feb 2017 02:46:36 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.5.6</generator>
	<item>
		<title>Three small steps toward genomically sensible healthcare</title>
		<link>http://genomena.com/2013/08/26/three-small-steps-toward-genomically-sensible-healthcare/</link>
		<comments>http://genomena.com/2013/08/26/three-small-steps-toward-genomically-sensible-healthcare/#respond</comments>
		<pubDate>Mon, 26 Aug 2013 08:44:30 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=55</guid>
		<description><![CDATA[Sailing a close reach, San Francisco Bay. (Image copyright Nathaniel Pearson) A talk at the Clinical Genome Conference resonated with some folks, who suggested sharing...]]></description>
				<content:encoded><![CDATA[<p><img src="/wp-content/uploads/2013/08/Sailing-1024x625.jpg" alt="Sailing a close reach, San Francisco Bay. (Image copyright Nathaniel Pearson)" /></p>
<p>Sailing a close reach, San Francisco Bay. (Image copyright Nathaniel Pearson)</p>
<p><a href="http://www.slideshare.net/NathanielPearson/pearsontcgc2013">A talk</a> at <a href="http://www.clinicalgenomeconference.com/">the Clinical Genome Conference</a> resonated with some folks, who suggested sharing it. Crucially, that crowd included doctors, who have much to both teach and learn in the brave new world of genomic medicine. With them on hand, the day&#8217;s session loosely echoed a <a href="http://en.wikipedia.org/wiki/Grand_rounds">grand rounds</a>, where the case was, soberingly, the tall order of making genomes widely useful in healthcare.</p>
<p>In such circles, a genomicist like me is a narrow specialist. Bigger speakers &#8212; general practitioners, so to speak &#8212; were lined up to cover the chronic, integrative needs on the <a href="http://www.sciencenews.org/view/generic/id/346343/description/Buzzword_bingo">conference bingo</a> card: <em>convening stakeholders</em>; <em>establishing standards for reporting and payment</em>; <em>managing big data</em>&#8230;a wordsquall that looms over our field, <a href="http://quoteinvestigator.com/2010/04/23/everybody-talks-about-the-weather/">easy to discuss</a> but hard to fix.</p>
<p>So rather than tackle such broad challenges, my talk stayed bite-size. Building from insights on genome structure, function, and variation, it urged three small but concrete ways to help put genome-informed healthcare on firmer footing.</p>
<ul>
<li>Use different reference genomes to align a person&#8217;s raw data (pick reference(s) most like her/him) versus store her/his finished genome (as clear or potential differences from the <em>human ancestral reference</em>).</li>
<li>Clinically classify <em>genotypes</em>, not variants.</li>
<li>Filter a genome against other <em>individuated genomes</em>, not allele frequency tables.</li>
</ul>
<p>Though these ideas aren&#8217;t new, they would break convention &#8212; so need justifying. But even if you skip the explanations that follow (starting with this post), know that the proposals reflect long thought on how current convention, rooted in sparse data, will ultimately fail for millions of whole human genomes. Thus consider them early course tweaks that can save bigger tacks later, en route to genomically informed healthcare for all of us.</p>
<h2>Small step I: The right reference genome(s)</h2>
<p>A reference genome &#8212; we&#8217;ll just say <em>reference</em> &#8212; is a long string of letters used as a common template for comparing the genomes of closely related organisms, such as people. As an archetype, a reference often shortens and simplifies real genomes,[1] to help <em>read</em>, <em>write</em>, or <em>interpret</em> them.</p>
<p>In teasing apart these tasks, note that today we use the same human reference for all three&#8230;and that it&#8217;s right for none of them. Below we&#8217;ll see why, and what we should do about it. But if you&#8217;re rushed, here&#8217;s the gist:</p>
<p><em>The current single reference is arbitrary and ethnocentric; inevitably misaligns most people&#8217;s raw data; and is poor for writing and interpreting genomes afterward, because it includes rare and risky variants, and muddles summary insights on data quality and evolution.</em></p>
<p>An alternative made of just common or putatively healthy variants would still be unreliable for aligning raw data, and as a foil for writing and interpreting genomes.</p>
<p>Instead, we should read your genome by aligning raw data to references most like you (we can usually guess which). We should then write all our genomes against the human ancestral reference &#8212; a solution that&#8217;s ethnically neutral, directly informative on data quality and evolution, and stabler than alternatives.</p>
<p>And we should give up on using any reference to proxy an idealized healthy genome. As later posts will detail, reliable health insight will instead require comparing your genome to the individuated whole genomes of many other people who, like each of us, get some diseases and not others.</p>
<p>Ok, now let&#8217;s walk through those reference tasks in detail, to better understand why we must do them differently.</p>
<h2>Read</h2>
<p>To read your genome &#8212; that is, to make out the long eye chart of letters that form it &#8212; a modern sequencer streams zillions of DNA snippets, each copying a chromosome tract roughly at random. By comparing each snippet to a good reference, a computer can find where it best fits, much as we match jigsaw puzzle pieces to the picture on the box. As snippets pile up, the computer surveys what DNA letter(s) amass over each spot, to guess what letter(s) your chromosomes carry there.[2] Conventionally, we&#8217;ve taken a one-size-fits-all approach to this task of <em>aligning</em> snippets, using the same reference, called <em>Hg#</em> (where today # = <a href="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips/">19</a>), to scaffold everyone&#8217;s genomes.[3] But Hg# wasn&#8217;t carved in stone. Instead, it&#8217;s quilted from <a href="http://seqanswers.com/forums/archive/index.php/t-22278.html">several real people</a>&#8216;s genomes that were read by costly, reference-free methods. And the haphazardly picked people who contributed to it have their own ancestry, which gives Hg# their genetic quirks.</p>
<p>As a result, some human genomes are more like Hg# than others. And if my genome resembles it more than yours does, my snippets will, on average, align more reliably. Conversely, because <a href="/ne/">big populations tend to be genetically diverse</a>, Hg# &#8212; like <em>any</em> single option &#8212; inevitably misaligns raw data from <em>most</em> people&#8217;s genomes, in ways both big (mutual gaps and rearrangements wreak havoc) and small (clustered small differences leave good snippets unaligned).</p>
<p>In the end, this means that <em>we can best read your genome today by first aligning to the available genome(s) most like it.</em>[4] Happily, <a href="http://en.wikipedia.org/wiki/SNP_array">skimming</a> your genome &#8212; or even just looking at you &#8212; strongly hints whose genome(s) might work best.[5] How well we can play reference sommelier depends on what options are on hand (more and more, starting with <a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002280">synthetic references</a> that proxy what&#8217;s common in a particular part of the world), and how saliently <em>mixed</em> your recent ancestry is. But if needed we can try <em>multiple</em> references, and see which work best for which snippets.</p>
<p>And that raises two deeper points. First, for some genome segments, such as a <a href="http://en.wikipedia.org/wiki/Human_leukocyte_antigen">stunningly diverse and health-relevant stretch of chromosome 6</a>, it&#8217;s hard to predict what your genome looks like <em>regardless</em> of where your forebears came from. For such segments, it makes sense to <em>always</em> align your snippets to many reference options.[6] Doing so takes a few more electrons, but usefully sharpens the resulting picture of your genome.</p>
<p>Second, aligning your snippets to even one whole <em>real</em> genome would itself be like aligning them to <em>two</em> versions of a conventional reference (with its one copy of each chromosome that&#8217;s paired in real genomes). Smartly, that could fully leverage new algorithms that track <em>everywhere</em> a snippet decently fits during alignment, rather than just picking one spot (often by tossup). And that, in turn, would let us read your genome more finely, without &#8212; yet &#8212; needing the compact simplicity of a conventional reference like Hg#.</p>
<p>Which brings us to the next use of a reference&#8230;</p>
<h2>Write</h2>
<p>After we read your genome in detail, a reference helps <em>write</em> it. Namely, because copies of a given human chromosome are all grossly alike, we can thriftily store yours by just noting where one or both mismatch a simple (single-copy) reference, or were read too poorly to tell.</p>
<p>Everywhere else &#8212; typically, >95% of the currently sequenceable parts of human chromosomes &#8212; we can assume that your copies match that reference. And because many of your poorly read sites will themselves clump in compressible tracts, we can shrink your genome >>20-fold in the end. That saves memory, of course, but also helps us query it &#8212; most usefully, by comparing your DNA (plus your phenotypes, ideally) to others&#8217;.</p>
<p>But there&#8217;s a catch. Because mutation anywhere on a chromosome can make one copy of it <em>longer</em> than another, genomes can best be compared if stored as differences from the same reference, so their mapping coordinates match. That way, like sailors agreeing to <a href="http://en.wikipedia.org/wiki/Prime_meridian">track longitude from Greenwich</a>, we can neatly record findings like <em>&#8216;One of your chromosome 7′s shows five more bases (ACGTA) than mine at reference site 1000; but one of mine shows three fewer bases at reference sites 2000-2002</em>&#8216;&#8230;</p>
<p>Note the dilemma here: to read genomes, we should align their snippets to <em>various</em> most-appropriate real reference(s); but to compare them, we should write them as differences from the <em>same</em> simple reference.[7] Bottom line, we need task-specific references.</p>
<p>But that still means picking one best reference for writing genomes. Given that so much work has gone into Hg#, we might ask whether <em>it&#8217;s</em> the right one. Which leads us to the third use of references&#8230;</p>
<h2>Interpret</h2>
<p>After shrinking your genome to a list of differences from a reference, we&#8217;d like to understand that list &#8212; what it says about how sequencing went and, more importantly, about you. We might even hope to use the reference to proxy a <em>healthy</em> genome, so that anything worrisome in your genome stands out from it.</p>
<p>Alas Hg# makes a poor interpretive foil for real genome data, starting with quality control: because Hg# comes from a few modern people, it&#8217;s poor not just for aligning, but also for writing, where it can conflate statistical signatures of lab-bench problems (sample contamination, chemistry failure, &amp;c.) with those of ancestry. QC that first compares <em>heterozygosity</em> of particular genome segments, rather than just counting reference sites called with any mismatch, will help there (an issue for another day&#8230;), but the problems with the current reference go deeper.</p>
<p>In particular, Hg# also includes many variants already <a href="http://www.ncbi.nlm.nih.gov/pubmed/21121051">implicated in diseases</a> &#8212; which means it won&#8217;t always flag your own worrisome DNA spellings[8] , and that it troublesomely differs from some <a href="http://www.lrg-sequence.org/">single-gene references</a> familiar to clinical geneticists. Moreover, Hg# includes many other variants that, while not yet well studied, are <a href="/rare-variants-and-health/">suspiciously rare</a> enough to be harmful too.</p>
<p>Given those shortcomings, many have suggested replacing Hg#&#8217;s rare and/or known risky variants with common and/or healthy alternatives, ostensibly yielding a new reference that reliably proxies a healthy, normal genome.</p>
<p>Alas, that won&#8217;t work, for two reasons. <em>What&#8217;s common varies. And what&#8217;s healthy depends.</em></p>
<h2>Informative beats (un)healthy</h2>
<p>At first glance, one of your DNA spelling variants may be rare enough on earth overall to intrigue us &#8212; but turn out to be boringly common among millions of mostly healthy people in some small patch of the planet. More profoundly, the commonest variant at a genomic site today or in five years may not be the commonest one next year or in ten years. That&#8217;s evolution &#8212; and it means that a common-only reference is inherently unstable.</p>
<p>On the health side, meanwhile, many variants aren&#8217;t simply good or bad. Their effects depend on what how many copies you have (0, 1, or 2), what disease we ask about, and what <em>other</em> variants lurk in your genome.</p>
<p>You may know a few such twists already. <em>One or two copies of T <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?position=chr11:5247982-5248482&amp;hgsid=345050099&amp;snp137Common=pack&amp;hgFind.matches=rs334,">here</a> help avoid malaria and high cholesterol &#8212; but two copies leave you with crippling anemia. One copy of A over <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgHubConnect.destUrl=..%2Fcgi-bin%2FhgTracks&amp;clade=mammal&amp;org=Human&amp;db=hg19&amp;position=chr17%3A41243941&amp;hgt.positionInput=chr17%3A41243941&amp;hgt.suggestTrack=knownGene&amp;Submit=submit&amp;hgsid=345051025">there</a> can drive breast cancer, but mainly if you also lack a working copy of the <a href="http://en.wikipedia.org/wiki/SRY">SRY</a> gene (which, on the flipside, helps you avoid testicular cancer, among other diseases&#8230;).</em> And so forth.</p>
<p>Data from millions of us will unfurl more astounding complexity, where variants throughout your genome &#8212; some inevitably present in any reference we use &#8212; interact in surprising ways with each other, and with habits and other factors, to favor some diseases and disfavor others.[9] Other posts will further explore how this hard truth should alter our approach to genomic healthcare. Here, it simply dooms any hope of using any reference to reliably proxy what&#8217;s healthy.</p>
<p>And more deeply, using a reference like Hg# as an interpretive yardstick also obscures how genomes change and, by extension, how various kinds of changes tend to affect health in the first place. Hg# can&#8217;t, for example, tell us whether a so-called <em>deletion</em> in your genome (where it&#8217;s missing a tract found in Hg#) really reflects a mutation that deleted bases in you or your forebear, or instead reflects an <em>insertion</em> of bases in someone who contributed to Hg#.</p>
<p>As such, because a given letter in a reference like Hg# could itself reflect a past mutation, writing <em>everyone&#8217;s</em> genomes as differences from Hg# makes statistical questions like <em>&#8216;How often does the snippet CG mutate to TG? And how well does that TG survive, over generations, if it changed a protein&#8217;s arginine to cysteine?&#8217;</em> trickier than they should be.</p>
<p>Such questions matter. They can unlock basic physiology (<em>How do mutations happen? Why do tumors correct them so poorly?</em>); hint how a new variant may affect health (<em>Does changing active-site arginine to cysteine often make an enzyme fail?</em>); and clarify how variants <em>interact</em> with each other, and with habits, to cause disease (<em>Why do some genetic variants, like <a href="http://snpedia.com/index.php/ApoE-%CE%B54">APOE-ε4</a>, make us sick but leave chimpanzees healthy?</em>).</p>
<p>Those big questions require the big data inside us. Even if no more than a handful of your DNA spellings alter your own healthcare, the rest of them, pooled with similar data from all of us, can shed light on many diseases to greatly refine care for our grandkids.</p>
<p>But using a conventional reference like Hg# needlessly hinders that effort. So while we must abandon the idea of any reference reliably proxying a healthy genome, can we at least find a sensible reference to write and compare the coming flood of genomes, to catalyze those deeper insights?</p>
<h2>An ancestral reference</h2>
<p>We can. The sensible yardstick for writing your genome is the <em>human ancestral reference</em> (<em>HAR</em>) &#8212; that is, a single-copy genome comprising, at each chromosomal site, the DNA letter carried by the last common ancestor of all people for that site.</p>
<p>In picturing the HAR (which <a href="https//twitter.com/suganthibala">Suganthi Balasubramanian</a> and colleagues have already <a href="http://genesdev.cshlp.org/content/25/1/1.long">described here)</a>[10] , note something very important: two genomic sites, even on the same chromosome, can trace to <em>different</em> last common ancestors. That&#8217;s because, when eggs and sperm are made, chromosomes pair up, swap segments, and move into different cells. Each copy of a chromosome thus quilts together pieces of earlier copies; so everyone&#8217;s last common ancestor for site 1000 may not be our last common ancestor for site 1001 (they may have even lived eons apart). Which also means it&#8217;s implausible that any person ever carried the whole HAR.[11] Among reference options for writing and comparing our genomes, the HAR uniquely combines several appealing features:</p>
<ul>
<li><em>It&#8217;s neutral.</em> As noted, no one ever carried the whole HAR. And because the mutations that distinguish our genomes from it have struck roughly randomly among our ancestors, your genome resembles it about as much as mine does.[12]
<p>In this important sense, the HAR belongs to none of us, and to all of us. Being roughly equidistant from everyone, it offers a uniform, non-ethnocentric baseline for assessing sequencing quality, and for reporting what&#8217;s genetically distinctive about you.</p>
</li>
<li>
<p><em>It&#8217;s stable.</em> The HAR actually looks a lot like a common-variants-only reference, because nearly all ancestral variants are common. But while a common-only reference would in principle need many edits each year to stay perfectly accurate, the HAR would need just a few (as atypically rare ancestral variants are newly found or, occasionally, learned to have gone extinct).[13]</p>
<p>Such editing isn&#8217;t urgent, because few variants with allele frequency near 50% are functionally intriguing enough, or surveyed precisely enough in the population, to day-trade anyway. But that just makes it even smarter to build a reference on the stable, reliably inferrable, and meaningful criterion that a variant be ancestral, rather than worry whether its allele frequency fell to 49.9%. That way, we get summary insights even from otherwise boring variants &#8212; and a low-maintenance reference to boot.</p>
</li>
<li>
<p><em>It&#8217;s compact but comprehensive.</em> Like conventional references, the HAR is a simple single-copy (<em>haploid</em>) genome. Real genomes, compressed against it, would yield files consistently intermediate in size between the biggest and smallest files compressed against Hg#.[14]</p>
<p>Nonetheless, because new chunks of DNA are usually copied from chunks elsewhere in the same genome, the HAR includes source DNA for nearly all chunks of real human genomes (missing only those recently copied from viruses or bacteria, or other oddities). Other reference options tend to be less comprehensive on these counts, which poses an ongoing dilemma of when to add a segmental copy (to make them more thorough), versus omit it (to keep them compact).[15]</p>
<p>That dilemma would still hold, but the HAR offers a framework for handling extra segments that we choose to include in the <em>extended</em> HAR that <a href="http://genesdev.cshlp.org/content/25/1/1.long">Subramanian <em>et al.</em> proposed</a>. For a newly arisen extra segment that some but not all people have, variation <em>among</em> such copies could in turn be mapped to common coordinates in the inferred earliest (nearest to universally ancestral) version of the new copy.</p>
</li>
<li>
<p><em>It&#8217;s directly informative.</em> Most importantly, the HAR is the only reference option that directly shows how human genomes change. As a foil for writing all our genomes, it would thus most quickly reveal summary patterns of change that in turn shed light on basic biology and health.As a concrete example, if shortening a particular bend shared by a particular family of proteins makes people sick, but lengthening it &#8212; or shortening another bend &#8212; leaves people healthy, the HAR would let clinical geneticists reliably spot this trend faster than other references would.</p>
</li>
</ul>
<p>The benefits of an ancestral reference for making sense of genomes, both individually and together, have long been starkly clear for geneticists studying the first fully sequenced human chromosome: the mitochondrial genome (mtDNA).</p>
<p>Starting in 1981, we used the <a href="http://www.ncbi.nlm.nih.gov/pubmed/7219534">first sequence of a person&#8217;s mtDNA</a> as a reference. That sequence forms a leaf on the <a href="http://www.phylotree.org/tree/main.htm">simple evolutionary tree</a> that binds all our mtDNA versions. And because each of us gets only our mom&#8217;s version of this short but gene-rich chromosome, with no backup from dad, that tree&#8217;s branches are key foci of health research.</p>
<p>But using a modern person&#8217;s mtDNA as a reference meant treating a leaf as if it were the treetrunk. This everted our view of the tree, prompting <a href="http://www.dailymotion.com/video/xerwsh_carl-sagan-videos-epicycles-of-ptol_tech">epicycle</a>-like contortions to figure out where your leaf was, and how your branch may or may not have mutated in telling ways.</p>
<p>In 2012, researchers cut that gordian knot, proposing the <a href="http://www.mtdnacommunity.org/">human ancestral mtDNA</a> as a reference for writing real genomes. That new reference lets you easily a) find your mtDNA leaf, and b) see how DNA has changed throughout the tree, to better understand key biological processes.</p>
<p>Having learned the hard way with mtDNA, we needn&#8217;t wait 31 years for our other chromosomes. In the end, by using multiple references to align raw data, and adopting the HAR to write and compare our finished genomes, we can best read, write, and learn from the millions of human genomes soon to be sequenced.</p>
<h2>Getting smarter with references</h2>
<p>So where are we, as a community, on human reference genomes?</p>
<p>There&#8217;s modest reason to hope. Researchers have begun to show how much better we might read genomes by <a href="http://www.plosgenetics.org/article/info:doi%2F10.1371%2Fjournal.pgen.1002280">aligning snippets to similar reference(s)</a>; that an <a href="http://genesdev.cshlp.org/content/25/1/1.long">ancestral reference helps</a> compare genomes <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322232/">more easily and informatively</a>; that <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmid/21121051/">Hg# doesn&#8217;t proxy a healthy genome</a>; and that <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3254080/">no alternative would reliably do so</a> either.</p>
<p>On the practical side, we&#8217;re accumulating diverse, ever better sequenced human genomes that can serve well as alignment references (and, as a bonus, help <a href="http://www.nist.gov/mml/bbd/biomolecular/genome_in_a_bottle_consortium.cfm">benchmark new sequencing methods</a>). And we&#8217;re getting better genomes from elsewhere in the great ape family tree, to refine the HAR.</p>
<p>Moreover, today&#8217;s <em>de facto</em> standard, Hg#, continues to improve via thoughtful work by <a href="http://infocus.nlm.nih.gov/2012/08/meet-ncbis-deanna-church-genom.html">Deanna Church</a>&#8216;s team and <a href="http://www.ncbi.nlm.nih.gov/pubmed/23932108">others</a>. Beyond fixing errors and filling in previously missing segments, the pending Hg20 version will include <em>multiple</em> versions of more segments, in part to better align raw data.</p>
<p>That&#8217;s a sensible stopgap, until more folks start picking from multiple alignment references from the start. But adding alternate versions of more segments to Hg# requires ongoing arbitrary choices, slows the task of writing finished genomes, and tends to statistically weaken comparisons of many genomes. The latter jobs are really better served by writing genomes against the HAR.</p>
<p>Communal habits like using Hg# for all human reference needs are hard to break &#8212; even for open-minded scientists (and maybe moreso in famously stubborn medicine). But given the clear flaws in our current approach to reference genomes, it&#8217;s likely better to break those bad habits now than let them entrench further, as we start sequencing patients&#8217; genomes by the thousands (and more).</p>
<p>Making all those genomes useful in healthcare, for us and future generations, will mean reading them well; writing them efficiently; and, as coming posts will explore further, interpreting them wisely.</p>
<p>All these goals rest on the bedrock of reference genomes. Let&#8217;s get them right.</p>
<p>[^l] Today&#8217;s ~2.9 billion-letter <a href="http://genome.ucsc.edu/cgi-bin/hgTracks">human reference</a>, for example, comprises just one version of each of the grossly distinct-looking molecules (chromosomes 1-22, X, Y, and M) in the >6.5 billion-letter genome of a man&#8217;s skin cell. That cell&#8217;s genome comprises <em>two</em> copies of most such chromosomes &#8212; and those copies, in turn, differ in chemical makeup (base sequence), and include tracts that have never been seen (or successfully sequenced), so are simply missing from the reference.</p>
<p>[^2] Importantly, many snippets from your genome differ from even their best-matching parts of the puzzlebox picture (otherwise, why bother sequencing?). But the reference template still helps piece them together faster than we otherwise could. And by piling <em>many</em> snippets over each site, we can tune out errors from cooking finicky chemicals under tiny image sensors &#8212; a bit like how astronomers, at the other end of the spatial scale, distinguish lasting light sources from noise by overlaying many pictures of the same part of the sky.</p>
<p>[^3] While researchers colloquially use the <em>Hg</em> prefix, this technically denotes the version stored at the University of California Santa Cruz&#8217;s popular <a href="http://genome.ucsc.edu/">genome data portal</a>; the current reference&#8217;s proper name is <em>GRCh##.p__, where ## (currently 37) is a _major assembly release</em> number higher than the # in Hg#, and _ is a <em>patch release</em> number.</p>
<p>[^4] That point was moot in 2003, when we had to use the hardwon sequence that became the current reference. But since then, we&#8217;re bootstrapping our way to good sequences of <em>many</em> human genomes from around the world &#8212; a pool that we should tap to better align newly sequenced genomes, as some folks have already shown.</p>
<p>[^5] Ideally, we&#8217;d use parents&#8217; genomes to align those of their kids&#8230;but when sequencing is common enough to make that practical, sequencers will likely make longer snippets that are easier to piece together from the start anyway, even without aligning to a reference.</p>
<p>[^6] Helpfully, Hg# itself includes several options for some such segments &#8212; and those who built and refine it plan to add more.</p>
<p>[^7] Note that even though we write them as differences from a simple reference, which has just one copy of each chromosome, we can still readily encode which spellings go <em>together</em> on each copy of your chromosomes (if our sequencing method was good enough to tell in the first place).</p>
<p>[^8] Especially if whoever compressed your genome didn&#8217;t bother noting where your genome was too poorly sequenced to know what it carries &#8212; a corner that geneticists too often cut.</p>
<p>[^9] Such insights stand to turn much of the noise that we currently sweep under the rug of <em>partial penetrance</em> into far better understood signal &#8212; think, for example, about how genetic insight turned the apparent random noise of why a baby was born female or male into causal signal tracing largely to the sex chromosomes.</p>
<p>[^10] For the remaining sites, we can&#8217;t reliably guess what variant our last common ancestor carried, because the state of variation we see among our copies extends to other great apes, suggesting that such variation has either arisen twice, or lasted too long to reliably unravel. In extreme cases, like the sex chromosomes, such lasting variation is already enshrined in the current reference (Hg# has one X sequence and one Y sequence, despite the fact that not everyone has the latter).</p>
<p>[^11] Even the last common ancestors who contributed to the HAR had variants in their own genomes that aren&#8217;t in it.</p>
<p>[^12] Planetwide, we have no idea whose genome happens to differ from it most, though that person &#8212; ironically, in some sense the most evolved (geneticists would say <em>derived</em>) of us &#8212; is almost certainly very sick, thanks to gross genetic changes&#8230;</p>
<p>[^13] Many thanks to <a href="http://gcbias.org/">Graham Coop</a> and <a href="http://www.genetics.wustl.edu/jflab/">Justin Fay</a> for helping think through the relevant numbers here.</p>
<p>[^14] Average compression is the main measure that could instead be optimized by a common-only reference. But the HAR has several substantive advantages over that less stable and informative option.</p>
<p>[^15] Note, btw, that this question matters most if we&#8217;re using a reference to align snippets &#8212; which we&#8217;re not proposing here. But we do need to map each of the alignment references themselves to the writing/comparing reference, which is where it helps to make sure the latter includes source DNA for the segments that those alignment references may have extra or fewer copies of.</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2013/08/26/three-small-steps-toward-genomically-sensible-healthcare/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The Shock of the New: What we can say about HeLa&#8217;s novel variants</title>
		<link>http://genomena.com/2013/04/01/the-shock-of-the-new-what-we-can-say-about-helas-novel-variants/</link>
		<comments>http://genomena.com/2013/04/01/the-shock-of-the-new-what-we-can-say-about-helas-novel-variants/#respond</comments>
		<pubDate>Mon, 01 Apr 2013 15:38:09 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=37</guid>
		<description><![CDATA[A human genome, remixed, and gone viral. In a nutshell, that&#8217;s the conditionally re-published genome of HeLa cells, which flourish in labs worldwide, but trace...]]></description>
				<content:encoded><![CDATA[<p>A human genome, remixed, and gone viral. In a nutshell, that&#8217;s the <a href="http://www.nature.com/news/deal-done-over-hela-cell-line-1.13511">conditionally re-published</a> genome of <a href="http://en.wikipedia.org/wiki/HeLa">HeLa</a> cells, which flourish in labs worldwide, but trace back seven decades to the unethically biopsied cervical tumor of American <a href="http://en.wikipedia.org/wiki/Henrietta_Lacks">Henrietta Lacks</a>.</p>
<p>Notably, HeLa&#8217;s genome offers vital glimpses of the woman whose life it stole. But as a posthumous portrait of Lacks, it&#8217;s at best a <a href="http://en.wikipedia.org/wiki/Cubism">cubist</a> one &#8212; her genomic likeness reduced to shards jumbled, mirrored, and spattered by decades of mutation. In time, though, it will prove to recognizably portray <em>her</em>, first in generically broad strokes, but eventually in more intimate ways that only she and her loved ones had known.</p>
<p>And that prospect sparked rancor in March, when the authors who first <a href="www.g3journal.org/content/early/2013/03/12/g3.113.005777.abstract">published</a> (and then withdrew, amid the hubbub) a HeLa genome patly <a href="http://phylogenomics.files.wordpress.com/2013/03/pr-diff.pdf">denied</a> that their data shed light on Lacks&#8217;s own <em>genome</em>, much less on who she was:</p>
<blockquote><p>
  [W]e cannot infer anything about Henrietta Lacks&#8217; genome, or of her descendants, from the data generated in this study [because] it is impossible to distinguish which parts of the genome sequenced here originate from Mrs. Lacks, her tumour, or laboratory adaptation.&#8217;
</p></blockquote>
<p>By this, they hoped to justify publishing a genome derived from one named person, without personal or family consent. That claim anticipated <a href="http://www.michaeleisen.org/blog/?p=1341">lively new discussion</a> of <a href="http://www.technologyreview.com/view/512966/the-dawn-of-genome-trolling/">genomic privacy</a> &#8212; but it was false, roughly like the finder of a rain-soaked wallet conveniently declaring the banknotes legible, but the IDs too wet to read.</p>
<p>Their own paper, after all, noted that HeLa shares nearly every noteworthy letter of its genome with other people (though their log-scaled bar plots downplayed this). And eventually, prodded by <a href="http://www.nytimes.com/2013/03/24/opinion/sunday/the-immortal-life-of-henrietta-lacks-the-sequel.html?src=me&amp;ref=general&amp;_r=1&amp;">other voices</a>, they conceded that we <em>can</em> learn and confirm some things about Lacks from HeLa&#8217;s DNA (or even <a href="http://www.genomesunzipped.org/2013/03/henrietta-lackss-genome-sequence-has-been-publicly-available-for-years.php">RNA</a>)[1] &#8212; but, they insisted, little if anything &#8216;new&#8217;.</p>
<p>And that&#8217;s truth&#8230;if not whole truth. For the authors apparently took <em>learning something</em> to mainly mean figuring out which of the 550 thousand or so <em>novel</em> (never before seen) variants they found had originally belonged to Lacks.[2] And yes, doing so without comparing HeLa&#8217;s genome directly to a non-HeLa genome from Lacks is indeed tough &#8212; at first glance, a genomic one-hand-clapping <a href="http://en.wikipedia.org/wiki/koan">koan</a>.</p>
<p>But even if identifying such variants really were the only route to learning something about Lacks, the authors ignored ways in which we <em>can</em> do so. Below, I&#8217;ll detail a few &#8212; which all boil down to thinking carefully about particular branches of the genetic family trees that link each segment of HeLa&#8217;s genome to the genomes of other cells and people.</p>
<p>Let&#8217;s start with novel variants that likely did <em>not</em> belong to Lacks.</p>
<h2>Novel variants that likely arose only in the HeLa cell lineage</h2>
<h3>Single-copy variants in duplicated haplotypes</h3>
<p>The HeLa genome is not easy reading. It&#8217;s basically a long, gibberish-spiked recipe for making tons of human cells. And if such a postmodern cookbook sounds riveting, you&#8217;ll be pleased to know that it&#8217;s written mostly in lawyerly triplicate.</p>
<p>In the decades after Lacks was born (perhaps in one big event, such as a faulty cell division), HeLa gained an extra copy of nearly every chromosome. And, though the new paper missed the next point, HeLa&#8217;s extensive <em><a href="http://en.wikipedia.org/wiki/Polyploid">triploidy</a></em> can help clarify which of its many distinctive variants arose by new mutation in the cell line, after Lacks was conceived (and perhaps after she died).</p>
<p>Basically, the duplication(s) that turned HeLa triploid effectively sprouted tiny new branches in the human genome&#8217;s family treetops, but <em>within</em> the HeLa genome itself. And some novel variants in HeLa can be confidently traced to those new branches, clarifying that they likely arose by mutation after Lacks was conceived.</p>
<p>Specifically, if a novel variant appears on just one of the three copies of a given chromosome in HeLa, <em>and the surrounding sequence is otherwise nearly identical to just one of the two other copies</em>, then the variant in question most likely arose by such new mutation. That is, Lacks likely wasn&#8217;t born with it. Similar logic can help us pick out likely new mutant variants in parts of the HeLa genome (such as long stretches of chromosomes 1, 5, and 11) that show even <em>more</em> than three copies.</p>
<p>Overall, to figure out which of the half-million or so heterozygous novel variants found in HeLa go into this bin, we&#8217;d need to systematically check which other variants they&#8217;re linked to on particular copies of particular chromosomes. With current lab methods, that&#8217;s hard &#8212; especially for novel variants, where we can&#8217;t turn to reference data from human populations &#8212; but feasible <strong>[Update: a new <a href="http://www.nature.com/nature/journal/v500/n7461/full/nature12064.html">paper</a> from <a href="http://www.gs.washington.edu/faculty/shendure.htm">Jay Shendure</a>&#8216;s group reports doing just that!]</strong>.</p>
<h3>Heterozygous mitochondrial variants</h3>
<p>Mitochondria are our cells&#8217; tiny furnaces. Each one houses a short, gene-crammed ring of DNA, attesting direct descent from a bacterium that, eons ago, slipped inside a bigger cell, to found <a href="http://en.wikipedia.org/wiki/Eukaryote">a line of ancestors</a> that we share mainly with plants, molds, and other animals.</p>
<p>Because all your mitochondria come from the few that your mother&#8217;s egg carried, they tend to be nearly identical within each cell, and throughout your body. That is, though their genomes do mutate (so can vary a bit from mitochondrion to mitochondrion), any resulting variation tends to wash out when we read a person&#8217;s DNA <em>en masse</em>.</p>
<p>As such, in a person&#8217;s mitochondria, we expect to find very few genomic sites &#8212; typically zero, in some people one &#8212; that look heterozygous (coming in two or more distinct flavors within a given genome) in the classic genetic sense. In fact, if we see more than one such site, we can guess fairly strongly that something has gone wrong either with sequencing itself (such as sample cross-contamination), or with the genome we&#8217;re looking at.</p>
<p>The HeLa researchers didn&#8217;t report how many of the novel mitochondrial variants, if any, appeared heterozygous in the HeLa genome. But if any did, it&#8217;s a good bet that most or all of them arose long after Lacks was born. And though this bin is likely very small, it&#8217;s one that can be fairly confidently assessed, in part by picking through raw sequence data more carefully; as such, it might tell us a <em>bit</em> about the point mutation rate in HeLa, relative to other tumor cell lines.</p>
<h3>Variants that don&#8217;t turn up in other HeLa lines</h3>
<p>This is the first big, obvious bin that will eventually subsume many of HeLa&#8217;s currently puzzling variants. Many variants that appear in the recently sequenced HeLa line will turn out not to appear in most or all other lines that we might eventually look at. While this can reflect <em>loss</em> of such variants, by deletion of segments of DNA in some HeLa lines, it more often will reflect <em>gain</em> of the variant in question in this particular line.</p>
<p>As such, fleshing out the genetic family tree of other HeLa lines will help clarify which such variants trace specifically to the branch that the new paper focuses solely on. Now, on to the other class of novel variants that interest us: those that likely belonged to Lacks from the start&#8230;</p>
<h2>Novel variants that Henrietta Lacks was likely conceived (and born) with</h2>
<h3>Novel ancestral variants</h3>
<p>In evolutionary parlance, an ancestral variant is, specifically, a variant carried by the last common ancestor of <em>everyone</em> in a population, for a given stretch of the genome, <em>before</em> any mutation there gave rise to a new variant that&#8217;s now found in some but not all individuals.</p>
<p>Most ancestral variants are still common among us, so were discovered long ago. But the haul of novel variants found in a given person may include the odd ancestral one that, til now, had never been hooked by researchers fishing in our gene pool.</p>
<p>Most such variants can be spotted easily, because they&#8217;re shared with many other mammals. Sometimes, such matches just reflect <em><a href="http://en.wikipedia.org/wiki/Convergent_evolution">homoplasy</a></em> – that is, mutation that happened to yield a variant already found in other organisms, perhaps at sites that tend to mutate a lot (such as Cs before Gs). But the commoner case &#8212; and the simpler guess, given how rarely mutation&#8217;s lightning strikes a typical site in the genome (and how rarer still it is for the mutation to happen to yield a particular variant seen elsewhere too) &#8212; is that a variant shared with most other mammals is really ancestral.</p>
<p>As such, among the novel variants called in HeLa, a few will confidently look ancestral &#8212; and most of these were, very likely, inherited by Lacks from her parents.</p>
<h3>Homozygous novel variants, outside LOH and/or cancer-implicated regions</h3>
<p>Whether ancestral or otherwise, novel variants tend to be rare &#8212; so rare that, by definition, they&#8217;d never been seen in a person before. As such, a given novel variant in your genome is very unlikely to also be carried by a randomly (er, carefully) chosen mate. And even if that person does carry a copy of the same very rare variant, there&#8217;s rarely more than a one in four chance that a child you&#8217;d have together will get two copies of it (one from each parent).</p>
<p>As such, nearly all novel variants found in your genome (or anyone&#8217;s) are <em>heterozygous</em> – that is, found on just one of your two copies of a given chromosome. Nonetheless, a smattering of novel variants may instead be homozygous, by the jackpot luck described above.</p>
<p>The HeLa genome reportedly harbors roughly 50 thousand homozygous novel variants &#8212; and that&#8217;s quite a lot, compared to most human genomes. Given that Lacks&#8217;s parents weren&#8217;t close relatives, I&#8217;d guess that many of these are in long stretches of the HeLa genome (such as most of chromosomes 6 and X) that look uniformly homozygous, likely due to <a href="http://en.wikipedia.org/wiki/Loss_of_heterozygosity">loss of heterozygosity</a>, where HeLa gained or lost copies of the given genome segment. Within these segments, and in genes that likely help govern HeLa&#8217;s growth, the origins of homozygous novel variants are hard to guess. Some may indeed have popped up by mutation in HeLa lineage, then become homozygous by secondary mutation. But others may have been in Lacks&#8217;s genome from the start.</p>
<p>But outside such regions, most homozygous variants found in HeLa &#8212; novel or otherwise &#8212; likely belonged to Lacks from the start. And we&#8217;ll eventually be able to confirm many of these guesses by looking at the next bin, below&#8230;</p>
<h3>Variants that turn up in other people</h3>
<p>This is the <em>other</em> big, obvious bin that will eventually subsume many of HeLa&#8217;s currently puzzling variants (the converse of the variants-that-don&#8217;t-turn-up-in-other-HeLa-lines bin described above). Of the 550-thousand-odd variants so far seen only in HeLa, many thousands will likely eventually be found in the genomes of living people whom we&#8217;ve yet to sequence. And most of these &#8212; especially if flanked by stretches of sequence that distinctively resemble the sequence flanking them in HeLa &#8212; will indeed be variants that Lacks was born with (rather than variants that arose in parallel by new mutation).</p>
<p>As such, it&#8217;s just a matter of time before we can trace many of the novel variants in HeLa to particular branches of the bigger human genetic family tree.</p>
<p>Importantly, some of these variants will, in turn, become functionally informative, once we&#8217;ve seen them enough times to see what kinds of traits people who carry them may distinctively share. Looking ahead, this is where HeLa&#8217;s genome will eventually show similarities to other genetic portraits of individuals, which in turn refine not just our picture of Lacks&#8217;s genome, but of what she was like as a person, inside and out.</p>
<h2>Further thoughts</h2>
<p>On first hearing that HeLa had been publicly sequenced, I smiled &#8212; a bit wistfully &#8212; and recalled stumping, in 2009, to do just that[3]. If you&#8217;re working at a genome interpretation company, and want to pick just one human(oid) genome to sequence for public good, you could do far worse than HeLa. It&#8217;s a ubiquitous cell line, after all, whose genome says a bit about cervical cancer; a bit more about how lab-farmed cells find ways to thrive; and, yes, a lot about Henrietta Lacks, who, suffering in rough conditions in a segregation-era cancer ward, made an unwitting, coerced, but remarkable bequest to human knowledge and health.</p>
<p>In these senses, the HeLa genome could anchor an engaging public panel discussion among geneticists, oncologists, ethicists, and, crucially, members of the Lacks family. Glibly, it might start with a bird&#8217;s eye look at cancer as <a href="/genomes-and-health/why-cancers-are-the-least-and-most-genetic-of-diseases#hive">doomed cellular mutiny</a>. Next, meeting Henrietta herself through family memory, we&#8217;d learn how her particular tumor <a href="/genomes-and-health/why-cancers-are-the-least-and-most-genetic-of-diseases#HeLa">eluded such doom</a> by appealing, ironically, to humanity&#8217;s needs. That notion raises the thorny ethics of using a useful but unethically obtained resource (with no direct recompense to the family who suffered most in giving it to the world), and in turn opens the meta-wormcan of genomic privacy. And new insights from the HeLa genome itself help weave these threads into emerging genomic portraits of the cell line and the little-sung woman it traces to.</p>
<p>As noted, such a discussion would most enlighten if it directly included members of the Lacks family, who could reflect personally on Henrietta, HeLa, and genetic privacy &#8212; and who embody how segments of her genome also live on, most vitally, in the kin she loved and would have loved.</p>
<p>Pat, yes. But HeLa&#8217;s genome clearly adds something to an already compelling tale &#8212; one that <a href="http://rebeccaskloot.com/">Rebecca Skloot</a>, who had <a href="http://www.jhu.edu/~jhumag/0400web/01.html">long been writing</a> about Lacks and her cells, soon told in much finer detail, in <em><a href="http://www.nytimes.com/2010/02/07/books/review/Margonelli-t.html?pagewanted=all">The Immortal Life of Henrietta Lacks</a></em>.</p>
<p>As that book makes clear, the HeLa saga has long blended travesty, triumph, and quandary in roughly equal parts. The debate over its genome is thus just a new eddy of an old storm. The last few months, like the previous 71 years, have seen useful data tainted by heedless ethics; public efforts to mend, if not undo, a wrong; and, perhaps most pressingly, lasting questions about how we should use scientific resources whose provenance is, at best, murky.</p>
<p>We haven&#8217;t dwelled here on how, in bypassing the Lackses, the first HeLa genome paper furthered a history of disregard for real <em>people</em> who carry forth Henrietta&#8217;s DNA, and who mourn her most deeply. Nor have we tried to define what genetic privacy means, given that far more of <em>any</em> genome is complicatedly shared &#8212; with kin, both known and unknown &#8212; than is truly private.</p>
<p>Rather, we&#8217;ve focused on the question of what we can indeed learn from a single genome, which matters for three main reasons.</p>
<p>First, the question cuts across the authors&#8217; initial claim that the genome of HeLa (read: any tumor or cell line) told us nothing about Henrietta Lacks (read: any person from whom such cells are taken). That technical claim doesn&#8217;t hold up, as we&#8217;ve seen.</p>
<p>Second, it points, more broadly, to what we can, and cannot, reasonably hope to learn today about a person just by looking at a genome. The paper itself confirmed, via HeLa&#8217;s genome, that Lacks was indeed a woman with largely west African and European ancestry. And some of us noted that we could go deeper, using <a href="http://www.1000genomes.org/">public data</a> to guess which variants were attached to which others in Lacks&#8217;s chromosomes, ultimately clarifying which parts of the world particular stretches of her genome trace to, and even which particular people are her close cousins in particular parts of the genome.</p>
<p>Moreover, the paper&#8217;s authors soon acknowledged that well studied variants that HeLa shares with other human genomes can let us guess at particular traits that Lacks may have had, and at various disease risks that she might have faced, had she not died of cervical cancer at thirty-one. But the bottom line on this front is that there&#8217;s still much we <em>can&#8217;t</em> guess from looking at one genome &#8212; even the <em>best</em> sequenced one. As more people&#8217;s genomes and traits are surveyed over time, however, the HeLa genome will indeed reveal deeper insights about who Lacks was. Facets of her uniqueness &#8212; voice and gait, quirks, ailments, and interests &#8212; may someday be traceable, in part, to distinctive genomic spellings that for now hide mum in plain sight.</p>
<p>Third, the technical question of what we can learn raises thornier <em>policy</em> questions about data privacy, ownership, publishing, and consent. And here, perhaps, likening HeLa&#8217;s genome to a remixed or cubist rendition of Lacks&#8217;s was apt. A person&#8217;s DNA uniquely quilts together genome segments shared with others (kin, both near and far), peppered with a few brand new variants (by mutation). As such, it&#8217;s neither fully original, nor fully derivative &#8212; like a work of art, necessarily influenced by broader culture. And, perhaps notably, tough questions about sampling and provenance (<a href="http://en.wikipedia.org/wiki/Elgin_Marbles">Elgin marbles</a>, anyone?) have long raged in the art world, with few clear answers. As such, prepare for a vocal, but likely indefinite societal conversation on that front.</p>
<p>[^1] The paper itself made this clear by showing, from HeLa’s genome, that Lacks was indeed a woman with largely west African and European ancestry. Some of us noted that we could go deeper, using public data to guess which variants were attached to which others in Lacks’s chromosomes, clarifying which parts of the world particular stretches of her genome trace to, and even which particular people are her close cousins in particular parts of the genome.</p>
<p>Moreover, some common variants that HeLa shares with other human genomes let us guess at traits that Lacks may have had, and at various disease risks that she might have faced, had she not died of cervical cancer at thirty-one.</p>
<p>[^2] In this sense, the HeLa genome controversy echoes recent debate over <a href="http://www.genome.gov/10005107">ENCODE</a>, an effort to survey which parts of our genomes govern how much of each protein gets made, where, and when. The ENCODE controversy has hinged, in part, on what the word <em>functional</em> means; for HeLa, the key question is what it means to &#8216;learn nothing&#8217; about a genome or person.</p>
<p>Note here that the HeLa authors might likewise have gone out of their way to tell us that we can learn &#8216;nothing about cell lines&#8217; from the HeLa genome. After all, it&#8217;s -possible- (though astoundingly unlikely) that the whole HeLa genome &#8212; triploid chromosomes and all &#8212; is exactly the same genome, letter for letter, that Lacks was born with.</p>
<p>[^3] While HeLa&#8217;s genome long intrigued me and others, several technical, funding, and strategic hurdles thwarted the idea back then. And here it bears note, at risk of sanctimony, that our brainstorming on HeLa was <em>always</em> premised on the blessing and direct participation of the Lacks family; had the project been green-lighted internally, step one would have been reaching out to them. So when I learned that the authors of the HeLa paper hadn&#8217;t engaged the Lackses, my heart, like others, sank&#8230;and when I then read the authors&#8217; original FAQ, which protested (too much) that their data said nothing about Lacks herself, my gorge rose, prompting this post.</p>
<hr />
<h2>Update (7 August 2013):</h2>
<p>Lacks&#8217;s descendants will <a href="http://www.nature.com/news/deal-done-over-hela-cell-line-1.13511">reportedly</a> anchor a new committee to consider, case-by-case, requests to use HeLa genome data in biomedical research. Brokered by the family, Skloot, <a href="https://en.wikipedia.org/wiki/Francis_Collins">Francis Collins</a>, and others, the arrangement won&#8217;t make the HeLa data fully open (so most useful), keeping it instead in NIH&#8217;s credential-walled <a href="http://www.ncbi.nlm.nih.gov/gap">dbGAP</a>. Nonetheless, it&#8217;s a gracious and scientifically substantive gesture by the family, whose preference understandably trumps other factors.</p>
<p>HeLa was, after all, derived surreptitiously from their own foremother (and, if all her descendants agreed, no one else&#8217;s) without informed consent. That wrong, along with the decades more of scientific ill-treatment of the Lackses that followed, remain the heart of the matter, even if the HeLa saga also highlights</p>
<ul>
<li>broader dilemmas around publishing genome data from explicitly named people, given that doing so makes it much easier to guess who else (their identifiable kin) likely carries many of their distinctive genetic spellings too.</li>
<li>more hearteningly, the potential benefits of directly engaging layfolk as partners in interpreting data they give for societal benefit</li>
</ul>
<p>For now, here&#8217;s to new discoveries that the soon-to-be accessible data will unlock, including those about Lacks herself. Happily that&#8217;s starting already: the data access plan itself was announced alongside a new <a href="http://www.nature.com/nature/journal/v500/n7461/full/nature12064.html">paper</a> dissecting the HeLa genome in unprecedented detail &#8212; notably, with phased haplotypes that should, as noted above, help us better guess which variants Lacks was indeed born with, and which arose only in the cells that became her tumor.</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2013/04/01/the-shock-of-the-new-what-we-can-say-about-helas-novel-variants/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Genodicy: sequencing, in pain but in vain, to understand violence</title>
		<link>http://genomena.com/2013/03/15/genodicy/</link>
		<comments>http://genomena.com/2013/03/15/genodicy/#respond</comments>
		<pubDate>Fri, 15 Mar 2013 12:34:29 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=34</guid>
		<description><![CDATA[Aftermath of Hurricane Sandy, Massachusetts (Image copyright Nathaniel Pearson) A journalist asked me last week to comment on reported plans to sequence &#8212; and ostensibly...]]></description>
				<content:encoded><![CDATA[<p><img src="/wp-content/uploads/2013/03/portal-3-1024x554.jpg" alt="" title="portal-3" /></p>
<p><strong>Aftermath of Hurricane Sandy, Massachusetts</strong> (Image copyright Nathaniel Pearson)</p>
<p>A journalist asked me last week to comment on <a href="http://www.nytimes.com/2012/12/25/science/scientists-to-seek-clues-to-violence-in-genome-of-gunman-in-newtown-conn.html">reported plans</a> to sequence &#8212; and ostensibly interpret &#8212; the genome of Adam Lanza, who killed 26 people in a Connecticut primary school last December.</p>
<p>When news of the pending Lanza sequencing originally hit, I was overseas with (thankfully) no internet access, so missed the ensuing <a href="https://twitter.com/search?q=lanza%20genome&amp;src=typd">chatter</a>. In particular, <a href="http://www.nature.com/news/no-easy-answer-1.12157">Nature</a> eloquently explained how such anecdotal genomics (<em>idiomics?</em>) soft-pedals the causal complexity of behavior &#8212; and could even risk stigmatizing people who share distinctive genetic variants, or phenotypes, with an infamous sequencee like Lanza.</p>
<p>While the ground here is thus well trod, please indulge a few more thoughts on the matter, starting with what I told the journalist:</p>
<blockquote><p>
  We already know, hands down, the best genetic predictor of human violence: it&#8217;s having a Y chromosome. And it&#8217;s an awful predictor; sensibly, we don&#8217;t lock people up for being male.</p>
<p>  Going further, to learn how <em>other</em> genetic variation helps govern behavior, will mean carefully studying <em>many</em> people&#8217;s genomes, and their traits and habits, over time.</p>
<p>  Only by doing so can we untangle how commoner, lifelong mental traits – some deemed normal, others not – interact with each other, and with social and other environmental factors, to shape our actions.</p>
<p>  That work will be soberingly tough. But however anxious we are to understand and prevent baffling acts like Lanza&#8217;s, we simply can&#8217;t make sense of one brutal outburst by looking at one genome.</p>
<p>  Thus the proposed study, however well meant, will likely be sadly futile. Lanza&#8217;s genome might be usefully interpreted only in a broader effort to first understand the roots not of mass murder, but of commoner, more readily studied mental traits – particularly, any well defined illness that he, like many other people who can also be sequenced, may have had.
</p></blockquote>
<p>Journalists live on red meat, so I tried to keep the foregoing short &#8212; by my windbag standards at least. Nonetheless, my comments will likely be whittled to a sentence or two, if published at all. But here I can pass them on in full, and even elaborate a bit.</p>
<p>It&#8217;s worth stressing, for example, that mental and behavioral problems we&#8217;ve already studied via many genomes &#8212; think depression, schizophrenia, autism, and such &#8212; have often proved, well, maddeningly hard to dissect. I&#8217;ll leave it to more informed voices to tell us which such phenotypes might, in turn, help predict otherwise apparently random violence (naively, is social withdrawal relevant?). But if any of them do, their own causal complexity likely only compounds the challenge of prophesying murder from the cradle.</p>
<p>We could likewise mull other questions hinted at in the quote about the project. Do such studies not only mislead, but risk eroding lay support for science? (Likely yes.) Does studying violence biologically <em>excuse</em> bigger factors like guns? (Not if done right &#8212; after all, we don&#8217;t grumble that sequencing lung tumors hampers anti-smoking efforts.) Can we learn anything about human violence by also considering not just environmental factors, but evolutionary insights too? (Likely yes, though beware pat answers (like this one).)</p>
<p>It&#8217;s the simple but telling Y chromosome point, however, that most bears fleshing out here. Namely, murderers indeed tend to be male; but murder is so much rarer than maleness that the predictive arrow really points upstream of where we&#8217;d like. That is, being a murderer strongly predicts having a Y chromosome &#8212; not vice-versa. Genomic <em>utility</em>? Add a failing <em>f</em> to the latter.</p>
<p>And that odds-making asymmetry dogs other, less fraught genetic insights too. Take a <a href="http://snpedia.com/index.php/ACTN3">well known dimorphism</a> in the length of <a href="http://en.wikipedia.org/wiki/ACTN3">alpha-actinin-3</a>, the protein encoded by our <em>ACTN3</em> gene. As first <a href="http://www.ncbi.nlm.nih.gov/pubmed/12879365">reported</a> by <a href="http://www.macarthurlab.org/">Daniel Macarthur</a> and colleagues in 2003, and affirmed in subsequent work by others, nearly every elite sprinter carries at least one copy of <em>ACTN3</em> that encodes a full-length version of the protein. But so, it turns out, do <em>most</em> people (yours truly excepted)! Yet how many of us are elite sprinters?</p>
<p>As such, <em>ACTN3</em> genotype hardly predicts medals. Rather, <em>medals strongly predict ACTN3 genotype</em>.</p>
<p>Picking a child&#8217;s afterschool sports via 23andme results thus might not be abusively cruel, but it&#8217;d certainly be foolish &#8212; and not just statistically. Given the vast, rich space of human personality and endeavor, and the scarcely scratched complexity of our genomes, pigeonholing someone by weak genomic associations understood today &#8212; or those yet to emerge from the murk of behavioral genomics &#8212; risks clipping otherwise powerful wings of individual drive, ability, and fulfillment.</p>
<p>And such questions of upbringing and personal complexity return us to Lanza&#8217;s genome &#8212; ultimately, to the question of what went wrong in his life, genetically or otherwise, that led him to brutally cut short so many others&#8217;. Sadly, sequencing his genome just grasps at <a href="http://www.wordnik.com/words/sideromancy">causal straw</a>, straining to trace a terrible act to some simple, reassuringly inevitable nugget.</p>
<p>If that wish echoes <a href="http://en.wikipedia.org/wiki/Theodicy">theodicy</a> in modern biological terms, we can at least take heart that our genomes, like no god, are real &#8212; so may eventually help better explain our brutality to each other. Because we work in <em>testably</em> mysterious ways, we can indeed hope to learn more about the genetic underpinnings of behavior than that maleness matters. But those better answers will take time &#8212; and, alas, far more than one genome &#8212; to find, to untangle, to absorb.</p>
<p><img src="/wp-content/uploads/2013/03/Leaves-1024x500.jpg" alt="" title="Leaves" /></p>
<p><strong>Leaves after Sandy, Massachusetts</strong> (Image copyright Nathaniel Pearson)</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2013/03/15/genodicy/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Cloning, Epilogue</title>
		<link>http://genomena.com/2009/03/10/cloning-epilogue/</link>
		<comments>http://genomena.com/2009/03/10/cloning-epilogue/#respond</comments>
		<pubDate>Tue, 10 Mar 2009 16:28:01 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=17</guid>
		<description><![CDATA[In an insightful essay (published in an excellent anthology edited by Martha Nussbaum and Cass Sunstein, which also includes other sources cited in this series...]]></description>
				<content:encoded><![CDATA[<p>In an insightful essay (published in an <a href="http://lccn.loc.gov/97051781">excellent anthology</a> edited by Martha Nussbaum and Cass Sunstein, which also includes other sources cited in this series of posts) that juxtaposed doppelgänger folklore to the modern cloning debate, historian <a href="http://divinity.uchicago.edu/faculty/doniger.shtml">Wendy Doniger</a> tellingly observed that &#8216;myths, like vampires, are undead.&#8217; Having met a few myths about asexual reproduction in the foregoing discussion, we can note another recursive parallel: like vampires, myths hate catching sight of themselves as such. But couching public policy debate in terms defined by myths obscures important, rigorously definable utilitarian issues that face humanity as technology advances. Meanwhile, those content to hurl brimstone at the prospect of human cloning, in particular, should realize that they risk soon stigmatizing real people. As in the case of so-called <em>illegitimate</em> children (and there are synonyms with yet harsher connotations) for centuries, such stigma may affront individual dignity far more than may any particular means of birth. So welcome to the world, Eve &#8212; when/where/whoever you are. May you grow old in joy, well-loved, with respect for your uniqueness and privacy, and unswayed by hokum. And if so, let&#8217;s hope there will be more of us like you.</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2009/03/10/cloning-epilogue/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The new emperor&#8217;s clones: Obama&#8217;s refreshing but tepid embrace of reasoned sci/tech policy</title>
		<link>http://genomena.com/2009/03/10/the-new-emperors-clones-obamas-refreshing-but-tepid-embrace-of-reasoned-scitech-policy/</link>
		<comments>http://genomena.com/2009/03/10/the-new-emperors-clones-obamas-refreshing-but-tepid-embrace-of-reasoned-scitech-policy/#respond</comments>
		<pubDate>Tue, 10 Mar 2009 09:23:15 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=61</guid>
		<description><![CDATA[On 9 March, President Obama issued an executive order to lift hobbling restrictions on embryonic stem cell research imposed by his predecessor, George Bush. In...]]></description>
				<content:encoded><![CDATA[<p>On 9 March, President Obama <a href="http://www.nytimes.com/2009/03/10/us/politics/10stem.html">issued an executive order</a> to lift hobbling restrictions on embryonic stem cell research imposed by his predecessor, George Bush. In its own right, Obama&#8217;s order enacts a substantive and welcome change, jumpstarting an exciting and clinically promising line of biomedical research. But the revival of stem cell research can also be seen as part of a larger, ongoing effort to strip away religiously motivated fetters placed on many fronts of American science by the Bush administration.</p>
<p>Sadly, despite some real progress, these fetters continue to ominously hinder the American public discourse on science &#8212; even on occasions, such as the the new executive order on stem cell research, that should be clear victories for empiricism. Like other reversals of Bush policy that have reinstated sound, empirically driven policy on such matters as greenhouse gas regulation, the stem cell order conveys a clear grasp of the societal value of the ever skeptical mode of systematic inquiry that we call science. Yet, in remarks at the signing ceremony, Obama was careful to solemnly profess his &#8216;faith&#8217;. In doing so, he likely meant to console opponents of stem cell research, many of whom are religious. Reaching out to opponents is a laudable goal, and one that Obama consciously cultivates in his role as a &#8216;uniter&#8217;. Yet it is frustrating that, in a national &#8216;teachable moment&#8217;, Obama could apparently find no better way to reach out to do so than by emphasizing a pat solidarity rooted in shared baseless beliefs.</p>
<p>And that wasn&#8217;t the only way in which, even while redressing another case of scientific sabotage by his predecessor, Obama shrank from fully championing a reasoned approach to policymaking. As a sign of specific future policy intentions, Obama&#8217;s latest profession of faith may have been less telling than his use of the signing ceremony to harshly, but emptily, condemn the prospect of <a href="http://www.geneticsandsociety.org/article.php?id=282">human reproductive cloning</a>. Speaking with customary force, but without laying out a chain of reasoning such as we&#8217;ve come to expect from him, he declared that &#8216;[cloning] is dangerous, profoundly wrong and has no place in our society or any society.&#8217; Obama&#8217;s mention of cloning as a foil to stem cell research reinforced a pattern that has emerged in American politics, in which the two issues seem to have become joined at the hip, appearing together with the rote predictability of a pantomime hero and villain. This rhetorical conflation (which is nearly always resolved with some dramatic flourish of contrast) may serve mainly to let supporters of stem cell research distance themselves from an ostensibly obvious evil, thereby reassuring us of their credentials as decent members of the human race. But if stem cell research is right, must reproductive cloning be wrong? Perhaps this moment &#8212; punctuated, as it is, by a major change in federal biomedical research policy &#8212; is a good one for reviewing the question. <a href="http://genomena.com/reproductive-technology-article/cloning-part-one-the-saga-of-bill-s812/">Below</a> is an attempt, in several parts, to do just that.</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2009/03/10/the-new-emperors-clones-obamas-refreshing-but-tepid-embrace-of-reasoned-scitech-policy/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Cloning, part one: The ballad of bill S.812</title>
		<link>http://genomena.com/2009/03/10/cloning-part-one-the-ballad-of-bill-s-812/</link>
		<comments>http://genomena.com/2009/03/10/cloning-part-one-the-ballad-of-bill-s-812/#respond</comments>
		<pubDate>Tue, 10 Mar 2009 08:42:44 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=23</guid>
		<description><![CDATA[Two years before Obama&#8217;s executive order to rescind limits on stem cell research, Senator Orrin Hatch introduced bill S.812, the &#8216;Human Cloning Ban and Stem...]]></description>
				<content:encoded><![CDATA[<p>Two years before Obama&#8217;s executive order to rescind limits on stem cell research, Senator Orrin Hatch introduced <a href="http://www.govtrack.us/congress/billtext.xpd?bill=s110-812">bill S.812</a>, the &#8216;Human Cloning Ban and Stem Cell Research Protection Act of 2007&#8217;. Though unmistakably right-wing, the veteran Hatch is renowned for shrewdly wielding the political olive branch (he famously joined Bono&#8217;s well-heeled campaign for foreign aid debt forgiveness; recorded a hearty <a href="http://www.newyorker.com/talk/2008/08/11/080811ta_talk_widdicombe">get well song</a> for Ted Kennedy; and stood up fairly early in support of Barack Obama&#8217;s embattled attorney general nominee, Eric Holder). It was no surprise, then, that Hatch front-loaded bill S.812 with a title aimed squarely at what has become known as &#8216;purple state&#8217; America. In admirably forthright wording, Hatch&#8217;s title evoked a sense of sober, considered balance: one could picture Justice herself, pausing for a moment&#8217;s due consideration before lifting a brightly glowing vial (stem cells) in one hand and discarding a soiled diaper (human cloning) with the other. In 2007, as now, <a href="http://www.pollingreport.com/science.htm">American adults</a>, eager for a therapeutic boon, tended to favor stem cell research; the same electorate, however, strongly opposed reproductive cloning &#8212; and, sadly, remained largely oblivious to key differences among stem cell lines that stood to greatly limit the pace and scope of their therapeutic payoff.</p>
<p>In particular, much of the public appeared unaware that some stem cells &#8212; the most developmentally versatile ones &#8212; are culled from young embryos (only those slated to be discarded), while others are collected later (usually after birth). Seen in this light, Hatch&#8217;s title, with its reference only to &#8216;stem cell research protection&#8217;, carried a deft ambiguity. In public rhetoric, Hatch and other Republican leaders often silently, but pointedly, exploit the distinction between embryonic and non-embryonic stem cells: their insistence that they are &#8216;pro-stem cell research&#8217; is asterisked with a qualification &#8212; only grudgingly voiced to anyone beyond their religious-right &#8216;base&#8217; &#8212; that they mean &#8216;adult&#8217; stem cells only.</p>
<p>But the invocation of &#8216;stem cell research&#8217; in the title of bill S.812 turned out to be even more strikingly devious, once one actually read beyond the title. In the body of the bill, the term &#8216;stem cell&#8217; appeared exactly twice: once, in a recursive mention of the bill&#8217;s title, and once in a boilerplate section defining &#8216;unfertilized <a href="http://en.wikipedia.org/wiki/Blastocyst">blastocyst</a>&#8216; &#8212; which, as the definition rightly noted, is not a stem cell (nor is it a golf ball, as long as we&#8217;re keeping track&#8230;). Perhaps other references to stem cells were stripped from Hatch&#8217;s bill during last-minute backroom revision, with someone forgetting to change the title. Or maybe the reference to &#8216;stem cell research protection&#8217; was to be read as an oblique threat, conditioning future Republican cooperation on any actually substantive stem cell research initiative to passage of S.812 (much as a mafioso might profess interest in the protection of kneecaps). In any case, the body of the bill did indeed mention &#8216;cloning&#8217; (28 times); indeed, every substantive clause aimed squarely to ban attempts &#8212; therapeutic or otherwise &#8212; to make a human embryo by cloning. And in this sentiment, bill S.812 was far from new.</p>
<p>Since 1998, the Senate had already seen at least three major motions to ban human cloning, each mirrored by a sibling bill in the House of Representatives. Each time, the House bill passed, but enough senators balked at the proposed ban to give a reprieve to the apparently non-existent technique of reproductive cloning. The round of legislative fuss over reproductive technology that preceded Hatch&#8217;s 2007 bill culminated in the 2003 passage of a House <a href="www.house.gov/hensarling/rsc/doc/Lb22703.pdf">bill</a> banning cloning. Right around that time, a tellingly relevant nugget of news went largely overlooked here in the US. It was a spat that played out in the British press; the words were lively enough, but it lacked in visual fireworks, so was hardly tabloid grist, and hardly registered as a blip in the American press. Looking back, though, one might say that the affair laid bare a clash of ideas at the core of our society&#8217;s debate over technology itself.</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2009/03/10/cloning-part-one-the-ballad-of-bill-s-812/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Cloning, part two: Hearsay, hearsay!</title>
		<link>http://genomena.com/2009/03/10/cloning-part-two-hearsay-hearsay/</link>
		<comments>http://genomena.com/2009/03/10/cloning-part-two-hearsay-hearsay/#respond</comments>
		<pubDate>Tue, 10 Mar 2009 08:35:36 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=30</guid>
		<description><![CDATA[The trouble started back in December 2002, when the BBC aired a documentary film that openly probed the tenets of a popular sect. The day...]]></description>
				<content:encoded><![CDATA[<p>The <a href="http://news.bbc.co.uk/1/hi/uk/2598357.stm">trouble</a> started back in December 2002, when the BBC aired a documentary film that openly probed the tenets of a popular sect. The day after the broadcast, Crispian Hollis, a cleric who worked for the sect in question, denounced the British state broadcasters for airing a film that dared to publicly challenge his beliefs. Hollis alleged that the filmmakers had indulged in &#8216;unfounded guesswork&#8217; &#8212; and he fumed, sensibly enough, that passing off any such guesswork as documentary fact is &#8216;not only unscholarly but runs the risk of undermining [journalistic] integrity.&#8217;</p>
<p>But closer inspection of the offending broadcast suggested that Hollis&#8217;s indignation might have been misplaced. In discussing their subject matter &#8212; the dogma of the sect in question &#8212; the filmmakers had actually made it a point to favor definitive evidence over unfounded hearsay. What sort of hearsay had they tried to get to the bottom of? It turns out that Hollis and his fellow clerics swear:</p>
<ul>
<li>that there is an invisible male ape watching over all of earth, in great detail, at all times (presumably never sleeping).</li>
<li>that this ape parses all human conversations (and even unspoken thoughts), in all languages, simultaneously.</li>
<li>and that the ape in question has been doing all this since being born &#8212; <em>asexually</em> &#8212; more than two millenia ago.</li>
</ul>
<p>So what evidence is there to support these claims? Sadly, the BBC filmmakers were apparently unable to reach the ape himself for comment, and the leaders of Hollis&#8217; sect insist that their claims rest on eye-witness accounts written by several men who interacted with the ape early in his life, when he was living in southwest Asia (rather than, say, in the sky). Though varying in some details, the cited accounts all portray the ape as readily visible, susceptible to sleep, and not &#8212; to the outside eye, at least &#8212; especially interested in, or even aware of, the everyday life of anyone in, say, Guatemala or Korea (to name two places where the sect in question has become particularly popular).</p>
<p>As hearsay goes, Hollis deserves points for boldness. In the annals of medicine and peer-reviewed research, there is little, if anything, that could rival his sect&#8217;s claims. Let&#8217;s take the two thousand year-plus lifespan detail, for example. That would be roughly eight-fold overripe even for a Galápagos tortoise, thought by most biologists to be the longest lived animal with eyes. Which brings us to the question of the ape&#8217;s remarkable worldwide vision. Most primates do indeed have quite good eyesight; nonetheless, watching the whole surface of a ball-shaped planet at once would seem to pose problems for the architecture typical of paired vertebrate eyes.</p>
<p>And then there&#8217;s the asexual birth bit. Some birds (turkeys, for example) may <a href="http://www.ncbi.nlm.nih.gov/pubmed/7163099?ordinalpos=4&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum">do it</a>. And, indeed, bees <a href="http://en.wikipedia.org/wiki/Haplodiploidy">do it</a>. But, so far, there&#8217;s no clear evidence of asexual reproduction in any wild (i.e., non-laboratory) mammal. Moreover, even if a female mammal <em>could</em> bear asexually conceived young, she would have to pull off a pretty neat trick to make a son, in particular: one of her X chromosomes (or some other part of the genome) would have to mutate, wholesale, into a Y chromosome, in order to masculinize the developing fetus. Birds &#8212; lucky them &#8212; don&#8217;t have this problem, as it is their females, not males, who have two highly distinct sex chromosomes like our X and Y.</p>
<p>So it seems understandable that the foregoing claims by Crispian Hollis&#8217;s sect might provoke some skepticism. But if we&#8217;re to gauge the boldness of a claim by the degree of hand-wringing among its makers, then it is the <em>apeness</em> of the ape that really stands out. When <a href="http://www.cin.org/users/james/files/message.htm">formally endorsed</a> by the sect in 1996 &#8212; ostensibly to square its dogma with a vast weight of empirical evidence on the matter &#8212; the fact that humans are apes was officially fudged with an &#8216;ontological leap&#8217;. This was apparently done largely to quell horror, among the sect&#8217;s adherents, at <em>being</em> (versus merely <em>descending from</em>) animals. Hollis&#8217; all-seeing sky ape, being a man (if an unusual one, in having been born asexually), was thus separated from all non-human apes by some sort of unprecedented evolutionary jump &#8212; a qualitative material difference that the sect, conveniently, has declined to try to characterize in any concrete and testable terms.</p>
<p>Here it&#8217;s worth noting that such careful dogma-pruning has long been used to help priesthoods stay afloat amid occasional tidal waves of new human knowledge that threaten to obviate pulpits themselves, along with fossilized bronze-age cosmologies. Hollis&#8217;s sect, for example, had already conceded, centuries earlier, that the earth revolves around the sun, rather than vice versa, despite the absence of this basic fact in the sacred texts that the sect typically relied on. But theological revision has limits &#8212; and when further tweaking would appear to gravely threaten priestly interests, smart clerics throw a trump card familiar to all parents (virgin or not): believe because we say so. And so many a sect, including Hollis&#8217; own, extols as virtue the belief in some ever-dwindling suite of empirically groundless claims, coddling the willful ignorance that is &#8216;faith&#8217;.</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2009/03/10/cloning-part-two-hearsay-hearsay/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Cloning, part three: Eve</title>
		<link>http://genomena.com/2009/03/10/cloning-part-three-eve/</link>
		<comments>http://genomena.com/2009/03/10/cloning-part-three-eve/#respond</comments>
		<pubDate>Tue, 10 Mar 2009 08:33:53 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=27</guid>
		<description><![CDATA[The same winter that saw controversy over the BBC documentary on Crispian Hollis&#8217;s sect, another claim was foisted on the world by another sect, who...]]></description>
				<content:encoded><![CDATA[<p>The same winter that saw controversy over the BBC documentary on Crispian Hollis&#8217;s sect, another claim was foisted on the world by another sect, who likewise urged others to believe their claims without evidence. On 26 December 2002, the day after a birthday celebrated for Hollis&#8217; allegedly asexually born boy, along came an asexually born girl. Or so said a priest in a newer, far less popular sect. Flush with a potent mix of giddiness and apparent relief, chemist-cum-cleric-cum-CEO <a href="http://en.wikipedia.org/wiki/Brigitte_Boisselier">Brigitte Boisselier</a> crowed to journalists that she had overseen the birth of a healthy girl who, if true to billing, was the first primate cloned from a somatic (i.e., non-germline) cell.</p>
<p>Pending so-called &#8216;independent&#8217; confirmation (which never came), Boisselier declared that day that the world had only a week to plausibly call her a fraud. And so began an intense, if brief, inquisition (perhaps enough to do the BBC proud, but Torquemada might have been unimpressed). Pressed for proof of her cloning claim, Boisselier retorted that any such evidence she might offer would be dismissed out of hand, presumably due to her priestly subjectivity and/or direct financial stake in the matter.</p>
<p>The rest of Boisselier&#8217;s sect put up a stiff front of secrecy regarding their claims, too, and a safe consensus emerged among pundits: Boisselier&#8217;s announcement had been a lie, serving as a commercial publicity stunt for her and her colleagues. And, indeed, in the several years since the birth announcement, the only major nugget of followup detail that the sect has offered publicly is that the asexually born child and her mother have, like their forerunners in Hollis&#8217;s sect, allegedly <a href="http://news.bbc.co.uk/2/hi/americas/2707211.stm">found refuge</a> somewhere near Jerusalem.</p>
<p>So what sort of evidence could Boisselier release to made her claim credible? Decisive evidence would comprise DNA sequences, from across the human nuclear genome, amplified in parallel from somatic (i.e., non-egg) cells of the alleged mother and daughter. Only if their genotypes matched each other perfectly in an overwhelming majority of cases (allowing for a few mismatches, for reasons we&#8217;ll get to below), and the derivation of sequences were proper and clear (there&#8217;s the rub, for Boisselier and any anonymity-minded sample donor), could one soundly infer that the donors derived from the same fertilized egg, and were, given their age disparity, most likely clone and clonee.</p>
<p>Absent such evidence, pundits understandably scoffed from the start. Belittling the chance that anyone &#8212; a non-MD to boot &#8212; might have pulled off such a rash stunt, talking heads shook scornfully, and took pains to mull the implications of human reproductive cloning in hypothetical terms only. Other empirically groundless claims of Boisselier&#8217;s sect &#8212; that humans have non-earthling forebears, for example &#8212; became fodder for doubting her truthfulness <em>a priori</em>. Suddenly, outspoken empirical rigor was in refreshing vogue!</p>
<p>And, perhaps miraculously, Crispian Hollis&#8217;s own priesthood called shotgun on the skeptics&#8217; bandwagon. Vatican spokesman Joaquín Navarro-Valls burdened himself to speak for empiricists everywhere, <a href="http://www.cnn.com/2002/HEALTH/12/28/cloning.vatican/index.html">declaring</a> that &#8216;[Boisselier&#8217;s] announcement without any element of proof has already raised the skepticism and the moral condemnation of the greater part of the international scientific community.&#8217; Such newfound skepticism made sense, too; after all, Navarro-Valls&#8217;s and Hollis&#8217;s <em>own</em> asexually bred darling hadn&#8217;t been born yesterday.</p>
<p>Duly steeled against falsehood, the world still awaited real tests of at several human cloning claims; no verdict came, and at least one alleged clonee cited lawsuits as deterring her from further disclosure, fueling <a href="http://www.wired.com/medtech/health/news/2003/01/57095">talk of a charlatan hoax</a>. But whether or not Boisselier, or anyone else, has yet cloned a woman using a skin cell nucleus, a sense of inevitability looms around that endeavor. Skepticism of human cloning claims, however fiercely it persists, may eventually look less like sincere evidentiary concern than like craven denial. Perhaps then, at this pregnant hour in the advent of human reproductive cloning, a broader survey of surrounding myths &#8212; not just those which are two millenia old &#8212; could help us decipher all the frightening scrawl on the wall.</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2009/03/10/cloning-part-three-eve/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Cloning, part four: Our genomes, ourselves</title>
		<link>http://genomena.com/2009/03/10/cloning-part-four-our-genomes-ourselves/</link>
		<comments>http://genomena.com/2009/03/10/cloning-part-four-our-genomes-ourselves/#respond</comments>
		<pubDate>Tue, 10 Mar 2009 08:30:21 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=21</guid>
		<description><![CDATA[Let&#8217;s start with the worn chestnut of identity. Ethicist Robert Williamson has posited &#8216;that there is a personal right, ethically based, to individuality, autonomy, and...]]></description>
				<content:encoded><![CDATA[<p>Let&#8217;s start with the worn chestnut of <em>identity</em>. Ethicist Robert Williamson has <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10226911">posited</a> &#8216;that there is a personal right, ethically based, to individuality, autonomy, and identity&#8230;[and that r]eproductive cloning crosses a significant boundary in removing the single most important feature of autonomy: the fact that each of us is genetically unique.&#8217; As is well reported, monozygotic siblings &#8212; so-called <em>identical</em> twins, triplets, &amp;c. &#8212; seem to pose a challenge to the assertion to any such right, which convention would deem a universally inborn attribute (at least within our non-universal little branch of life&#8217;s tree). Williamson skirts this problem, distinguishing violations of rights by &#8216;nature&#8217; from those &#8216;reproduced deliberately by man.&#8217; Ignoring dilemmas raised in trying to justify or even rigorize such a distinction, he retreats still further, exempting monozygotic siblings as a special case because &#8216;even if they closely resemble each other, they do not resemble anyone else.&#8217; Unfortunately for Williamson&#8217;s argument, the same can be said of clones and their clonees (except, say, if one of them happens to have a monozygotic twin, too&#8230;).</p>
<p>But even beyond the problems of defining rights, the identity question is moot. Simply put, <em>clones are not genetically identical</em>. To understand why, first recall an admonition that often pops up in the cloning debate: that environmental factors ensure the uniqueness of individual personality, aptitudes, and other large-scale (read: &#8216;non-genetic&#8217;) traits. If, for example, you rue the hackneyed vision of an army of Michael Jordan clones turning professional basketball into a Midas&#8217;s hell, check the numbers for a real life clone-of-a-sports-superstar. Outfielder <a href="http://www.baseball-reference.com/c/canseoz01.shtml">Osvaldo Canseco</a> hit exactly zero home runs in his major league baseball career; over the same seasons, his injury-prone monozygotic twin <a href="http://www.baseball-reference.com/c/cansejo01.shtml">José</a> hit 73, and swatted 389 more while Ozzie was relegated to the minor leagues or the celebrity impersonator circuit. Incidentally, José has reportedly <a href="http://lccn.loc.gov/2005042834">written</a> that both he and Ozzie used anabolic steroid supplements during their careers; while that particular environmental factor may have helped José considerably, it apparently did little for his twin.</p>
<p>Vagaries of environment turn out to affect more than athletic performance (which, of course, has genetic underpinnings too); they also affect the genome. Because genomes mutate, each potential clonee has a mildly genetically diverse population of cells from which to clone; further mutation during the development of a clone itself nearly assures its genetic uniqueness. Bottom line: assuming plausible rates of cell division and random mutation, fewer than one in ten billion newborn clones would have <em>even one cell</em> genetically identical to the sperm-fertilized egg from which he or she was derived.</p>
<p>Moreover, cloning &#8212; especially for males &#8212; may require eggs whose proteins (which influence gene expression, and, thus, subtleties of development) and non-nuclear genomes (such as those of mitochondria, the rife bacteria which long ago established symbiosis within ancestors of our cells) differ from those of the fertilized egg that became the clonee. A somatic genome transferred into such a <em>tabula non rasa</em> egg has its own pattern of functionally meaningful chemical modification (called <a href="http://en.wikipedia.org/wiki/Genomic_imprinting">genomic imprinting</a>), too, which typically needs thorough overwriting for cloning. More bizarrely still, many living humans are technically <a href="http://www.microchimerism.org/">chimeras</a>, harboring small numbers of somatic cells derived from a mother, child, or even sibling; such cells sometimes cross placental barriers during gestation, engrafting amid much larger populations of cells derived from another zygote. In all the above cases, a clone derived from a single somatic cell represents a truly novel mix of heritable molecular inputs.</p>
<p>All told, the lack of strict genetic identity between clones and clonees voids any claim that cloning violates a right to genetic individuality. More importantly, it gives us more reason to treat clones as unique individuals, rather than to saddle them with overwrought expectations based on their genetic likeness to those we already know. Sadly, Brigitte Boisselier&#8217;s sect announced their own plans for the latter: to try to shoehorn &#8212; wholesale &#8212; the unique personalities and memories of clonees into the uselessly novel brains of their wards. Cloning&#8217;s own technical challenges pale next to those of such a cruel endeavor; and such doubly vain goals alarm those who fear cloning as a means to human eugenics.</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2009/03/10/cloning-part-four-our-genomes-ourselves/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Cloning, part five: Eugenics</title>
		<link>http://genomena.com/2009/03/10/cloning-part-five-eugenics/</link>
		<comments>http://genomena.com/2009/03/10/cloning-part-five-eugenics/#respond</comments>
		<pubDate>Tue, 10 Mar 2009 08:28:20 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genetics, evolution, and policy]]></category>

		<guid isPermaLink="false">http://genomena.com/?p=19</guid>
		<description><![CDATA[But in assessing the fear that cloning will lead to eugenics, note that human mate choice is already eugenic. Lest you doubt, take an extreme...]]></description>
				<content:encoded><![CDATA[<p>But in assessing the fear that cloning will lead to eugenics, note that human mate choice is already eugenic. Lest you doubt, take an extreme example: if female, ask yourself whether, all else being strictly equal, you would rather mate with a kind, five-foot-tall millionaire man with Down syndrome than with a kind, six-foot-tall millionaire man without Down syndrome. If male, ask yourself which man you would rather father your grandchild.</p>
<p>Now, because both adult height and Down syndrome status correlate strongly with genotype, any significant height- or Down status-based mating bias is inherently eugenic (at least in the short term). Human sexual reproduction is no blind crapshoot (ask any sperm banker); our varied, if often predictable, preferences constantly shape the profiles of genetically correlated traits in new adult generations. We&#8217;re picky breeders of tomatoes, dogs, and ourselves &#8212; and human cloning won&#8217;t change that.</p>
<p>In this context, consider a law forbidding infertile or gay people, those with Down syndrome, and/or anyone else with sub-par reproductive prospects, to act to boost those prospects. State-enforced eugenics, no? Indeed, and you have just imagined a cloning ban, which would most affect those short on other reproductive options. &#8216;Eve&#8217; &#8212; mint green apple of Brigitte Boisselier&#8217;s eye &#8212; was allegedly born to a clonee who foresaw little chance to otherwise reproduce; a ban might deem such folks congenitally out of luck. If unconvinced that this constitutes eugenics, note conversely that the spread of cloning, by raising the mean metabolic cost of reproduction (<em>i.e.</em>, one might have to work more to afford clonal than sexual pregnancy) and slowing the potentially adaptive genetic diversification that sex mediates, could make the future human population more vulnerable to environmental change. This outcome is hardly what societal eugenicists claim to support (though their frequent antipathy toward human diversity certainly favors the same risk).</p>
]]></content:encoded>
			<wfw:commentRss>http://genomena.com/2009/03/10/cloning-part-five-eugenics/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
