<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Genomes and health &#8211; Genomena</title>
	<atom:link href="/category/genomes-and-health/feed/" rel="self" type="application/rss+xml" />
	<link>http://genomena.com</link>
	<description>As genomes evolve, so does the world.</description>
	<lastBuildDate>Wed, 01 Feb 2017 02:46:36 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.5.6</generator>
	<item>
		<title>Threesomics</title>
		<link>/2014/04/06/threesomics/</link>
		<comments>/2014/04/06/threesomics/#respond</comments>
		<pubDate>Sun, 06 Apr 2014 11:57:03 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genomes and health]]></category>

		<guid isPermaLink="false">/?p=65</guid>
		<description><![CDATA[Her heart beat in frightened counterpoint to the rhythm of the mitochondrion. &#8212; Madeleine l&#8217;Engle, A Wind in the Door (1973) As you&#8217;ve likely heard,...]]></description>
				<content:encoded><![CDATA[<blockquote><p>
  Her heart beat in frightened counterpoint<br />
  to the rhythm of the mitochondrion.<br />
  &#8212; Madeleine l&#8217;Engle, <em>A Wind in the Door</em> (1973)
</p></blockquote>
<p>As you&#8217;ve likely heard, we&#8217;re doomed. Earth will soon be overrun by babies with three – <em><a href="http://onpoint.wbur.org/2014/02/26/mitochondria-genetically-modified-babies-dna">three</a></em> – biological parents each.</p>
<p>More parents per child will mean, of course, more <a href="http://www.cnn.com/2009/LIVING/02/04/girl.scout.cookie.ethics/index.html">colleagues hawking Girl Scout cookies</a> next year. Facebook friend feeds, along with all organized toddler soccer, will become even less tolerable. If the news pleased anyone, you&#8217;d think it might be people who fret about families with <a href="http://www.linacre.org/clone2.html">too few parents</a>. But it turns out that even they&#8217;re upset, now that children might have <a href="http://archive.catholicherald.co.uk/article/4th-september-2009/3/bioethicists-criticise-procedure-involving-three-p">too <em>many</em></a> genetically invested adults caring about them.</p>
<p>All this because <a href="http://www.scientificamerican.com/article/making-babies-with-3-genetic-parents-gets-fda-hearing/">American</a> and <a href="http://www.reuters.com/article/2014/02/27/us-ivf-3parent-idUSBREA1Q12420140227">British</a> regulators may endorse women replacing faulty <a href="http://wikipedia.org/mitochondria">mitochondria</a> in their eggs, with working ones from other women, in order to have healthy kids. As fertilizing eggs <a href="http://news.bbc.co.uk/2/hi/health/1431489.stm">still</a> requires sperm, such <a href="http://en.wikipedia.org/wiki/Three-parent_baby">mitochondrial rescue</a> really does make kids with three genomic parents. And yes, it willfully and heritably modifies human genomes in doing so.</p>
<p>But while those facts make juicy <a href="http://www.nytimes.com/2014/02/24/opinion/genetically-modified-babies.html">headlines</a>, they&#8217;re actually <a href="http://www.nytimes.com/2001/05/05/health/05DNA.html">old news</a>. For a start, note that <em>every</em> way to have kids (except <a href="/reproductive-technology-article/reproductive-cloning/reproductive-cloning-intro/">cloning</a>) heritably modifies genomes &#8212; and does so, in part, to favor specific traits that parents like (in each other). And among those various ways to breed, Parenting v3, in particular, debuted not last month (or in the future) but in <a href="http://www.thelancet.com/journals/lancet/article/PIIS0140-6736(05)62353-7/fulltext">1997</a>.</p>
<p>Early three-parent outcomes were <a href="http://humupd.oxfordjournals.org/content/7/4/428.long">reviewed</a> in 2001, when the FDA slowed things down in the US, to watch how healthy the first such toddlers stayed over time. Answer? Apparently as <a href="http://well.blogs.nytimes.com/2013/12/16/three-biological-parents-and-a-baby/?_php=true&amp;_type=blogs&amp;_r=0">healthy</a> as any other anecdotally small sample of teenagers. And now, with some of those kids heading to college, regulators are appraising <a href="http://www.fda.gov/downloads/advisorycommittees/committeesmeetingmaterials/bloodvaccinesandotherbiologics/cellulartissueandgenetherapiesadvisorycommittee/ucm385461.pdf">a new version</a> of mitochondrial rescue that cuts one of original method&#8217;s already evidently low risks.[1]</p>
<p>Ok, so three-parent baby stories apparently pop up like cicadas, every ~13 or 17 years, for a new chorus of buzz. With them, us, and regulators all brooding anew, keep a few points in mind:</p>
<h2><em>Every</em> human genome unites many parents.</h2>
<p>If you worry that kids with three parents are somehow an affront to nature, remember that your own genome melds the DNA of not just three, but four other people. You call them grandparents.</p>
<p>Well, actually, your genome comprises DNA from eight people: your great-grand-parents.</p>
<p>Ok, wait, it&#8217;s really sixteen&#8230; You get the point. Formally, mitochondrial rescue (and any other trick to make a <a href="http://wikipedia.org/zygote">zygote</a> from more than two people&#8217;s cells) just speeds up what villages have long done by serially remixing DNA in batch after batch of kids.</p>
<p>You may scoff at likening earlier generations of forebears to parents. But genealogically, the latter just bundle up the former. And here&#8217;s the bottom line: <em>Your average great-great-great-great-great-great-great-great-great-great-grandparent gave you roughly a hundred times more DNA than a mitochondrial donor gives her three-parent child.</em></p>
<p>By that measure,[2] your own birth should have been tabloid news (&#8216;4096 parents!&#8217;). That it wasn&#8217;t so says much less about genetic inheritance than, perhaps, about our tangled obsessions with sex and technology.</p>
<h2>Yes, mitochondria are heritable. Sometimes.</h2>
<p>Future lab methods may let men transmit mitochondria to kids. But for now only women can do so. As such, only about half of kids born today (whether conventionally or by mitochondrial rescue) can pass on mitochondria at all.</p>
<p>So yes, mitochondrial rescue heritably alters genomes, but only for daughters &#8212; and specifically, those who grow up healthy enough to securely have kids of their own. Which circularly underscores the point of effective mitochondrial rescue in the first place.[3]</p>
<p>And more pointedly, old-fashioned two-parent breeding heritably alters genomes too (if that means we should ban sex, well, good luck to us&#8230;).</p>
<h2>Donors and their kids are just like cousins, via time travel.</h2>
<p>A key twist to mitochondrial rescue is that mothers hand down their mitochondrial chromosomes <em>whole</em>, without the <a href="http://en.wikipedia.org/wiki/Genetic_recombination">recombination</a> that, over generations, splits other chromosomes to smithereens. As such, your own ring of mitochondrial DNA is a tiny but sturdy heirloom, inherited &#8212; typically fully intact[4] &#8212; from a long chain of foremothers, the earliest of whom have given you <em>nothing</em> more of their genomes.</p>
<p>Which makes mitochondrial rescue like sneaking several centuries up into your family tree, with some grafting shears, to set up the last such woman&#8217;s father with a different mate. But crucially, and unlike most time travel, this trip brings no (great&#8230;) <a href="http://en.wikipedia.org/wiki/Grandfather_paradox">grandfather paradox</a> for you: beyond swapping your mitochondrial type, it would leave <em>no</em> trace in your genome.[5]</p>
<p>In fact, it would leave you only faintly related &#8212; <em>exactly</em> like a distant <a href="https://www.umanitoba.ca/faculties/arts/anthropology/tutor/fundamentals/m-pline.html">matrilineal</a> cousin &#8212; to your mitochondrial parent. Such gossamer kinship doesn&#8217;t typify parents and children today, but we know it well in adoptive and extended families. And as reproductive technologies advance, we can handle such new kinds of relationships just fine. If you aren&#8217;t horrified by cousins, don&#8217;t lose sleep over three-parent kids.</p>
<h2>For those who most want it, mitochondrial rescue is much <em>safer</em> than conventional sex.</h2>
<p>Methods of mitochondrial rescue carry three main kinds of risk: needle damage to cells; nucleus-cytoplasm incompatibility; and loss of transferred mitochondria. You can read about them in footnotes[6] below, but the evidence suggests that they&#8217;re each small.</p>
<p>As such, if you would ban mitochondrial rescue because it <em>might</em> make sick kids, you really should speak up to ban women with some mitochondrial diseases from having kids at all.[7] Because those born conventionally <em>will</em> be sick.</p>
<p>Mitochondrial rescue aims to give a child, whom a woman has freely decided to bear (a right that most societies recognize), a good chance at a healthy life at all. As such, rescue is simply some people&#8217;s <em>safest</em> way to have a baby &#8212; or to be born.[8]</p>
<h2>This is about mate choice.</h2>
<p>In discussing reproductive health, we often use &#8216;choice&#8217; as a euphemism for abortion. But choice is a broader ideal in family policy, as strides in marriage and adoption law in many countries highlight.</p>
<p>Yes, mitochondrial rescue uses fancy tools to remix genomes, splicing our family trees in ways that might surprise the ancestors involved. But it does so for an age-old need that they&#8217;d readily understand: to let people find compatible mates. That is, having a healthy child means, for some women, picking two mates: one to provide working mitochondria, and another to provide a male-<a href="http://en.wikipedia.org/wiki/Genomic_imprinting">imprinted</a> copy of each nuclear chromosome.</p>
<p>As ever, such mates don&#8217;t have to live together. They are, most simply, adults who agree to mingle DNA &#8212; a choice that we protect in everyday sexual freedom, egg and sperm banking, marriage, and other settings. And in mitochondrial rescue the three people who mate do so specifically for a potential child&#8217;s well-being; if only the same could be said of every conventional pregnancy.</p>
<h2>Methods make real people.</h2>
<p>Barring an apocalypse, reproductive technology will likely go far beyond mitochondrial rescue. We may see kids born with no new mating at all (clones) &#8212; and kids born to sets of <em>many</em> mates (more than just three). We may find ways for people of <em>only</em> one sex to have kids together, by <a href="http://en.wikipedia.org/wiki/Genomic_imprinting">epigenetically modifying</a> chromosomes. And we may start <a href="http://en.wikipedia.org/wiki/CRISPR">editing human genomes</a>, letter by DNA letter.</p>
<p>Such breakthroughs, like any technology, can help and hurt people. Many kids will grow up in loving families who otherwise couldn&#8217;t have had them. But others will suffer from adults&#8217; hubris (&#8216;<em>Editing that <a href="http://wikipedia.org/codon">codon</a> didn&#8217;t do what we expected&#8230;</em>&#8216;), <a href="http://www.thisamericanlife.org/radio-archives/episode/291/reunited-and-it-feels-so-good">unfair demands</a> (&#8216;<em>My clone will be the violinist I could have been&#8230;</em>&#8216;), or even crass neglect (&#8216;<em>Ladies and gentlemen, the 2020 Superbowl Champions&#8217; commemorative child&#8230;&#8217;</em>).</p>
<p>Having kids has always allowed harmful choices, of course. So new ways to breed, giving us often foolish adults more options, stand to steepen some already slippery slopes. But a bigger question will continue to loom: how we treat each new child, regardless of how (s)he came to be. And on that front, fretting about methods that are medically useful, <a href="http://blogs.law.stanford.edu/lawandbiosciences/2014/03/02/heather-has-three-parents/">but socially novel</a>, risks stigmatizing real people born by them (think of prejudices long faced by people born outside conventional bounds of marriage or ethnicity&#8230;).</p>
<p>Moreover, such stigma can, in turn, deepen healthcare inequity, by discouraging insurers from covering needed procedures. Our track record there is mixed: <a href="http://en.wikipedia.org/wiki/In_vitro_fertilisation">IVF</a> has become common, and its beneficiaries may now face little societal prejudice; but in some places it&#8217;s hard to find <a href="http://www.ncsl.org/research/health/insurance-coverage-for-infertility-laws.aspx">insurance for it</a>. As such, mitochondrial rescue may be a good test case of how we welcome people born by methods that tinker ever more deeply with how we&#8217;re made, in order to have healthier kids.</p>
<p>Those three-parent kids are here among us already. They&#8217;ve celebrated many birthdays, and perhaps some quinceañeras and <a href="http://en.wikipedia.org/wiki/Bar_and_Bat_Mitzvah">bnei mitsva</a>, the latter from Hebrew for &#8216;children of a good deed&#8217;. Extra parents may not, of course, mean extra party gifts; but reaching such milestones, in good health, seems reason to celebrate.</p>
<p>A good deed, indeed.</p>
<p><img src="/wp-content/uploads/2014/04/LineOfDescent.jpg" alt="Line of descent, London. (image by Nathaniel Pearson)" /></p>
<p>Line of descent, London. (image by Nathaniel Pearson)</p>
<p>[l] The new methods reduce the chance that any of the original, faulty mitochondria remain in the developing child&#8217;s body.</p>
<p>[2] Some age-old phenomena can let one foreparent contribute even more than usual to your genome: there&#8217;s inbreeding, of course &#8212; but also <a href="http://wikipedia.org/uniparental_disomy">uniparental disomy</a>, where, by fluke of DNA replication in germline or early embryonic cells, both of a child&#8217;s copies of a given chromosome come from the same parent. In extreme such cases, a grandmother can provide more than a third of her grandson&#8217;s genome.</p>
<p>And that leaves aside other ways in which we&#8217;re already walking mixtures of cells from various parents, via maternal-fetal microchimerism, organ donation, and so forth&#8230;</p>
<p>[3] Some women born by rescue may even choose to swap their <em>own</em> kids&#8217; mitochondria &#8212; perhaps even back to an earlier family haplotype, if particular mitochondrial diseases are otherwise readily curable a generation from now.</p>
<p>[4] New mutation, of course, can alter the heirloom as it&#8217;s handed down &#8212; sometimes causing the grave mitochondrial disease that makes mitochondrial rescue sensible.</p>
<p>[5] As such, a genome made by mitochondrial rescue looks much like any other: scanning just your own DNA, we couldn&#8217;t confidently say whether or not you were born by mitochondrial rescue.</p>
<p>By contrast, if we instead replaced one of your long autosomes &#8212; say, a copy of chromosome 1 &#8212; you <em>might</em> look obviously triparental, if your parents&#8217; ancestries varied so much that conventional two-parent breeding couldn&#8217;t plausibly have given you exactly one whole long autosome, but nothing else, from a particular ancestral population</p>
<p>[6] Foreseeable risks in mitochondrial rescue include</p>
<h2>Needle damage</h2>
<p>Tinkering with a cell can, of course, harm membranes, chromosomes, or otherwise components. While it&#8217;s a mechanistic stretch, such damage could in principle affect tissues descended from the originally altered cell, leaving a child sick in ways tracing not to mitochondria themselves, but just to the nano-surgery needed to transplant them.</p>
<p>As in macro-scale surgery, we weigh such procedural risk against likely benefit. And by that measure, mitochondrial rescue works well: so far, kids born by it show no problems attributable to needle damage, and overall tend to be as healthy as typical kids &#8212; which is also, of course, far healthier than they would have otherwise been.</p>
<h2>Incompatibility</h2>
<p>All methods of mitochondrial rescue entail separating an egg&#8217;s long chromosomes from some or all of its <em>cytoplasm</em> (the rest of its cellular guts, including mitochondria with their tiny chromosomes). The idea is to then pool healthy cytoplasm with the long chromosomes of a second, already fertilized egg. In a very loose everyday analogy, picture separating a chicken egg&#8217;s yolk and whites, in order to put whites together with a new yolk.</p>
<p>But even after an egg&#8217;s chromosomes are removed, its cytoplasm reverberates with chemical crosstalk from all of them, including commands to make so-much of specific proteins and other important molecules. When that chatter bathes new chromosomes, of the child-to-be, some of those commands may help the cell thrive (which is the point of mitochondrial transfer). But other commands may, in principle, go awry if they&#8217;re out of sync with other messages in the newly formed cell, or when they&#8217;re parsed through DNA spellings in the new nucleus that differ from those of the chromosomes that sent them.</p>
<p>While we can&#8217;t yet track (let alone understand or modify) all the messages in question, known examples of such problematic messaging are scarce; and their effects would likely be much milder than those of the original mitochondrial disease that rescue served to prevent.</p>
<p>In future, we may start to make zygotes by transferring nuclear chromosomes from cells other than other fertilized eggs (or sperm themselves). In such methods, we&#8217;ll need to carefully steward chemical flags, called epigenetic marks, that govern when particular genome segments are read. Patterns of such flags vary importantly from cell to cell; and while those of eggs complement those of sperm (perhaps in a well matched tug of war), other cells&#8217; marks may need to be overwritten to make a healthy zygote.</p>
<h2>Loss of rescuing mitochondria</h2>
<p>Early methods of mitochondrial rescue mixed cytoplasm from one egg into the cytoplasm of another, to add genetically hardier mitochondria to the zygote&#8217;s existing stock. As cells grow and divide, so do their mitochondria, some of which go into each descendant cell. So a given cell line that starts with both healthy and unhealthy mitochondria might slowly lose the healthy ones &#8212; whether by chance, or by mechanisms that purge newly arrived mitochondria (<em>e.g.</em>, those from sperm) that are replicating out-of-sync with others already there. And such loss of the healthy mitochondria could risk resurgence of the disease that mitochondrial rescue aimed to avoid.</p>
<p>The new methods that may win American and British approval, however, don&#8217;t mix cytoplasm, but instead move <em>nuclei</em> (bags of long chromosomes) into cytoplasm containing only healthy mitochondria, so original unhealthy aren&#8217;t there to resurge in the developing child. As such, the risk of losing the new, healthy mitochondria is minimized.</p>
<p>[7] This would, of course, be a kind of state-enforced eugenics.</p>
<p>[8] Legal arguments about the health of kids who might not otherwise have be born are apparently <a href="https://blogs.law.harvard.edu/billofhealth/2014/02/24/fda-mitochondrial-manipulation-three-parent-children-and-the-ny-times/">nuanced</a>, however.</p>
]]></content:encoded>
			<wfw:commentRss>/2014/04/06/threesomics/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Harmful by any other name: On clinical variant classification</title>
		<link>/2013/05/22/harmful-by-any-other-name-on-clinical-variant-classification/</link>
		<comments>/2013/05/22/harmful-by-any-other-name-on-clinical-variant-classification/#respond</comments>
		<pubDate>Wed, 22 May 2013 06:42:30 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genomes and health]]></category>

		<guid isPermaLink="false">/?p=67</guid>
		<description><![CDATA[In Phoenix this week, clinical geneticists have gathered at ACMG to catch up on health-relevant genomic findings and tools, and decide how to best put...]]></description>
				<content:encoded><![CDATA[<p>In Phoenix this week, clinical geneticists have gathered at <a href="http://www.acmgmeeting.net/acmg2013/public/enter.aspx">ACMG</a> to catch up on health-relevant genomic findings and tools, and decide how to best put them into broad practice. Tonight, in particular, features a workshop on how to classify DNA spelling variants, and report them to patients and caregivers. Chief among questions at hand will be what to <em>call</em> such variants.</p>
<p>Led by the ever-thoughtful <a href="http://www.bizjournals.com/boston/stories/2010/10/11/focus38.html">Heidi Rehm</a>, the workshop is a great chance to define a sensible set of classification bins that best convey how each genetic variant likely affects the health of a person who might carry it.</p>
<p>With an eye to that question, here&#8217;s the most widely used binning scheme, put forth by <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21681106">Kearney <em>et al.</em></a>[1]:</p>
<ul>
<li>Pathogenic</li>
<li>Unknown significance, Likely pathogenic</li>
<li>Unknown significance</li>
<li>Unknown significance, Likely benign</li>
<li>Benign</li>
</ul>
<p>And here, with some explanation to follow below, is an alternative to kick around:</p>
<ul>
<li>Harmful</li>
<li>Likely harmful</li>
<li>Not yet interpretable</li>
<li>Likely harmless</li>
<li>Harmless
<ul>
<li>Family-relevant</li>
<li>Treatment-relevant</li>
</ul>
</li>
</ul>
<p>In weighing these two schemes, first note that Kearney&#8217;s thoughtfully builds on a legacy that&#8217;s deeply familiar to many clinical geneticists &#8212; and has proved a good, if imperfect, rubric for current use. But as we slowly integrate genomic insights into healthcare for ever more people, from birth to old age, Kearney&#8217;s shortcomings may limit its shelf life. Let&#8217;s review.</p>
<h2>What the Kearney scheme gets right</h2>
<p><em>It&#8217;s sensibly simple.</em> Importantly, Kearney (the first scheme above) sorts variants by our best guesses about how they affect health &#8212; and uses a plain word (<em>likely</em>) to neatly distinguish weak guesses from strong ones, without dickering about likelihoods of likelihoods. No arbitrary hair-splitting.</p>
<p><em>It&#8217;s realistically fluid.</em> By making uncertainty explicit, Kearney accomodates changing knowledge. A variant understood poorly today, but well tomorrow, can just be reclassified from the cautiously neutral middle of the rubric toward one of its poles &#8212; or back again.</p>
<p><em>It&#8217;s accurate.</em> Kearney implicitly uses the generalizable term <em>variant</em>, not alternatives like <em>mutation</em> and <em><a href="/concepts-genetics-jargon/polymorphism/">polymorphism</a></em>, whose usages have drifted in misleading ways. In the long run, this can help people understand their genomes accurately &#8212; more on that another time.</p>
<h2>What the Kearney scheme gets wrong</h2>
<p><em>It calls carrier variants pathogenic.</em> Some variants &#8212; <em>dominant</em> ones &#8212; cause disease if found on either of a person&#8217;s copies of a chromosome. But others &#8212; mostly <em>recessive</em> &#8212; do harm only if found on <em>each</em> of that person&#8217;s copies, or along with another variant elsewhere in the genome. Kearney lumps all such variants together, so a man with one copy of a dominant variant, one copy of an autosomal recessive variant, one copy of an X-linked recessive variant, and two copies of an ovarian cancer-causing variant, will misleadingly see all four identically called <em>pathogenic</em> in his report. This problem reflects how such schemes simplistically focus on <em>variants</em>, instead of <em>genotypes</em>. But, as we&#8217;ll see, they could nonetheless clarify which worrisome variants will or won&#8217;t make a given person sick. Moreover, distinguishing such variants can help us know which to watch for in other family members (especially future children, who might get copies from both parents, or who might crucially differ in sex from the parent who gave them a particular variant).</p>
<p><em>It says nothing about drug (or other treatment) response.</em> Beyond calling predictably harmless carrier variants <em>pathogenic,</em> Kearney&#8217;s scheme ignores variants that predict response to drugs and other disease treatments. Notably, such variants are, for now, among the most informative variants in our genomes &#8212; especially for fairly healthy adults (that is, those of us not <em>urgently</em> scanning our genomes for key insight into disease). As is, Kearney would thus glaringly omit key insight from a typical whole-genome report.</p>
<p><em>It&#8217;s unclear.</em> Like much medical jargon (see discussion below), Kearney uses fancy, euphemistic words from Greek and Latin (<em>pathogenic</em> = &#8216;sick-making&#8217;, <em>benign</em> = &#8216;good&#8217;) instead of plainer words understood by more people[3]. Given the bully pulpit that ACMG has at the dawn of genomic medicine, the scheme that it picks will likely shape how people learn about their health for years to come. As such, the jargon-versus-plain-English question is key: Will genome reports coddle <em>doctors</em> by sticking with the latinate jargon they love, or will we use plainer synonyms that <em>patients</em> too understand?</p>
<p><em>It&#8217;s awkward.</em> In the Kearney-inspired parlance of clinical geneticists, the ungainly phrase <em>Variant of Unknown Significance</em> has long stuck out like a Thumb of Unrelenting Soreness on a <a href="http://www.urbandictionary.com/define.php?term=R.O.U.S.">Rodent of Unusual Size</a>. Unlike the other bin names, <em>VUS</em> isn&#8217;t an adjective. Meanwhile, <em>pathogenic</em> and <em>benign</em> are odd antonyms, as the former may sound like it involves germs, and the latter more often opposes <em>malignant</em>, in describing tumors. Such inconsistency &#8212; and the whiffs of germ and cancer talk &#8212; may not help patients understand easily.</p>
<h2>An alternative</h2>
<p>With those shortcomings in mind, let&#8217;s review the newly proposed alternative scheme. While closely resembling Kearney, it arguably refines it in several ways.</p>
<p>First, by using plain words like <em>harmful</em> and <em>harmless</em>, it would be immediately clear to many layfolk, and avoid patronizing euphemism. As such, a fresh step toward empowering patients in the coming era of genomically informed healthcare.</p>
<p>Second, the proposed scheme comprises only adjectives &#8212; mostly short ones &#8212; with poles defined by simple <em>-ful</em> and <em>-less</em> antonyms[4]. The adjective phrase <em>not yet interpretable</em>, while still a bit awkward, plainly conveys both neutrality and openness to change.</p>
<p>Third, it highlights both carrier variants (<em>family-relevant</em>) and <em>treatment-relevant</em> variants in plain ways, accessible to both doctor and patient, that don&#8217;t unduly alarm or reassure. Note that while I&#8217;ve tentatively filed both new classes under <em>harmless</em>, based on their effects for the patient at hand, they don&#8217;t fit neatly there. Note too, that many drug response variants, in particular, may even be <em>helpful –</em> a category generally lumped in under <em>benign/harmless</em> in all these schemes.</p>
<p>At tonight&#8217;s workshop, I may float some of the ideas proposed here. While response may well be lukewarm at best, given the momentum that Kearney-like clinical genetics nomenclature has already gained, there may be no better time to speak up on the matter.</p>
<p>[1] The scheme was first proposed for variants involving long repetitive chunks of the genome, but has now been adopted more broadly.</p>
<p>[2] It&#8217;s worth asking, however, whether those two terms are too hard to uniquely abbreviate (<em>HF</em>? <em>HL</em>?), or would be readily confused in typing, reading, or speaking. If so, could an even shorter term like <em>OK</em> could sub for <em>harmless</em>?</p>
<p>[3] This habit likely dates to when Latin was the key shared language of European scholars – letting healers in, say, England, Italy, and Poland easily share new discoveries with each other.</p>
<p>Today, of course, science&#8217;s <a href="http://en.wikipedia.org/wiki/Lingua_franca">lingua franca</a> is English (and tomorrow perhaps Mandarin&#8230;). But while English taps two big stocks of words for the same things – pithy Germanic (<em>shit, yearly</em>) and flowery Greco-latinate (<em>excrement, annually</em>) – doctors (like lawyers and us scientists) find that continuing to cloak their work in layers of Greco-latinate jargon does three handy things.</p>
<p>First, it discourages DIY (or other) alternatives to their services. (<em>Ah, I have a contusion that may become a hematoma? Wow, thanks, doc – and I thought it was just a bad bruise&#8230;</em>).</p>
<p>Second, it hides knowledge from others – sometimes rightly (eavesdroppers in a hospital lift), sometimes not (an adult patient who wants frank talk, not patronizing euphemism).</p>
<p>Third, it binds them (well, us scientists too) in clubby solidarity, perhaps safeguarding a sense of prestige (<em>As we learned in med school, the reeeal word for right pinky is digitus minimus manus dextra&#8230;</em>).</p>
]]></content:encoded>
			<wfw:commentRss>/2013/05/22/harmful-by-any-other-name-on-clinical-variant-classification/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Harmful by any other name: On clinical variant classification</title>
		<link>/2013/03/22/harmful-by-any-other-name-on-clinical-variant-classification-2/</link>
		<comments>/2013/03/22/harmful-by-any-other-name-on-clinical-variant-classification-2/#respond</comments>
		<pubDate>Fri, 22 Mar 2013 06:42:14 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genomes and health]]></category>

		<guid isPermaLink="false">/?p=70</guid>
		<description><![CDATA[In Phoenix this week, clinical geneticists have gathered at ACMG to catch up on health-relevant genomic findings and tools, and decide how to best put...]]></description>
				<content:encoded><![CDATA[<p>In Phoenix this week, clinical geneticists have gathered at <a href="http://www.acmgmeeting.net/acmg2013/public/enter.aspx">ACMG</a> to catch up on health-relevant genomic findings and tools, and decide how to best put them into broad practice. Tonight, in particular, features a workshop on how to classify DNA spelling variants, and report them to patients and caregivers. Chief among questions at hand will be what to <em>call</em> such variants.</p>
<p>Led by the ever-thoughtful <a href="http://www.bizjournals.com/boston/stories/2010/10/11/focus38.html">Heidi Rehm</a>, the workshop is a great chance to define a sensible set of classification bins that best convey how each genetic variant likely affects the health of a person who might carry it.</p>
<p>With an eye to that question, here&#8217;s the most widely used binning scheme, put forth by <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21681106">Kearney <em>et al.</em></a>[1]:</p>
<ul>
<li>Pathogenic</li>
<li>Unknown significance, Likely pathogenic</li>
<li>Unknown significance</li>
<li>Unknown significance, Likely benign</li>
<li>Benign</li>
</ul>
<p>And here, with some explanation to follow below, is an alternative to kick around:</p>
<ul>
<li>Harmful</li>
<li>Likely harmful</li>
<li>Not yet interpretable</li>
<li>Likely harmless</li>
<li>Harmless
<ul>
<li>Family-relevant</li>
<li>Treatment-relevant</li>
</ul>
</li>
</ul>
<p>In weighing these two schemes, first note that Kearney&#8217;s thoughtfully builds on a legacy that&#8217;s deeply familiar to many clinical geneticists &#8212; and has proved a good, if imperfect, rubric for current use. But as we slowly integrate genomic insights into healthcare for ever more people, from birth to old age, Kearney&#8217;s shortcomings may limit its shelf life. Let&#8217;s review.</p>
<h2>What the Kearney scheme gets right</h2>
<p><em>It&#8217;s sensibly simple.</em> Importantly, Kearney (the first scheme above) sorts variants by our best guesses about how they affect health &#8212; and uses a plain word (<em>likely</em>) to neatly distinguish weak guesses from strong ones, without dickering about likelihoods of likelihoods. No arbitrary hair-splitting.</p>
<p><em>It&#8217;s realistically fluid.</em> By making uncertainty explicit, Kearney accomodates changing knowledge. A variant understood poorly today, but well tomorrow, can just be reclassified from the cautiously neutral middle of the rubric toward one of its poles &#8212; or back again.</p>
<p><em>It&#8217;s accurate.</em> Kearney implicitly uses the generalizable term <em>variant</em>, not alternatives like <em>mutation</em> and <em><a href="/concepts-genetics-jargon/polymorphism/">polymorphism</a></em>, whose usages have drifted in misleading ways. In the long run, this can help people understand their genomes accurately &#8212; more on that another time.</p>
<h2>What the Kearney scheme gets wrong</h2>
<p><em>It calls carrier variants pathogenic.</em> Some variants &#8212; <em>dominant</em> ones &#8212; cause disease if found on either of a person&#8217;s copies of a chromosome. But others &#8212; mostly <em>recessive</em> &#8212; do harm only if found on <em>each</em> of that person&#8217;s copies, or along with another variant elsewhere in the genome. Kearney lumps all such variants together, so a man with one copy of a dominant variant, one copy of an autosomal recessive variant, one copy of an X-linked recessive variant, and two copies of an ovarian cancer-causing variant, will misleadingly see all four identically called <em>pathogenic</em> in his report. This problem reflects how such schemes simplistically focus on <em>variants</em>, instead of <em>genotypes</em>. But, as we&#8217;ll see, they could nonetheless clarify which worrisome variants will or won&#8217;t make a given person sick. Moreover, distinguishing such variants can help us know which to watch for in other family members (especially future children, who might get copies from both parents, or who might crucially differ in sex from the parent who gave them a particular variant).</p>
<p><em>It says nothing about drug (or other treatment) response.</em> Beyond calling predictably harmless carrier variants <em>pathogenic,</em> Kearney&#8217;s scheme ignores variants that predict response to drugs and other disease treatments. Notably, such variants are, for now, among the most informative variants in our genomes &#8212; especially for fairly healthy adults (that is, those of us not <em>urgently</em> scanning our genomes for key insight into disease). As is, Kearney would thus glaringly omit key insight from a typical whole-genome report.</p>
<p><em>It&#8217;s unclear.</em> Like much medical jargon (see discussion below), Kearney uses fancy, euphemistic words from Greek and Latin (<em>pathogenic</em> = &#8216;sick-making&#8217;, <em>benign</em> = &#8216;good&#8217;) instead of plainer words understood by more people[3]. Given the bully pulpit that ACMG has at the dawn of genomic medicine, the scheme that it picks will likely shape how people learn about their health for years to come. As such, the jargon-versus-plain-English question is key: Will genome reports coddle <em>doctors</em> by sticking with the latinate jargon they love, or will we use plainer synonyms that <em>patients</em> too understand?</p>
<p><em>It&#8217;s awkward.</em> In the Kearney-inspired parlance of clinical geneticists, the ungainly phrase <em>Variant of Unknown Significance</em> has long stuck out like a Thumb of Unrelenting Soreness on a <a href="http://www.urbandictionary.com/define.php?term=R.O.U.S.">Rodent of Unusual Size</a>. Unlike the other bin names, <em>VUS</em> isn&#8217;t an adjective. Meanwhile, <em>pathogenic</em> and <em>benign</em> are odd antonyms, as the former may sound like it involves germs, and the latter more often opposes <em>malignant</em>, in describing tumors. Such inconsistency &#8212; and the whiffs of germ and cancer talk &#8212; may not help patients understand easily.</p>
<h2>An alternative</h2>
<p>With those shortcomings in mind, let&#8217;s review the newly proposed alternative scheme. While closely resembling Kearney, it arguably refines it in several ways.</p>
<p>First, by using plain words like <em>harmful</em> and <em>harmless</em>, it would be immediately clear to many layfolk, and avoid patronizing euphemism. As such, a fresh step toward empowering patients in the coming era of genomically informed healthcare.</p>
<p>Second, the proposed scheme comprises only adjectives &#8212; mostly short ones &#8212; with poles defined by simple <em>-ful</em> and <em>-less</em> antonyms[4]. The adjective phrase <em>not yet interpretable</em>, while still a bit awkward, plainly conveys both neutrality and openness to change.</p>
<p>Third, it highlights both carrier variants (<em>family-relevant</em>) and <em>treatment-relevant</em> variants in plain ways, accessible to both doctor and patient, that don&#8217;t unduly alarm or reassure. Note that while I&#8217;ve tentatively filed both new classes under <em>harmless</em>, based on their effects for the patient at hand, they don&#8217;t fit neatly there. Note too, that many drug response variants, in particular, may even be <em>helpful –</em> a category generally lumped in under <em>benign/harmless</em> in all these schemes.</p>
<p>At tonight&#8217;s workshop, I may float some of the ideas proposed here. While response may well be lukewarm at best, given the momentum that Kearney-like clinical genetics nomenclature has already gained, there may be no better time to speak up on the matter.</p>
<p>[1] The scheme was first proposed for variants involving long repetitive chunks of the genome, but has now been adopted more broadly.</p>
<p>[2] It&#8217;s worth asking, however, whether those two terms are too hard to uniquely abbreviate (<em>HF</em>? <em>HL</em>?), or would be readily confused in typing, reading, or speaking. If so, could an even shorter term like <em>OK</em> could sub for <em>harmless</em>?</p>
<p>[3] This habit likely dates to when Latin was the key shared language of European scholars – letting healers in, say, England, Italy, and Poland easily share new discoveries with each other.</p>
<p>Today, of course, science&#8217;s <a href="http://en.wikipedia.org/wiki/Lingua_franca">lingua franca</a> is English (and tomorrow perhaps Mandarin&#8230;). But while English taps two big stocks of words for the same things – pithy Germanic (<em>shit, yearly</em>) and flowery Greco-latinate (<em>excrement, annually</em>) – doctors (like lawyers and us scientists) find that continuing to cloak their work in layers of Greco-latinate jargon does three handy things.</p>
<p>First, it discourages DIY (or other) alternatives to their services. (<em>Ah, I have a contusion that may become a hematoma? Wow, thanks, doc – and I thought it was just a bad bruise&#8230;</em>).</p>
<p>Second, it hides knowledge from others – sometimes rightly (eavesdroppers in a hospital lift), sometimes not (an adult patient who wants frank talk, not patronizing euphemism).</p>
<p>Third, it binds them (well, us scientists too) in clubby solidarity, perhaps safeguarding a sense of prestige (<em>As we learned in med school, the reeeal word for right pinky is digitus minimus manus dextra&#8230;</em>).</p>
]]></content:encoded>
			<wfw:commentRss>/2013/03/22/harmful-by-any-other-name-on-clinical-variant-classification-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Rare variants, disease, and population size</title>
		<link>/2012/05/28/rare-variants-disease-and-population-size/</link>
		<comments>/2012/05/28/rare-variants-disease-and-population-size/#respond</comments>
		<pubDate>Mon, 28 May 2012 15:33:10 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genomes and health]]></category>

		<guid isPermaLink="false">/?p=52</guid>
		<description><![CDATA[Three new papers spotlight a glut of rare variants in our genomes, with key insights for human history and health.]]></description>
				<content:encoded><![CDATA[<p><em>Three new papers spotlight a glut of rare variants in our genomes, with key insights for human history and health.</em></p>
<p><img src="/wp-content/uploads/2012/05/ginkobough-3.jpg" alt="Rare fruit burdens the boughs. Female gingko, Chicago<br /> (image copyright Nathaniel Pearson)&#8221; title=&#8221;ginkobough-3&#8243; /></p>
<p><strong>Rare fruit burdens the boughs.</strong> Female gingko, Chicago (image copyright Nathaniel Pearson)</p>
<h2>Rarity abounds</h2>
<p>Data-rich <a href="http://www.sciencemag.org/content/early/2012/05/21/science.1219240">new</a> <a href="http://www.sciencemag.org/content/early/2012/05/16/science.1217876">papers</a> from teams led by <a href="http://www.gs.washington.edu/faculty/akey.htm">Josh Akey</a> and <a href="http://www.eeb.ucla.edu/Faculty/Novembre/">John Novembre</a>, and a brief <a href="http://www.ncbi.nlm.nih.gov/pubmed/22582263">theory paper</a> from <a href="http://keinanlab.cb.bscb.cornell.edu/">Alon Keinan</a> and the prolific <a href="http://www.sciencemag.org/content/336/6082/740">Andy Clark</a>, highlight a bounty of rare genetic variants in our genomes &#8212; and point out why we should care.</p>
<p>Bolstered by the papers&#8217; data from more than 80 million copies of individual human genes, the growing catalog of such rare variants casts our recent ancestors&#8217; rampant population growth into sharper temporal relief &#8212; and should, in the long run, help finely trace the geographic sojourns of particular copies of human chromosome segments. More importantly, however, many of those rare variants likely figure centrally in our health.</p>
<p>These <a href="http://www.nature.com/ncomms/journal/v1/n8/full/ncomms1130.html">basic</a> <a href="http://www.nature.com/nrg/journal/v11/n6/abs/nrg2779.html">insights</a> have been clear to geneticists for a <a href="http://www.ege.fcen.uba.ar/materias/ecomolecular/Material/Coalescencia/bibliografia/mismatch_distribution.pdf">long</a> <a href="http://www.neurociencias.org.ve/cont-cursos-laboratorio-de-neurociencias-luz/SNPs.pdf">time</a>, and it&#8217;s great to see them percolate through the <a href="http://www.nytimes.com/2012/05/18/science/many-rare-mutations-may-underpin-diseases.html">lay press</a>. The new data papers scoured every letter of many genes in thousands of people, and found a bumper crop of spelling variants that are each found in just one or a few of those people. The third paper summarized what such findings suggest about precisely how big the human population has been over time, and roughly what they mean for efforts to understand disease.</p>
<p>Altogether, the findings cast such bright light on our origins and health because, under simple assumptions[1], geneticists can predict how often variants that do (or don&#8217;t) greatly alter proteins should pop up in a given proportion of people, if our ancestors were steady in number, and if proteins weren&#8217;t especially important for health. And those are two big ifs.</p>
<p>The new data highlight that <em>real</em> patterns of such variant frequencies in our genomes drastically flout those null expectations &#8212; and they call sensible attention to rare variants, which underlie that deviation, as we search for the genetic basis of disease. More specifically, the papers all underscore two broad insights that have been clear for several years:</p>
<ul>
<li>Our population has skyrocketed, but just for the past few millenia &#8212; a trend that&#8217;s left a strong signature of many young, rare spelling variants in our genomes.</p>
</li>
<li>
<p>Many of those rare variants may be making us sick.</p>
</li>
</ul>
<h2>A tippy tree, laden with rare fruit</h2>
<p>The findings in the new papers hinge on a simple insight: the more widely common a genetic variant is, the older it likely is. This is because old variants have typically been carried down many branches of the growing human family tree, spreading far and wide on the planet. By contrast, variants that just arose recently are typically confined to recently sprouted, geographically narrow branches of the tree.</p>
<p>While details of very early human population dynamics are hard to precisely infer[2], the new data, along with much other genetic and ancillary historical evidence (see Keinan and Clark&#8217;s reference citations, for starters), suggest that our population has grown extremely fast in the past few millenia. Such growth has, effectively, stretched the human family tree at its tips: the tree&#8217;s young twigs look longer[3], in units of generations, than we&#8217;d otherwise expect, given how long the trunk and inner branches are. And because new genetic variants pop up roughly randomly (by mutation) on the branches as they grow, the long, fast-growing tips of the tree harbor more of its total load of mutations than they would have, had the tree grown at a constant rate.</p>
<p>You can picture each such mutation as if it were a little brainstorm in the head of the late <a href="http://en.wikipedia.org/wiki/Dr._Seuss">Dr. Seuss</a>. Had Seuss drawn genomic family trees, he might have represented each mutation as an odd, never-before-seen kind of fruit, confined to the branch (big or small, and including its sub-branches) where the mutation struck. Many of the rife rare variants in our genomes can thus be thought of as distinctive fruits, each confined to just one or a few twigs amid a great, bushy tree.</p>
<p>In this light, the new papers affirm what&#8217;s become clear over the past few years, as we <a href="http://en.wikipedia.org/wiki/Whole_genome_sequencing">sequence more and more people&#8217;s whole genomes</a>: we&#8217;ll still be finding new human genetic variants for a long time, even after having sequenced many more of us.[4] And, as long as our population continues to dramatically balloon &#8212; a system out of equilibrium, in population genetic terms &#8212; the tree will continue to loosely resemble an <a href="http://en.wikipedia.org/wiki/Inflation_(cosmology)">inflationary universe</a>, its various branches speeding apart from each other via new mutations. In this analogy, the genetic counterpart of the <a href="http://en.wikipedia.org/wiki/Redshift">red-shift</a> that signals cosmic expansion is, roughly speaking, the overall <a href="http://en.wikipedia.org/wiki/Tajima%27s_D">skew in frequency</a>, toward rarity, of our genetic variants.</p>
<h2>Rare variants in disease</h2>
<p>Visions of the human family tree, tips bent toward our inquisitive grasp by newfound fruit, may recall the myth of <a href="http://www.wikipaintings.org/en/lucas-cranach-the-elder/fall-of-man-1537">another tree</a>. Apt, then, that the crop of rare variants in our genomes may include much of the fruit of human affliction.</p>
<p>Rare variants are thought to figure centrally in disease for two related reasons: as we&#8217;ve seen, most such variants are rare because they arose recently, so haven&#8217;t had time to spread widely among people; <em>and young variants, by definition, haven&#8217;t withstood natural selection for long.</em></p>
<p>Such selection &#8212; often assisted by chance &#8212; tends to keep harmful variants rare, or purge them from the population altogether. Non-harmful rare variants, by contrast, are in principle free to get more common (though chance often strikes them down too).</p>
<p>That is, over time, consistently harmful variants tend to vanish, especially if the population is big enough to stably harbor a rich variety of alternative variants; meanwhile, variants that happen <em>not</em> to harm their carriers are free to spread, whether by chance or, in rare cases, by helping their carriers have more kids than others do.</p>
<p>Together, these trends mean that a snapshot of the rare variants we carry today, like a minute&#8217;s worth of the world&#8217;s newest <a href="http://twitter.com/">tweets</a>, is likely enriched for items that will soon be either gone[5] or, in a few cases, more common.</p>
<p>And they help explain why surveys of the common genetic variants covered by fast, cheap <a href="http://en.wikipedia.org/wiki/SNP_genotyping#SNP_microarrays">SNP chip</a> screens rarely offer clear insight into disease risk. For a given stretch of the genome, such common variants do distinguish big branches of the human family tree from each other, making them quite informative of ancestry. But a consensus has emerged that the long tail of human genetic diversity &#8212; all those rare variants &#8212; is where we&#8217;ll find much of the genetic contribution to disease risk.</p>
<p>Spotting <em>which</em> rare variants harm us, however, turns out to be tough.</p>
<h2>Proof of burden</h2>
<p>Take the extreme case of a variant found in just one woman, among everyone on earth. If we split humanity into those who get a given disease in life, and those who don&#8217;t, our chosen woman must fall into one group or the other. And if we look at enough diseases, she&#8217;ll eventually fall into the sick group for at least one of them.</p>
<p>But it&#8217;s clearly too far a leap to infer that the unique variant she carries made her sick. That is, the variant&#8217;s distribution among people with and without the disease simply <em>can&#8217;t</em> be statistically significant, given how rare it is overall.</p>
<p>To meet this inherent challenge to squarely implicating a given rare variant in a given disease, geneticists look to leverage other insights. If the variant really is too rare to show up on further screening of more sick or healthy people &#8212; and that&#8217;s a place where the <a href="http://evs.gs.washington.edu/">new data</a> are already helping in my own work with researchers seeking to trace disease risks to variation in specific tracts of our genomes &#8212; they next ask how readily it may affect physiology by altering the amount or chemical makeup of a protein encoded by a gene that either harbors the variant itself, or sits near it in the genome.</p>
<p>And, next, they may look at more people with the disease in question, and ask whether <em>other</em> rare variants tend to cluster nearby in their genomes, moreso than other people&#8217;s. In recent years, as richly detailed data on human genetic variation has started to flow, geneticists have been honing <a href="http://genome.sph.umich.edu/wiki/Rare_variant_tests">rare variant burden tests</a> specifically to find such regions. Refining such tests, and gathering more genetic and phenotypic data to feed them, stands to bring many key insights into the genetic basis of disease (and on a time frame shorter, we can certainly hope, than that needed for natural selection itself to weed all those harmful variants from the crop of rare variants we carry!).</p>
<h2>A new drug</h2>
<p>To thoroughly catalog the rare variants that pepper our genomes, of course, we have to read what DNA letters we carry at each site in the genome, rather than just at those sites already known to vary in spelling (as in SNP chips). The newly published work furthers that effort, by carefully sifting through particular sets of genes in many thousands of people &#8212; more people than have ever been so comprehensively sequenced together.</p>
<p>Notably, the Novembre group&#8217;s paper focuses on a few hundred genes already thought to help govern how the body responds to particular drugs. Such genes are actually an intriguing testing ground for the notion that rare variants crucially shape not just disease risk, but other phenotypes (outward traits) too.</p>
<p>Many drugs derive from defense chemicals made by plants and molds &#8212; nature&#8217;s organic chemists extraordinaires &#8212; that our ancestors have long eaten, breathed, and otherwise touched. But modern folks have also tinkered greatly with such drugs, concentrating, combining, and diversifying them in our quest to prevent and cure diseases. As such, many drugs, and cocktails thereof, are (like other facets of our overall <a href="http://en.wikipedia.org/wiki/Thrifty_gene_hypothesis">diets</a>) fairly new parts of the human environment.</p>
<p>Drugs we take are thus exposing even the most <em>common</em> (read: oldest) variants in our genomes to novel regimes of natural selection. Many such drugs work better, at particular doses, in some people than others &#8212; and such variation may often trace largely to variation in our genomes.</p>
<p>Looking ahead, I&#8217;m intrigued to see whether rare genetic variants turn out to explain unusual responses to particular drugs as well (or better) as they explain particular diseases &#8212; or, alternatively, whether such variation in drug response traces largely to common variants in our genomes.</p>
<h2>Tall trees: the diversity skyline</h2>
<p>An intriguing tidbit in the Akey group&#8217;s paper is a spatial contour of overall genetic diversity across thousands of genes in our genomes. Plotting the classic measure of <a href="/concepts-genetics-jargon/polymorphism-2/polymorphism-iii/">nucleotide diversity</a> &#8212; that is, <em>how often two randomly chosen chromosomal copies of a genome site differ in spelling</em> &#8212; Akey&#8217;s post-doc <a href="http://www.gs.washington.edu/academics/postdocs/tennessen.htm">Jacob Tennessen</a> <em>et al.</em> predictably found the strongest peak in diversity in the <a href="http://en.wikipedia.org/wiki/Human_leukocyte_antigen">HLA</a> gene cluster on chromosome 6&#46; Expressed on the surface of immune response cells, these genes work, in large part, to help us fight infection &#8212; a job thought to be well served by great genetic diversity within a genome, which presumedly helps its carrier respond to many kinds of germs.</p>
<p>Byzantine in its sequence variation, HLA turns out to play surprising functional roles in mate choice, drug response, and diseases from <a href="http://www.ncbi.nlm.nih.gov/pubmed/22586495">multiple sclerosis</a> to <a href="http://www.ncbi.nlm.nih.gov/pubmed/17002906">narcolepsy</a>. Notably, women and other female great apes likely <a href="http://www.ncbi.nlm.nih.gov/pubmed/9326314">pick their mates</a> in part (and unconsciously) by how they smell, thanks partly to what versions of HLA they and their suitors carry. Such preferences are thought to help preserve genetic variation longer here than elsewhere in the genome &#8212; so well, in fact, that your copies of some HLA genes more closely resemble some gorillas&#8217; copies than some other people&#8217;s copies&#8230;and those gorillas&#8217; HLA genes are likewise closer to yours than to each others&#8217;!</p>
<p>Essentially, even the <em>inner</em> branches of the family tree of this part of the genome are <a href="http://www.ncbi.nlm.nih.gov/pubmed/8411099">incredibly long</a>, stretching back ten-fold more generations than is typical. As we&#8217;ll see in a coming post, the overall depth of the tree for a given part of the genome can be thought of as a rough proxy for how big the ancestral population for that part of the genome has, over time, tended to be.</p>
<p>Other peaks in genetic diversity &#8212; lower than HLA, but still prominent &#8212; include odorant receptor and keratin (hair/skin protein-making) genes, which are widely presumed to accumulate functionally unimportant variation, reflecting less stringent evolutionary constraint in people than in some other mammals. Strikingly, however, the Akey group also found that another immune response gene, <a href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=DEFB108B"><em>DEFB108B</em></a>, marks a peak in genetic diversity roughly as tall as that of the much better known HLA cluster. It&#8217;ll be intriguing to learn more about what <em>DEFB108B</em> does in our bodies, and whether its remarkable diversity reflects HLA-like importance, or keratin-like dispensability.</p>
<p>Stay tuned on that front. As more of us are sequenced and phenotyped, we&#8217;ll learn much more about which of our variants &#8212; among the common ones, and the newly commonplace rare ones &#8212; matter most, and how. Much of what we learn will speak directly to the pending challenges of genomically personalized medicine, as framed in fervent discussion of another recent <a href="http://stm.sciencemag.org/content/4/133/133ra58.full.pdf">paper</a>, both at <a href="http://stm.sciencemag.org/content/4/135/135lr3">large</a>, and in <a href="/genomes-and-health/on-twins-genomes-and-health/">these</a> <a href="/genomes-and-health/why-cancers-are-the-least-and-most-genetic-of-diseases/">pages</a>.</p>
<p>[1] Back-of-the-envelope estimates typically ignore any complications from non-random mating, variation in mutation rate, and so forth &#8212; but are quite robust.</p>
<p>[2] Moreover, the history of human population change has, of course, varied in space (among regional sub-populations), as well as over time. Notably, the new papers suggest that such variation may be fairly minor in the grand scheme, dwarfed by the remarkable overall recent growth. And Keinan and Clark note that sample sizes, in particular, may add roughly as much noise to the picture as do real underlying variables.</p>
<p>[3] Ultimately, the length of these twigs tracks how long many randomly chosen pairs of extant copies of our chromosomes have descended along separate lines.</p>
<p>[4] In the end, you likely harbor a dozen or so brand new genetic variants that arose by mutation only in you. But you also likely harbor plenty of other very rare variants that, til we sequence your genome, will have never been spotted in anyone else.</p>
<p>[5] Note that this doesn&#8217;t mean that no one with harmful variants has kids &#8212; after all, <em>everyone</em> carries some such variants, and people are breeding just fine. Rather, because a given variant can be inherited independently of other variants in the same genome, and may wreak harm only in combination with another copy of itself (or some other variant), people simply tend to have more kids who inherit more copies of healthier alternative variants than kids who inherit more copies of harmful ones. Moreover, much of the natural selection in question likely happens beyond our view, before pregnancy begins, when unhealthy early embryos fail to implant and thrive in the womb.</p>
]]></content:encoded>
			<wfw:commentRss>/2012/05/28/rare-variants-disease-and-population-size/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Why cancers are the least — and most — genetic of diseases</title>
		<link>/2012/04/19/why-cancers-are-the-least-and-most-genetic-of-diseases/</link>
		<comments>/2012/04/19/why-cancers-are-the-least-and-most-genetic-of-diseases/#respond</comments>
		<pubDate>Thu, 19 Apr 2012 13:28:16 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genomes and health]]></category>

		<guid isPermaLink="false">/?p=14</guid>
		<description><![CDATA[The plot thickens. The crab [&#8230;] only runs backwards. – Arsenio Rodríguez, Cangrejo Fue a Estudiar In time, finer data and statistical models will test...]]></description>
				<content:encoded><![CDATA[<blockquote><p>
  The plot thickens. The crab [&#8230;] only runs backwards.<br />
  – Arsenio Rodríguez, <em>Cangrejo Fue a Estudiar</em>
</p></blockquote>
<p>In time, finer data and statistical models will test the specific claims of a widely discussed recent <a href="http://stm.sciencemag.org/content/early/2012/04/02/scitranslmed.3003380">paper</a>, by Bert Vogelstein and colleagues, on the prospect of genomic risk prediction. Though the paper&#8217;s <a href="http://en.wikipedia.org/wiki/Meta-analysis">meta-analysis</a> of twin studies has taken some <a href="http://www.lastwordonnothing.com/2012/04/06/what-the-limits-of-dna-story-reveals-about-the-challenges-of-science-journalism-in-the-big-data-age/">heat</a> for repackaging longstanding knowledge about heritability, the fuss over it has usefully underscored real complexities of disease and healthcare. And <a href="http://www.genomesunzipped.org/2012/04/identical-twins-usually-do-not-die-from-the-same-thing.php">thoughtful responses</a> to the paper have made clearer, to the public, that few geneticists are the zealous determinists of caricature. Rather, we tend to grasp the causal basis of disease (and other phenotypes) with inclusive nuance.</p>
<p>But while the dialog may have enlightened many, both the paper and <a href="http://www.ucsf.edu/news/2012/04/11864/value-genomics-and-personalized-medicine-wrongly-downplayed">its</a> <a href="http://storify.com/Erika_Check/the-power-of-predictive-sequencing">critiques</a> have largely missed two pertinent points:</p>
<ol>
<li>In the world of genetic risk, cancers are big exceptions.</li>
<li>Inasmuch, they prove some underlying rules.</li>
</ol>
<p>And in the context of the paper itself, these points nestle into one nutshell:</p>
<blockquote><p>
  In research, twins with tumors are no longer really twins.
</p></blockquote>
<p>To see why, and fold this point into the wider discourse on genomically personalized healthcare, let&#8217;s peer briefly through the looking-glass of cancer genetics.</p>
<h2>Gemini, Cancer, and genomic horoscopes</h2>
<p>As <a href="/category/genomes-and-health/">noted</a>, the new paper aimed to summarize what studies of twins say about how well our genomes, alone, may predict what diseases we&#8217;ll get. The premise, of course, is that twins from the same fertilized egg resemble live runs of a telling thought experiment: <em>if you and your genome lived twice, would you get the same diseases?</em></p>
<p>The basic answer has long been clear: twins don&#8217;t always get sick the same way. But Vogelstein and colleagues reasonably asked, for particular common adult diseases, how often they <em>do</em>.</p>
<p>Sensibly, they reviewed available twin data for several complex (and interrelated) epidemic killers, such as <a href="http://www.mayoclinic.com/health/heart-disease/DS01120">heart disease</a>, <a href="http://www.mayoclinic.com/health/diabetes/DS01121">diabetes</a>, and <a href="http://www.mayoclinic.com/health/stroke/DS00150">stroke</a>. To boot, they also combed the literature on <a href="http://en.wikipedia.org/wiki/Dystocia">pregnancy complications</a> (an intriguing evolutionary nexus of health), some <a href="http://en.wikipedia.org/wiki/Central_nervous_system_disease">nerve</a> and <a href="http://en.wikipedia.org/wiki/Autoimmune_disease">autoimmune diseases</a>, and more mysterious ailments like <a href="http://www.mayoclinic.com/health/chronic-fatigue-syndrome/DS00395">chronic fatigue</a> and <a href="http://www.mayoclinic.com/health/irritable-bowel-syndrome/DS00106">irritable bowel</a> syndromes.</p>
<p>But nine of the twenty-four diseases they surveyed &#8212; <em>the bulk, by class</em> &#8212; were <a href="http://en.wikipedia.org/wiki/Cancer">cancers</a>. And this choice sapped any suspense from their findings. For while cancers indeed kill many people (so <em>demand</em> study), they are long known to be far less heritable &#8212; that is, to show a smaller portion of cases running in families &#8212; than many other grave diseases. As Vogelstein (a <a href="http://en.wikipedia.org/wiki/Bert_Vogelstein">renowned</a> cancer researcher) surely knows, loading a heritability survey with cancers is like padding a Russian presidential ballot with token dissidents. Conclusion: foregone.</p>
<p>To be fair, the authors likely didn&#8217;t set out to mislead. After all, they could only survey available studies of twins. And a <a href="http://www.nejm.org/doi/full/10.1056/NEJM200007133430201">big study</a> of cancers was ripe to include (in the end, it supplied all their data on the question). While that study did find some evidence of genetic heritability in cancers, it concluded that such heritability plays little role in most cases, bolstering the well-established bottom line: in the grand scheme, cancers rarely run in families.</p>
<p>Where the authors shouldn&#8217;t be excused so easily is in <em>a</em>) actively hyping their findings as novel, while <em>b</em>) burying this key grain of salt, far from headlines and press releases, in a brief aside near the end of the paper:</p>
<blockquote><p>
  For diseases with a lower heritable component, such as most forms of cancer, whole-genome based genetic tests will be even less informative.
</p></blockquote>
<p>Which any clinical geneticist could have told you, thirty years ago.</p>
<h2>And yet&#8230;</h2>
<p>Unsurprisingly, all nine surveyed cancers in Vogelstein&#8217;s survey were deemed less genetically predictable (heritable) than the other diseases studied. Nonetheless, any oncologist will tell you that cancers are <em>quintessentially</em> genetic diseases. As we&#8217;ll see, they require &#8212; and are even defined by &#8212; genomes gone awry.</p>
<p>Indeed, some of the most widely screened-for genetic risk variants underlie rare familial forms of <a href="http://www.cancer.gov/cancertopics/factsheet/Risk/BRCA">breast</a> and <a href="http://en.wikipedia.org/wiki/APC_(gene)">colon cancer</a>. And whole genome interpretation is finding its <a href="http://www.pbs.org/wgbh/nova/body/cracking-your-genetic-code.html">first vital clinical use</a> in treating tumors, along with sick children.</p>
<p>So what gives? How can cancers be intrinsically genetic, yet so hard to predict from our genomes?</p>
<p>The answer, it turns out, is that it matters which genomes we mean.</p>
<h2>The hive within</h2>
<p>To understand cancer&#8217;s peculiar nature as a genetic disease, first picture your body&#8230;as a colony of bees.</p>
<p>Bear with me here: both entities &#8212; you and the swarm &#8212; are, ultimately, teeming masses of individuals (cells or bees), most of whom work hard to help just a few of their kind (eggs or sperm, or the hive&#8217;s queen and few males) breed on their behalf.</p>
<p>In this view, the two basic kinds of cells in your body &#8212; gametes (which biologists also call the <a href="http://en.wikipedia.org/wiki/Germline">germline</a>) and <a href="http://en.wikipedia.org/wiki/Somatic_cell">somatic</a> cells &#8212; respectively resemble a bee colony&#8217;s two main <a href="http://www.dummies.com/how-to/content/how-to-identify-the-three-castes-of-bees.html">castes</a>, breeders and workers.</p>
<p>In a sexual many-celled organism, like you, somatic cells vary kaleidoscopically in form, embodying the many specialized tissues that that help a body thrive from moment to moment. Gametes, by contrast, specialize mainly in storing DNA for coming generations, and &#8212; as big, slow, costly eggs, or small, fast, cheap sperm &#8212; in finding a partner gamete to merge with.</p>
<p>In most animals, the cells that become the germline split from other cells very early in embryonic development. As with bee royalty, it&#8217;s nearly impossible for a somatic commoner to infiltrate their reproductively privileged ranks later.</p>
<p>While details prompt <a href="http://whyevolutionistrue.wordpress.com/2010/08/30/a-misguided-attack-on-kin-selection/">debate</a>[1], natural selection is thought to have driven the emergence of the soma-germline and worker-queen distinctions, and in both cases to have brokered a lasting <a href="http://en.wikipedia.org/wiki/Eusociality">social</a> <a href="http://en.wikipedia.org/wiki/Social_compact">compact</a> between the two kinds of individuals.</p>
<p>In this contract, a somatic cell in your hand, like a dutiful worker bee, effectively says to your gametes</p>
<blockquote><p>
  <em>Cousins, if you spread our shared genes, I&#8217;ll give my life to help you. Count on me to build sturdy shelter, find good food, fend off attackers, write honeyed words to entice a mate, and care tenderly for the children that come after. You do the rest, and send our line forth.</em>
</p></blockquote>
<p>Seen in this <a href="http://en.wikipedia.org/wiki/Social_contract#Thomas_Hobbes.27_Leviathan_.281651.29">Hobbesian</a> light, a tumor is, effectively, a mutiny of somatic cells, who break that evolutionary covenant with their germline cousins. A budding tumor cell effectively says &#8216;Hell no, I won&#8217;t work and die for other cells &#8212; I&#8217;ll reproduce, myself, instead,&#8217; and starts proliferating unchecked.[2]</p>
<h2>A disease of genomes</h2>
<p>Crucially, the cellular treachery of each cancer case typically traces to one or more sudden changes &#8212; mutations &#8212; in the genome of a somatic cell. The mutations in question may be slight &#8212; say, the miscopying of one DNA letter from the parent cell&#8217;s genome. Or they may be drastic, as when a rogue gamma ray shatters a whole chromosome. In the latter case, the cell may, in mending the resulting fragments of DNA, inadvertently scramble them.</p>
<p>Whether slightly or severely altering DNA sequence, the mutations that turn cells into tumors tend to do so in particular ways. Typically, they either throw, or freeze stuck, one or more functional switches in the budding tumor cell&#8217;s genome &#8212; switches that had, til then, tightly governed the reproduction of that cell&#8217;s immediate ancestors, yoking their proliferation to the overall best interests of the developing body.</p>
<p>The switches in question are often protein-coding genes that directly govern <em>a</em>) whether a cell divides or, instead, takes a moribund, tissue-specific form; and/or <em>b</em>) how the cell exchanges signals with other cells, <em>e.g.</em>, halting growth at the touch of a neighboring cell, or telling that neighbor to build more blood vessels, to bring more food and oxygen. In some cases, the key switch may govern how well the cell corrects DNA copying errors; one mutation in such a gene may thus spark many more, some of which eventually knock out other switches that more directly reined in the cell&#8217;s growth.</p>
<p>In the end, a tumor grows unchecked thanks directly to one or more newly arisen genetic variants that distinguish it from other cells in the same body.[3] Importantly, such cellular mutinies are (with some exceptions[4]) typically doomed: in their greed for resources, the rebelling cells weaken the body overall, and, with no way to escape, sink with the ship that they&#8217;ve commandeered.</p>
<h2><em>Sui genetis</em></h2>
<p>All this switch-throwing ultimately means that tumors grow, spread, and kill <em>specifically because their genomes differ from other genomes</em>. As such, cancers are not just genetic in origin, but break a key assumption underlying Vogelstein&#8217;s paper: that monozygotic twins are genetically identical.</p>
<p>That is, as soon as a person is diagnosed with a tumor, all bets premised on her genetic identity – or even near identity – to a twin are off.[5] Not only do billions of her cells now differ genetically from those of her twin, but they differ in ways that are, by definition, biologically <em>important</em>. That is, the distinctive genetic variants in question drastically change the way that cells work, letting them divide unchecked.</p>
<p>In this sense, a tumor genetically distinguishes its host &#8212; of whom it is intrinsically part &#8212; from other people <em>de facto</em>, much as inherited genetic variants may distinguish someone with a strongly heritable disease from other people. Notably, this insight tempers any expectation that the person&#8217;s twin should get the same disease (a cancer driven by genetic variants that the twin likely doesn&#8217;t carry). And it underscores the uniformitarian rule that cancers, in their poor heritability, seem at first to violate: <em>in diseases of all stripes, genomic differences matter</em>.</p>
<h2>Tumor genomes: noisy, mixed, changing</h2>
<p>The first active clinical uses of whole genome sequencing have been in pediatrics and oncology. And this makes sense. In bluntly formal terms, sick kids and tumors are both masses of cells &#8212; one beloved, the other loathed &#8212; that are growing awry. And they&#8217;re both growing, as such, too fast for us to wait for sequencing to get cheaper, or for medical knowledge to get deeper. We&#8217;ll sequence <em>now</em>, if we can afford to, in order to gain some foothold into the medical mystery at hand.</p>
<p>For tumors in particular, we hope that sequencing the tumor (and, importantly, healthy tissue for comparison) will reveal key genetic clues to how it arose, grows, <a href="http://en.wikipedia.org/wiki/Metastasis">spreads</a>, and might be slowed or killed. Alas that turns out to be hard to do, for three key reasons.</p>
<p>First, as noted, tumor genomes are pocked by mutation. Small spelling changes often abound, hiding a few functionally important ones (called <em>drivers</em>) in a cacophony of incidental noise (<em>passengers</em>). And, at bigger scales, long segments of chromosomes are often repeated, missing, or scrambled. Such rampant genetic variation is not just tough to functionally interpret, but also makes it hard to draw an accurate picture of the genome in the first place. Why? It turns out that the computer algorithms used in modern sequencing work poorly for genomes that differ greatly from the standard reference genome, because snippets of raw sequence data that don&#8217;t match up well to that genome (like puzzle pieces that don&#8217;t closely match the picture on the box) are hard to correctly place. Moreover, tracts of DNA letters that appear in multiple spots in a genome (like uniform fenceposts in a farmscape puzzle) are especially hard to accurately sequence &#8212; an acute challenge in tumors, where rampant mutational copying, cutting, and pasting turns the genome into a bewildering <a href="http://www.nature.com/nature/journal/v483/n7391/full/nature10910.html">house of broken mirrors</a>.</p>
<p>Second, each tumor actually harbors not one, but a <em>mix</em>, of such noisy genomes &#8212; often with non-tumor cell genomes inadvertently mixed in. While a tumor is indeed a clump of closely related cells that distinctively share particular variants, it&#8217;s also a <em>population</em> of genetically varied cell lines, each effectively striving to grow faster than the others, thanks to its own secondary stock of functionally relevant variants. But because sequencing today requires pooling many cells, genetic variation in the tumor tends to get homogenized, as if in a blender. An important variant carried only in a few cells may not be prominent enough to show up in the final reckoning of a singular tumor genome sequence.</p>
<p>Third, the mix of genomes in the tumor <em>evolves</em>, partly in response to treatment. Thus we may want to track how treating the tumor with a particular regimen kills of some cell lines in the tumor, while letting other lines, by chance resistant to the treatment, spread quickly. To best characterize and treat a tumor, we might want to see a <em>movie</em>, rather than just a snapshot, of its mix of genomes, letting us watch how they change in response to treatment. But doing so is, for the foregoing reasons, tough &#8212; and will be until we can sequence fewer cells at a time, for less money, and with longer snippets of raw sequence (analogous to bigger puzzle pieces that can be more reliably pieced together to get the whole moving picture).</p>
<h2>Epilogue</h2>
<p>On a late summer afternoon when I was six, my mom came to my room, sat next to me, and showed me fresh bruises on her arms and legs. Speaking with determined nonchalance, she trained a young boy&#8217;s restless attention to a moment of revelation.</p>
<p>Each squall-blush in her skin was, I learned, real bloodshed from a war below. In the marrow of her bones, delinquent cells were teeming, wrecking the nurseries of sticky <a href="http://en.wikipedia.org/wiki/Platelet">platelets</a> that she needed to heal small, everyday blood vessel leaks. The bruises were collateral damage from that mutiny &#8212; her own cells betraying her, and those she loved.</p>
<p>She sought treatment, but <a href="http://www.mayoclinic.com/health/leukemia/DS00351">leukemia</a> wore her down quickly. On Halloween night, muted by breathing tubes in intensive care, she could welcome my visit only with a tiny nod and a waiting cup of candy. She died a few days later, at 34&#46; My first-grade classmates, struggling to comprehend from afar, sketched colorful cards of condolence that I still keep.</p>
<p>Writing today, on her birthday, I&#8217;m older than she got to be. Childhood reading, long before Vogelstein&#8217;s paper, taught me that leukemia shows little heritability. Yet I still watch for bruises&#8230;and admit to a tinge of affirmed relief that, among the diseases that the paper assayed for genetic predictability, leukemia came in last.</p>
<p>But the comfort is cold. Cancers remain an especially vexing kind of plague: sprung from our own selves, tumors are, in the paper&#8217;s geminal terms, something like evil conjoined twins. Growing relentlessly, ever changing, they cloak genomic secrets in genomic smoke, and evade our harshest treatments. They are the last horcrux.</p>
<p>At a recent conference, I listened to Washington University&#8217;s <a href="http://genome.wustl.edu/people/mardis_elaine">Elaine Mardis</a> explain how she and her colleagues are systematically characterizing the genomes of thousands of tumors. Their yeoman work is building a broadly useful critical mass of detailed knowledge about how tumors arise, grow, and spread. But their findings are also helping real doctors and patients, today, choose treatments that lengthen lives and lessen suffering.</p>
<p>After Mardis&#8217;s talk, I told her how touched I am not just by her team&#8217;s work itself, but to know that <a href="http://genome.wustl.edu/">Wash U</a> (where my mom earned her <a href="http://books.google.com/books/about/Marriage_in_the_Novels_of_Arnold_Bennett.html?id=maj6tgAACAAJ">PhD</a>) and Barnes Hospital (where she gave birth to me, and died) now spearhead a data-driven fight against leukemia and other cancers.</p>
<p>And I&#8217;m proud that my own work at <a href="http://www.knome.com/">Knome</a> supports such efforts. By thoroughly characterizing tumor genomes, and developing algorithms to do so better, we&#8217;re helping clinical researchers spot genetic variants that directly drive tumor growth or, more rarely, predispose some families to recurrent cancers. That work, like Mardis&#8217;s, is already leveraging individuated genome data to help people live longer.</p>
<p>Looking ahead, those of us lucky enough to afford good healthcare today will likely be talking a lot about tumor genomes, with our families and friends, in coming decades. Relevant insights will likely guide vital choices for many of us and those we love.</p>
<p>We&#8217;ll see how soon other major adult diseases &#8212; think of those surveyed by Vogelstein <em>et al.</em>, but also of liver and kidney diseases, mental illnesses, breathing problems (asthma, respiratory tract infections), bone diseases, <em>etc</em>. &#8212; likewise become more amenable to personal genomic insights, first for diagnosis, and, in the long run, for prognosis too. Here&#8217;s to those twin prospects.</p>
<p>[1] In bees (as well as ants and termites), this compact is thought to be reinforced by the fact that a queen and her worker sister may be especially closely related &#8212; often moreso than either would be to her own daughter. This odd twist of kinship follows from male bees having just one copy of each chromosome (having hatched from unfertilized eggs), while females have two (hatching from fertilized eggs). As a result, bee full sisters share, on average, three-fourths of their DNA with each other, while mothers and daughters share just half.</p>
<p>In a real hive, the numbers are complicated by many pairs of workers being just half-sisters (with different dads). But that turns out not to matter much; the strong social contract of eusociality can be mathematically understood even without the extra-closeknit kinship that bee sisters share.</p>
<p>[2] As in many personified examples of rivalry between organisms, the tumor cell&#8217;s effective strategy is not conscious, but rather a mathematical truism &#8212; that is, cells in which a chance mutation tears the web of reproductive constraint that natural selection has woven will, at least in the short term, tend to outgrow neighboring cells.</p>
<p>[3] Importantly, a person may be born with at least one such genetic switch already thrown; this scenario underlies rare cases of strong familial cancer risk. Cancer can then strike as soon as a second switch &#8212; a copy of the same gene, or another gene &#8212; is randomly thrown by a new mutation.</p>
<p>People born with one broken copy of the cell growth-suppressing <em>RB1</em> gene, for example, tend to eventually get tumors in both eyes. In such cases, their working second copy of <em>RB1</em> tends to eventually mutate in one of the fast-dividing, light-bombarded cells of the retina of one eye, leading to a first tumor. Later, a second such mutation may strike a retinal cell in the other eye, spawning a tumor there too.</p>
<p>Thus even in rare forms of cancers that run in families, getting a tumor requires some new mutation in a somatic cell. And that mutation may, in turn, be driven by some environmental factor (such as radiation, toxins, or even infectious germs), highlighting that both genetic and environmental factors are key to understanding cancers &#8212; just as in other diseases.</p>
<p>[4] One tumor that beat the odds, living on beyond its original victim, belonged to American <a href="http://en.wikipedia.org/wiki/Henrietta_Lacks">Henrietta Lacks</a>. As detailed in Rebecca Skloot&#8217;s compelling <a href="http://rebeccaskloot.com/the-immortal-life/">personal and cellular biography</a>, in 1951, Lacks&#8217;s doctor took cells, without her consent, from the ovarian tumor that was killing her, and grew them in dishes. Though born in suffering and scientific misconduct, those cells nonetheless proved remarkably resilient in laboratory culture, and have since spread worldwide as a key resource for six decades of biomedical research.</p>
<p>Another tumor that slipped the mortal coil of its origins is the contagious facial cancer that plagues Tasmanian devils &#8212; the subject of a fascinating recent <a href="http://www.cell.com/retrieve/pii/S0092867412000815">paper</a>. And some rare <a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002420">cancers of <em>germline</em></a> (rather than somatic) cells may manage to make gametes healthy enough to transmit themselves &#8212; rampant cell growth and all &#8212; to coming generations.</p>
<p>[5] Of course, even monozygotic twins without tumors aren&#8217;t really genetically identical. The genome&#8217;s great size, and the many generations of mutation-prone cell division needed to build the myriad cells in their bodies, nearly assure that no single cell in either one is genetically identical to the fertilized egg from which they came &#8212; much less to a typical cell pulled from the other twin.</p>
<p>But tumors flout the genetic identity assumption of twin studies even more severely. The many cells that make up a tumor tend to be especially closely related to each other, thanks to their recent growth spurt. Thus the newly arisen genetic variants that the tumors&#8217; cells share, and that distinguish them from other cells in the body (and even moreso from cells of the other twin, thanks to the extra rounds of cell division that separate the two cell lineages in question), are especially common among the cells of the person overall. As such, if we think of a person&#8217;s genome as comprising, at each site, a weighted mix of the genotypes of all her cells, then a tumor makes her particularly genetically distinctive &#8212; even beyond the prospect that those differences may be more functionally important in tumors than in other kinds of cells.</p>
]]></content:encoded>
			<wfw:commentRss>/2012/04/19/why-cancers-are-the-least-and-most-genetic-of-diseases/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>On twins, genomes, and health</title>
		<link>/2012/04/04/on-twins-genomes-and-health/</link>
		<comments>/2012/04/04/on-twins-genomes-and-health/#respond</comments>
		<pubDate>Wed, 04 Apr 2012 14:09:51 +0000</pubDate>
		<dc:creator><![CDATA[Nathaniel Pearson]]></dc:creator>
				<category><![CDATA[Genomes and health]]></category>

		<guid isPermaLink="false">/?p=50</guid>
		<description><![CDATA[Twins have sometimes no resemblance to each other. – Aristotle, The History of Animals A much-discussed new paper from Nicholas Roberts, Bert Vogelstein, and colleagues...]]></description>
				<content:encoded><![CDATA[<blockquote><p>
  Twins have sometimes no resemblance to each other.<br />
  – Aristotle, <em>The History of Animals</em>
</p></blockquote>
<p>A much-<a href="http://blogs.nature.com/news/2012/04/questioning-value-personal-genomics.html">discussed</a> new <a href="http://stm.sciencemag.org/content/early/2012/04/02/scitranslmed.3003380">paper</a> from Nicholas Roberts, Bert Vogelstein, and colleagues starts with a simple but important observation: so-called identical twins don&#8217;t always get the same diseases.</p>
<p>They use that to tackle an age-old question: how genetically <a href="http://en.wikipedia.org/wiki/Heritability"><em>heritable</em></a> is each disease overall? And, beyond that, how does genetic risk vary among people? That is, for a given disease, do genomes contribute a portion of risk that varies widely, but smoothly, from person to person&#8211;or a portion of risk that varies more sharply from person to person&#8211;or some mix in between?</p>
<p>Below are some thoughts on reading the paper (which, while <a href="http://www.michaeleisen.org/blog/">not originally freely available</a>, is now free to download if you register at <a href="http://stm.sciencemag.org/">Science Translational Medicine&#8217;s</a> site).</p>
<h2>Twin studies: Gee, these findings look familiar&#8230;</h2>
<p>Geneticists have long looked to twins to measure genetic heritability, and the new paper &#8212; which presents no new data, but instead re-analyzes past twin studies &#8212; follows in that tradition. Unsurprising, then, that the main conclusion boils down to the well established starting observation: <em>even <a href="&lt;http://en.wikipedia.org/wiki/Monozygotic_twins#Monozygotic>_.28.22identical.22.29_twins&#8221;>twins from the same egg</a>, who are as genetically similar as people can be, don&#8217;t always get sick the same way.</em></p>
<p>Where Vogelstein and colleagues help refine that classic question of heritability in twins is in trying to <em>sort</em> particular diseases by estimating how well, on average, one might forecast those diseases from genomes alone.</p>
<p>To do so, they propose a simplistic model of the genetic basis of disease risk. Note that simplistic models are common in science &#8212; and, in statistical genetics, in particular, we still have a lot to learn about the joint distributions of</p>
<ul>
<li><a href="http://en.wikipedia.org/wiki/Allele_frequency">allele frequency</a> (<em>How common is a given genetic variant in people around the world?</em>)</p>
</li>
<li>
<p>allelic effects on <a href="http://en.wikipedia.org/wiki/Phenotype">phenotype</a> (<em>How much does having a copy (or two) of this variant affect one&#8217;s odds of having or getting disease X, overall?</em>)</p>
</li>
<li>
<p><a href="http://en.wikipedia.org/wiki/Epistasis">epistasis</a> (<em>How do the effects of this variant depend on the presence of other variants in your genome?</em>)</p>
</li>
</ul>
<p>and</p>
<ul>
<li><a href="http://en.wikipedia.org/wiki/Pleiotropy">pleiotropy</a> (<em>What other phenotypes does this variant help govern, and how do those effects interact?</em>).</li>
</ul>
<p>Given those open challenges, I won&#8217;t dwell here on how well assumptions in the author&#8217;s model likely fit biological reality, other than to note that the authors do implicitly presume simple answers (often necessarily so) to the foregoing questions, along with some arbitrary hard thresholds for defining what it means for two people to informatively differ in disease risk. Having set their model, they then tune it to previous twin data for each of 24 diseases, to estimate how robust and variable (from person to person) resulting estimates of individual genetic risk might be.</p>
<p>They forecast that, for the average person, the spectrum of such risk estimates based solely on the genome may be just modestly helpful overall &#8212; a guess that, again, accords the basic observation that monozygotic twins don&#8217;t always get the same diseases. The authors go on to posit that genome data may best help clarify our personal risks better for some particular diseases –- they call out autoimmune diseases like <a href="http://www.mayoclinic.com/health/type-1-diabetes/DS00329">type-I diabetes</a> and forms of <a href="http://thyroid.about.com/cs/hypothyroidism/a/hashivshypo.htm">thyroid disease</a>, as well as the nerve disease <a href="http://www.mayoclinic.com/health/alzheimers-disease/DS00161">Alzheimer&#8217;s</a> &#8212; than for others.</p>
<p>And here arises a key caveat: many of the diseases they look at are <em>cancers</em>, which have long been known to be less genetically heritable than some other diseases. And, in this sense, the paper recasts old-hat knowledge as if it were a new grain of salt for the coming era of genomically personalized healthcare.</p>
<h2>Health: the integrative view abides.</h2>
<p>In full disclosure, my work at <a href="http://www.knome.com/">Knome</a> entails analyzing human whole genomes in order to better understand health; thus we clearly have a stake in public discussion of the prospective clinical utility of whole genome sequencing. But, in that role (where I&#8217;ve had to explain many a genome to many a curious, fairly healthy person), I&#8217;ve always stressed that one key duty of our work is to help people understand what our genomes can &#8212; <em>and can&#8217;t</em> &#8212; tell us. And, looking ahead to the prospect of genomically personalized healthcare, we have always understood that genomes will crucially <em>complement</em>, not replace, more conventional cornerstones of clinical care.</p>
<p>That is, face-to-face doctor visits, family history, lab tests, and so forth will remain essential pieces of the healthcare puzzle, soon to be joined by genome sequences. But no such piece, alone, stands to tell us everything we need to know about disease risk: lab tests often happen too late, for example; and family history is limited in utility for precisely the same reasons that this new paper highlights.</p>
<p>As a rough analog, note that weather forecasters use <a href="http://en.wikipedia.org/wiki/Satellite_imagery">satellite photos</a> every day, for remarkably detailed insight into what&#8217;s happening in the atmosphere; nonetheless, they wouldn&#8217;t try to predict today&#8217;s high temperature in Springfield (any of them) solely from such photos. Rather, they merge the modern, comprehensive data from orbiting cameras with data from older earthbound instruments to predict the weather (and, of course, they still get it wrong sometimes&#8230;).</p>
<p>Likewise, modern, comprehensive data from our genomes will likely come to play a key role in healthcare for many of us &#8212; but always together with other key sources of medical insight. Good geneticists know that it&#8217;s crucial not to hype genomes as silver bullets of healthcare. To do so would a) disserve the public, and b) in raising unreasonable expectations, risk a backlash from private and public funders of healthcare and genomic research itself.</p>
<p>Yet here it&#8217;s worth stressing that <a href="http://www.pbs.org/wgbh/nova/body/cracking-your-genetic-code.html">discoveries from individual genomes</a> have already helped many families for many decades, ever since the first readily screened <a href="http://en.wikipedia.org/wiki/Penetrance">penetrant</a> genetic disease variant, for <a href="http://en.wikipedia.org/wiki/Sickle_cell_anemia">sickle cell anemia</a>, was <a href="http://www.ncbi.nlm.nih.gov/pubmed/13369537">characterized</a> in the late 1950s. Looking ahead, as we survey more people&#8217;s whole genomes, along with their diseases and other traits, our DNA will indeed tell us more and more about what makes each of us unique, and about the distinctive health risks we face.</p>
<h2>Toward cautious optimism</h2>
<p>Overall, findings from the new paper model temper any undue expectation that whole genome interpretation might offer slam-dunk insight into longterm risks for the spectrum of major diseases for a typical fairly healthy person. But, perhaps promisingly, they suggest that whole genome interpretation may nonetheless offer most people &#8212; the authors estimate more than nine in ten of us &#8212; some significant hint of distinctive genetic risk for at least one such major disease.</p>
<p>As noted, they call out particular diseases as likely most often informatively predictable; admirably, in doing so, they offer specific predictions that will be testable as science progresses. In the long run, we will indeed learn whether readily genetically predictable common diseases tend to fit particular profiles; given that, as a class, cancers appear not to fit that bill, it&#8217;s plausible that some other classes of disease will tend to fit better.</p>
<p>Moreover, the paper acknowledges the clear point that whole genome sequencing can help us spot strong risks for much rarer serious diseases that may lurk in our genomes. This may be especially useful in planning families, where we&#8217;d like to know what few rare but potentially functionally important variants, if any, we may share with a spouse (and that might be harmful if, by chance, inherited together in a child).</p>
<h2>Genomic medicine <em>presumes</em> it&#8217;s not all in your genes.</h2>
<p>It&#8217;s important to understand whether the new paper debunks some notion that our fates are all written in our genes. But spinning the &#8216;<a href="http://www.salon.com/2012/04/02/gene_mapping_for_everyone_study_says_not_so_fast/">no crystal ball</a>&#8216; finding as big news in itself would be spinning a straw man: we&#8217;ve long known that genetic risk is not immutably deterministic –- and the whole field of genomic medicine is actually <em>founded</em> on that fact. That is, in trying to understand genetic risk, we hope learn how to mitigate it, by changing the environment of our habits –- through what we eat, what drugs we take, and how/where we otherwise spend our time. Our genomes should eventually help many of us refine those choices in key ways.</p>
<h2>Whole genome sequencing and cancers</h2>
<p>Although the paper rehashes the long-known point that cancer risk tends to show fairly little genetic heritability (other than for sex-specific cancers tracing to the sex chromosomes themselves), some of the first clear examples of how useful genome sequencing can be are nonetheless in familial cancers, such as cases of breast and ovarian cancers tracing to inherited variants in the <a href="http://www.omim.org/entry/113705"><em>BRCA1</em></a> and <em><a href="http://www.omim.org/entry/600185?search=brca2&amp;highlight=brca2">BRCA2</a></em> genes.</p>
<p>Moreover, sequencing the genomes of <em>tumors</em> (which the paper doesn&#8217;t address) is already revolutionizing how cancer is treated, by finding key changes to the genomes of particular cells in the body that let them grow out of control. Such sequencing is reshaping how oncologists think of cancers – from a simplistic tissue-specific view, to one that highlights recurrent genetic changes shared by tumors from different tissues, which may represent vulnerable targets for particular drugs or other treatments.</p>
<h2>Further quibbles: Twins R Us?</h2>
<p>In re-analyzing data from earlier twin studies, the paper risks some small concerns that always dog such studies. As the authors acknowledge, their papers presumes that European monozygotic twins validly represent everyone, <em>i.e.,</em> that their assertions:</p>
<ul>
<li>will generalize to other ethnicities</p>
</li>
<li>
<p>aren&#8217;t confounded by ascertainment bias (&#8216;<em>Doc, my twin has disease X. Do I?</em>&#8216;&#8230;). Note that such bias may which might tend to <em>overestimate</em> heritability (if twins tend to be more thoroughly diagnosed, or more often misdiagnosed thanks to another person&#8217;s diagnosis, than do other people), or <em>underestimate</em> it (if one twin, on seeing the other get sick, takes better and earlier preventive measures than (s)he would otherwise have taken).</p>
</li>
<li>
<p>aren&#8217;t distorted by monozygotic twins&#8217; distinctive health profiles. Potential factors to consider include: low birth weight (perhaps reflecting unusually stringent competition for resources in the womb); distinctive profiles of maternal age or genotype; subtle underlying quirks of early embryonic cell division/other physiology; lifelong social support from having a very similar sibling; etc.</p>
</li>
</ul>
<p>The last few concerns are less damning than those above them; after all, as noted, the basic findings of the paper boil down to the fact that twins don&#8217;t get the same diseases. But, like all the foregoing, they&#8217;re worth keeping in mind as research into the causes of disease &#8212; genetic and otherwise &#8212; continues on all our behalf.</p>
]]></content:encoded>
			<wfw:commentRss>/2012/04/04/on-twins-genomes-and-health/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
