---
publish-date: 'Friday, March 22nd, 2013, 1:42 pm'
tags: Genomes and health
---

# Harmful by any other name: On clinical variant classification

In Phoenix this week, clinical geneticists have gathered at [ACMG](http://www.acmgmeeting.net/acmg2013/public/enter.aspx) to catch up on health-relevant genomic findings and tools, and decide how to best put them into broad practice. Tonight, in particular, features a workshop on how to classify DNA spelling variants, and report them to patients and caregivers. Chief among questions at hand will be what to _call_ such variants.

Led by the ever-thoughtful [Heidi Rehm](http://www.bizjournals.com/boston/stories/2010/10/11/focus38.html), the workshop is a great chance to define a sensible set of classification bins that best convey how each genetic variant likely affects the health of a person who might carry it.

With an eye to that question, here's the most widely used binning scheme, put forth by [Kearney _et al._](http://www.ncbi.nlm.nih.gov/pubmed/?term=21681106)[1]:

- Pathogenic
- Unknown significance, Likely pathogenic
- Unknown significance
- Unknown significance, Likely benign
- Benign

And here, with some explanation to follow below, is an alternative to kick around:

- Harmful
- Likely harmful
- Not yet interpretable
- Likely harmless
- Harmless

  - Family-relevant
  - Treatment-relevant

In weighing these two schemes, first note that Kearney's thoughtfully builds on a legacy that's deeply familiar to many clinical geneticists -- and has proved a good, if imperfect, rubric for current use. But as we slowly integrate genomic insights into healthcare for ever more people, from birth to old age, Kearney's shortcomings may limit its shelf life. Let's review.

## What the Kearney scheme gets right

_It's sensibly simple._ Importantly, Kearney (the first scheme above) sorts variants by our best guesses about how they affect health -- and uses a plain word (_likely_) to neatly distinguish weak guesses from strong ones, without dickering about likelihoods of likelihoods. No arbitrary hair-splitting.

_It's realistically fluid._ By making uncertainty explicit, Kearney accomodates changing knowledge. A variant understood poorly today, but well tomorrow, can just be reclassified from the cautiously neutral middle of the rubric toward one of its poles -- or back again.

_It's accurate._ Kearney implicitly uses the generalizable term _variant_, not alternatives like _mutation_ and _[polymorphism](http://genomena.com/concepts-genetics-jargon/polymorphism/)_, whose usages have drifted in misleading ways. In the long run, this can help people understand their genomes accurately -- more on that another time.

## What the Kearney scheme gets wrong

_It calls carrier variants pathogenic._ Some variants -- _dominant_ ones -- cause disease if found on either of a person's copies of a chromosome. But others -- mostly _recessive_ -- do harm only if found on _each_ of that person's copies, or along with another variant elsewhere in the genome. Kearney lumps all such variants together, so a man with one copy of a dominant variant, one copy of an autosomal recessive variant, one copy of an X-linked recessive variant, and two copies of an ovarian cancer-causing variant, will misleadingly see all four identically called _pathogenic_ in his report. This problem reflects how such schemes simplistically focus on _variants_, instead of _genotypes_. But, as we'll see, they could nonetheless clarify which worrisome variants will or won't make a given person sick. Moreover, distinguishing such variants can help us know which to watch for in other family members (especially future children, who might get copies from both parents, or who might crucially differ in sex from the parent who gave them a particular variant).

_It says nothing about drug (or other treatment) response._ Beyond calling predictably harmless carrier variants _pathogenic,_ Kearney's scheme ignores variants that predict response to drugs and other disease treatments. Notably, such variants are, for now, among the most informative variants in our genomes -- especially for fairly healthy adults (that is, those of us not _urgently_ scanning our genomes for key insight into disease). As is, Kearney would thus glaringly omit key insight from a typical whole-genome report.

_It's unclear._ Like much medical jargon (see discussion below), Kearney uses fancy, euphemistic words from Greek and Latin (_pathogenic_ = 'sick-making', _benign_ = 'good') instead of plainer words understood by more people[3]. Given the bully pulpit that ACMG has at the dawn of genomic medicine, the scheme that it picks will likely shape how people learn about their health for years to come. As such, the jargon-versus-plain-English question is key: Will genome reports coddle _doctors_ by sticking with the latinate jargon they love, or will we use plainer synonyms that _patients_ too understand?

_It's awkward._ In the Kearney-inspired parlance of clinical geneticists, the ungainly phrase _Variant of Unknown Significance_ has long stuck out like a Thumb of Unrelenting Soreness on a [Rodent of Unusual Size](http://www.urbandictionary.com/define.php?term=R.O.U.S.). Unlike the other bin names, _VUS_ isn't an adjective. Meanwhile, _pathogenic_ and _benign_ are odd antonyms, as the former may sound like it involves germs, and the latter more often opposes _malignant_, in describing tumors. Such inconsistency -- and the whiffs of germ and cancer talk -- may not help patients understand easily.

## An alternative

With those shortcomings in mind, let's review the newly proposed alternative scheme. While closely resembling Kearney, it arguably refines it in several ways.

First, by using plain words like _harmful_ and _harmless_, it would be immediately clear to many layfolk, and avoid patronizing euphemism. As such, a fresh step toward empowering patients in the coming era of genomically informed healthcare.

Second, the proposed scheme comprises only adjectives -- mostly short ones -- with poles defined by simple _-ful_ and _-less_ antonyms[4]. The adjective phrase _not yet interpretable_, while still a bit awkward, plainly conveys both neutrality and openness to change.

Third, it highlights both carrier variants (_family-relevant_) and _treatment-relevant_ variants in plain ways, accessible to both doctor and patient, that don't unduly alarm or reassure. Note that while I've tentatively filed both new classes under _harmless_, based on their effects for the patient at hand, they don't fit neatly there. Note too, that many drug response variants, in particular, may even be _helpful –_ a category generally lumped in under _benign/harmless_ in all these schemes.

At tonight's workshop, I may float some of the ideas proposed here. While response may well be lukewarm at best, given the momentum that Kearney-like clinical genetics nomenclature has already gained, there may be no better time to speak up on the matter.

[1] The scheme was first proposed for variants involving long repetitive chunks of the genome, but has now been adopted more broadly.

[2] It's worth asking, however, whether those two terms are too hard to uniquely abbreviate (_HF_? _HL_?), or would be readily confused in typing, reading, or speaking. If so, could an even shorter term like _OK_ could sub for _harmless_?

[3] This habit likely dates to when Latin was the key shared language of European scholars – letting healers in, say, England, Italy, and Poland easily share new discoveries with each other.

Today, of course, science's [lingua franca](http://en.wikipedia.org/wiki/Lingua_franca) is English (and tomorrow perhaps Mandarin...). But while English taps two big stocks of words for the same things – pithy Germanic (_shit, yearly_) and flowery Greco-latinate (_excrement, annually_) – doctors (like lawyers and us scientists) find that continuing to cloak their work in layers of Greco-latinate jargon does three handy things.

First, it discourages DIY (or other) alternatives to their services. (_Ah, I have a contusion that may become a hematoma? Wow, thanks, doc – and I thought it was just a bad bruise..._).

Second, it hides knowledge from others – sometimes rightly (eavesdroppers in a hospital lift), sometimes not (an adult patient who wants frank talk, not patronizing euphemism).

Third, it binds them (well, us scientists too) in clubby solidarity, perhaps safeguarding a sense of prestige (_As we learned in med school, the reeeal word for right pinky is digitus minimus manus dextra..._).
